<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en-US" xml:lang="en-US">
<head>
  <meta charset="utf-8" />
  <meta name="generator" content="pandoc" />
  <meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes" />
  <meta name="author" content="Jesse G. Meyer" />
  <meta name="author" content="Dina Schuster" />
  <meta name="author" content="Amit Kumar Yadav" />
  <meta name="author" content="Muralidharan Vanuopadath" />
  <meta name="author" content="Devasahayam Arokia Balaya Rex" />
  <meta name="author" content="Benjamin A. Neely" />
  <meta name="author" content="Martín L. Mayta" />
  <meta name="author" content="Emma H. Doud" />
  <meta name="dcterms.date" content="2022-07-12" />
  <meta name="keywords" content="markdown, publishing, manubot" />
  <title>A Practical Beginner’s Guide to Proteomics</title>
  <style>
    code{white-space: pre-wrap;}
    span.smallcaps{font-variant: small-caps;}
    span.underline{text-decoration: underline;}
    div.column{display: inline-block; vertical-align: top; width: 50%;}
    div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
    ul.task-list{list-style: none;}
    div.csl-bib-body { }
    div.csl-entry {
      clear: both;
    }
    .hanging div.csl-entry {
      margin-left:2em;
      text-indent:-2em;
    }
    div.csl-left-margin {
      min-width:2em;
      float:left;
    }
    div.csl-right-inline {
      margin-left:2em;
      padding-left:1em;
    }
    div.csl-indent {
      margin-left: 2em;
    }
  </style>
  <!--
  Manubot generated metadata rendered from header-includes-template.html.
  Suggest improvements at https://github.com/manubot/manubot/blob/main/manubot/process/header-includes-template.html
  -->
  <meta name="dc.format" content="text/html" />
  <meta name="dc.title" content="A Practical Beginner&#39;s Guide to Proteomics" />
  <meta name="citation_title" content="A Practical Beginner&#39;s Guide to Proteomics" />
  <meta property="og:title" content="A Practical Beginner&#39;s Guide to Proteomics" />
  <meta property="twitter:title" content="A Practical Beginner&#39;s Guide to Proteomics" />
  <meta name="dc.date" content="2022-07-12" />
  <meta name="citation_publication_date" content="2022-07-12" />
  <meta name="dc.language" content="en-US" />
  <meta name="citation_language" content="en-US" />
  <meta name="dc.relation.ispartof" content="Manubot" />
  <meta name="dc.publisher" content="Manubot" />
  <meta name="citation_journal_title" content="Manubot" />
  <meta name="citation_technical_report_institution" content="Manubot" />
  <meta name="citation_author" content="Jesse G. Meyer" />
  <meta name="citation_author_institution" content="Department of Computational Biomedicine, Cedars Sinai Medical Center" />
  <meta name="citation_author_orcid" content="0000-0003-2753-3926" />
  <meta name="twitter:creator" content="@j_my_sci" />
  <meta name="citation_author" content="Dina Schuster" />
  <meta name="citation_author_institution" content="Department of Biology, Institute of Molecular Systems Biology, ETH Zurich, Zurich 8093, Switzerland" />
  <meta name="citation_author_institution" content="Department of Biology, Institute of Molecular Biology and Biophysics, ETH Zurich, Zurich 8093, Switzerland" />
  <meta name="citation_author_institution" content="Laboratory of Biomolecular Research, Division of Biology and Chemistry, Paul Scherrer Institute, Villigen 5232, Switzerland" />
  <meta name="citation_author_orcid" content="0000-0001-6611-8237" />
  <meta name="twitter:creator" content="@dina_sch" />
  <meta name="citation_author" content="Amit Kumar Yadav" />
  <meta name="citation_author_institution" content="Translational Health Science and Technology Institute" />
  <meta name="citation_author_orcid" content="0000-0002-9445-8156" />
  <meta name="twitter:creator" content="@theoneamit" />
  <meta name="citation_author" content="Muralidharan Vanuopadath" />
  <meta name="citation_author_institution" content="School of Biotechnology, Amrita Vishwa Vidyapeetham, Kollam-690 525, Kerala, India" />
  <meta name="citation_author_orcid" content="0000-0002-9364-917X" />
  <meta name="twitter:creator" content="@V_MuraleeDhar" />
  <meta name="citation_author" content="Devasahayam Arokia Balaya Rex" />
  <meta name="citation_author_institution" content="Center for Systems Biology and Molecular Medicine, Yenepoya Research Centre, Yenepoya (Deemed to be University), Mangalore 575018, India" />
  <meta name="citation_author_orcid" content="0000-0002-9556-3150" />
  <meta name="twitter:creator" content="@rexprem" />
  <meta name="citation_author" content="Benjamin A. Neely" />
  <meta name="citation_author_institution" content="Chemical Sciences Division, National Institute of Standards and Technology, NIST Charleston" />
  <meta name="citation_author_orcid" content="0000-0001-6120-7695" />
  <meta name="twitter:creator" content="@neely615" />
  <meta name="citation_author" content="Martín L. Mayta" />
  <meta name="citation_author_institution" content="School of Medicine and Health Sciences, Center for Health Sciences Research, Universidad Adventista del Plata, Libertador San Martín 3103, Argentina" />
  <meta name="citation_author_institution" content="Molecular Biology Department, School of Pharmacy and Biochemistry, Universidad Nacional de Rosario, Rosario 2000, Argentina" />
  <meta name="citation_author_orcid" content="0000-0002-7986-4551" />
  <meta name="twitter:creator" content="@MartinMayta2" />
  <meta name="citation_author" content="Emma H. Doud" />
  <meta name="citation_author_institution" content="Center for Proteome Analysis, Indiana University School of Medicine, Indianapolis, Indiana, USA" />
  <meta name="citation_author_orcid" content="0000-0003-0049-0073" />
  <meta name="twitter:creator" content="@fireinlab" />
  <link rel="canonical" href="https://jessegmeyerlab.github.io/proteomics-tutorial/" />
  <meta property="og:url" content="https://jessegmeyerlab.github.io/proteomics-tutorial/" />
  <meta property="twitter:url" content="https://jessegmeyerlab.github.io/proteomics-tutorial/" />
  <meta name="citation_fulltext_html_url" content="https://jessegmeyerlab.github.io/proteomics-tutorial/" />
  <meta name="citation_pdf_url" content="https://jessegmeyerlab.github.io/proteomics-tutorial/manuscript.pdf" />
  <link rel="alternate" type="application/pdf" href="https://jessegmeyerlab.github.io/proteomics-tutorial/manuscript.pdf" />
  <link rel="alternate" type="text/html" href="https://jessegmeyerlab.github.io/proteomics-tutorial/v/1527fe26c63f130aba0979d41345a20680c6e77a/" />
  <meta name="manubot_html_url_versioned" content="https://jessegmeyerlab.github.io/proteomics-tutorial/v/1527fe26c63f130aba0979d41345a20680c6e77a/" />
  <meta name="manubot_pdf_url_versioned" content="https://jessegmeyerlab.github.io/proteomics-tutorial/v/1527fe26c63f130aba0979d41345a20680c6e77a/manuscript.pdf" />
  <meta property="og:type" content="article" />
  <meta property="twitter:card" content="summary_large_image" />
  <link rel="icon" type="image/png" sizes="192x192" href="https://manubot.org/favicon-192x192.png" />
  <link rel="mask-icon" href="https://manubot.org/safari-pinned-tab.svg" color="#ad1457" />
  <meta name="theme-color" content="#ad1457" />
  <!-- end Manubot generated metadata -->
  <!--[if lt IE 9]>
    <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.7.3/html5shiv-printshiv.min.js"></script>
  <![endif]-->
</head>
<body>
<header id="title-block-header">
<h1 class="title">A Practical Beginner’s Guide to Proteomics</h1>
</header>
<p><small><em>
This manuscript
(<a href="https://jessegmeyerlab.github.io/proteomics-tutorial/v/1527fe26c63f130aba0979d41345a20680c6e77a/">permalink</a>)
was automatically generated
from <a href="https://github.com/jessegmeyerlab/proteomics-tutorial/tree/1527fe26c63f130aba0979d41345a20680c6e77a">jessegmeyerlab/proteomics-tutorial@1527fe2</a>
on July 12, 2022.
</em></small></p>
<h2 id="authors">Authors</h2>
<ul>
<li><p><strong>Jesse G. Meyer</strong><br>
<img src="images/orcid.svg" class="inline_icon" width="16" height="16" alt="ORCID icon" />
<a href="https://orcid.org/0000-0003-2753-3926">0000-0003-2753-3926</a>
· <img src="images/github.svg" class="inline_icon" width="16" height="16" alt="GitHub icon" />
<a href="https://github.com/jessegmeyerlab">jessegmeyerlab</a>
· <img src="images/twitter.svg" class="inline_icon" width="16" height="16" alt="Twitter icon" />
<a href="https://twitter.com/j_my_sci">j_my_sci</a><br>
<small>
Department of Computational Biomedicine, Cedars Sinai Medical Center
· Funded by Grant R21 AG074234; Grant R35 GM142502
</small></p></li>
<li><p><strong>Dina Schuster</strong><br>
<img src="images/orcid.svg" class="inline_icon" width="16" height="16" alt="ORCID icon" />
<a href="https://orcid.org/0000-0001-6611-8237">0000-0001-6611-8237</a>
· <img src="images/github.svg" class="inline_icon" width="16" height="16" alt="GitHub icon" />
<a href="https://github.com/dschust-r">dschust-r</a>
· <img src="images/twitter.svg" class="inline_icon" width="16" height="16" alt="Twitter icon" />
<a href="https://twitter.com/dina_sch">dina_sch</a><br>
<small>
Department of Biology, Institute of Molecular Systems Biology, ETH Zurich, Zurich 8093, Switzerland; Department of Biology, Institute of Molecular Biology and Biophysics, ETH Zurich, Zurich 8093, Switzerland; Laboratory of Biomolecular Research, Division of Biology and Chemistry, Paul Scherrer Institute, Villigen 5232, Switzerland
</small></p></li>
<li><p><strong>Amit Kumar Yadav</strong><br>
<img src="images/orcid.svg" class="inline_icon" width="16" height="16" alt="ORCID icon" />
<a href="https://orcid.org/0000-0002-9445-8156">0000-0002-9445-8156</a>
· <img src="images/github.svg" class="inline_icon" width="16" height="16" alt="GitHub icon" />
<a href="https://github.com/aky">aky</a>
· <img src="images/twitter.svg" class="inline_icon" width="16" height="16" alt="Twitter icon" />
<a href="https://twitter.com/theoneamit">theoneamit</a><br>
<small>
Translational Health Science and Technology Institute
· Funded by Grant BT/PR16456/BID/7/624/2016 (Department of Biotechnology, India); Grant Translational Research Program (TRP) at THSTI funded by DBT
</small></p></li>
<li><p><strong>Muralidharan Vanuopadath</strong><br>
<img src="images/orcid.svg" class="inline_icon" width="16" height="16" alt="ORCID icon" />
<a href="https://orcid.org/0000-0002-9364-917X">0000-0002-9364-917X</a>
· <img src="images/github.svg" class="inline_icon" width="16" height="16" alt="GitHub icon" />
<a href="https://github.com/vanuopadathmurali">vanuopadathmurali</a>
· <img src="images/twitter.svg" class="inline_icon" width="16" height="16" alt="Twitter icon" />
<a href="https://twitter.com/V_MuraleeDhar">V_MuraleeDhar</a><br>
<small>
School of Biotechnology, Amrita Vishwa Vidyapeetham, Kollam-690 525, Kerala, India
</small></p></li>
<li><p><strong>Devasahayam Arokia Balaya Rex</strong><br>
<img src="images/orcid.svg" class="inline_icon" width="16" height="16" alt="ORCID icon" />
<a href="https://orcid.org/0000-0002-9556-3150">0000-0002-9556-3150</a>
· <img src="images/github.svg" class="inline_icon" width="16" height="16" alt="GitHub icon" />
<a href="https://github.com/ArokiaRex">ArokiaRex</a>
· <img src="images/twitter.svg" class="inline_icon" width="16" height="16" alt="Twitter icon" />
<a href="https://twitter.com/rexprem">rexprem</a><br>
<small>
Center for Systems Biology and Molecular Medicine, Yenepoya Research Centre, Yenepoya (Deemed to be University), Mangalore 575018, India
</small></p></li>
<li><p><strong>Benjamin A. Neely</strong><br>
<img src="images/orcid.svg" class="inline_icon" width="16" height="16" alt="ORCID icon" />
<a href="https://orcid.org/0000-0001-6120-7695">0000-0001-6120-7695</a>
· <img src="images/github.svg" class="inline_icon" width="16" height="16" alt="GitHub icon" />
<a href="https://github.com/neely">neely</a>
· <img src="images/twitter.svg" class="inline_icon" width="16" height="16" alt="Twitter icon" />
<a href="https://twitter.com/neely615">neely615</a><br>
<small>
Chemical Sciences Division, National Institute of Standards and Technology, NIST Charleston
· Funded by NIST
</small></p></li>
<li><p><strong>Martín L. Mayta</strong><br>
<img src="images/orcid.svg" class="inline_icon" width="16" height="16" alt="ORCID icon" />
<a href="https://orcid.org/0000-0002-7986-4551">0000-0002-7986-4551</a>
· <img src="images/github.svg" class="inline_icon" width="16" height="16" alt="GitHub icon" />
<a href="https://github.com/martinmayta">martinmayta</a>
· <img src="images/twitter.svg" class="inline_icon" width="16" height="16" alt="Twitter icon" />
<a href="https://twitter.com/MartinMayta2">MartinMayta2</a><br>
<small>
School of Medicine and Health Sciences, Center for Health Sciences Research, Universidad Adventista del Plata, Libertador San Martín 3103, Argentina; Molecular Biology Department, School of Pharmacy and Biochemistry, Universidad Nacional de Rosario, Rosario 2000, Argentina
</small></p></li>
<li><p><strong>Emma H. Doud</strong><br>
<img src="images/orcid.svg" class="inline_icon" width="16" height="16" alt="ORCID icon" />
<a href="https://orcid.org/0000-0003-0049-0073">0000-0003-0049-0073</a>
· <img src="images/github.svg" class="inline_icon" width="16" height="16" alt="GitHub icon" />
<a href="https://github.com/edoud1">edoud1</a>
· <img src="images/twitter.svg" class="inline_icon" width="16" height="16" alt="Twitter icon" />
<a href="https://twitter.com/fireinlab">fireinlab</a><br>
<small>
Center for Proteome Analysis, Indiana University School of Medicine, Indianapolis, Indiana, USA
</small></p></li>
</ul>
<h2 class="page_break_before" id="abstract">Abstract</h2>
<p>Proteomics is the large scale study of protein structure and function from biological systems.<br />
“Shotgun proteomics” or “bottom-up proteomics” is the prevailing strategy, in which proteins are hydrolyzed into peptide that are analyzed by mass spectrometry.
Proteomics studies can be applied to diverse studies ranging from simple protein identification to studies of protein-protein interactions, absolute and relative protein quantification, post-translational modifications, and protein stability.
To enable this range of different experiments, there are diverse strategies for proteome analysis.
The nuances of how proteomic workflows differ may be difficult to understand for new practitioners.
Here, we provide a comprehensive tutorial of different proteomics methods.
Our tutorial covers all necessary steps starting from protein extraction and ending with biological interpretation.
We expect that this work will serve as a basic resource for new practitioners of the field of shotgun or bottom-up proteomics.</p>
<h2 class="page_break_before" id="introduction">Introduction</h2>
<p>Proteomics is the large scale study of protein structure and function.
Proteins are translated from mRNAs that are transcribed from the genome.
Although the genome encodes potential cellular functions and states, the study of proteins is necessary to truly understand biology.
Currently, proteomic studies are facilitated by mass spectrometry, although alternative methods are being developed.</p>
<p>Modern proteomics started around the year 1990 with the introduction of soft ionization methods that enabled, for the first time, transfer of large biomolecules into the gas phase without destroying them <span class="citation" data-cites="4Ajjx5fo 15Ye4Za7Y">[<a href="#ref-4Ajjx5fo" role="doc-biblioref">1</a>,<a href="#ref-15Ye4Za7Y" role="doc-biblioref">2</a>]</span>.
Shortly afterward, the first computer algorithm for matching peptides to a database was introduced <span class="citation" data-cites="pG7zq91Q">[<a href="#ref-pG7zq91Q" role="doc-biblioref">3</a>]</span>.
Another major milestone that allowed identification of over 1000 proteins were actually improvements to chromatography <span class="citation" data-cites="ujahSo3G">[<a href="#ref-ujahSo3G" role="doc-biblioref">4</a>]</span>.
As the volume of data exploded, methods for statistical analysis transitioned use from the wild west to modern informatics based on statistical models <span class="citation" data-cites="mw5jxASm">[<a href="#ref-mw5jxASm" role="doc-biblioref">5</a>]</span> and the false discovery rate <span class="citation" data-cites="y9Bki24q">[<a href="#ref-y9Bki24q" role="doc-biblioref">6</a>]</span>.
<!-- Todo: figure 1: major milestones in proteomics technology since 1990 --></p>
<p>Two strategies of mass spectrometry-based proteomics differ fundamentally by whether proteins are cleaved into peptides before analysis: “top-down” and “bottom-up”.
Bottom-up proteomics (also refered to as shotgun proteomics) is defined by the hydrolysis of proteins into peptide pieces <span class="citation" data-cites="16QSCuG38">[<a href="#ref-16QSCuG38" role="doc-biblioref">7</a>]</span>.
Therefore, bottom-up proteomics does not actually measure proteins, but must infer their presence <span class="citation" data-cites="mw5jxASm">[<a href="#ref-mw5jxASm" role="doc-biblioref">5</a>]</span>.
Sometimes proteins are infered from only one peptide sequence representing a small fraction of the total protein sequence predicted from the genome.
In contrast, top-down proteomics attempts to measure all proteins intact <span class="citation" data-cites="a6Z8FNxW">[<a href="#ref-a6Z8FNxW" role="doc-biblioref">8</a>]</span>.
The potential benefit of top-down proteomics is the ability to measure proteoforms <span class="citation" data-cites="h00Trxkt">[<a href="#ref-h00Trxkt" role="doc-biblioref">9</a>]</span>.
However, due to analytical challenges, the depth of protein coverage that is achievable by top-down proteomics is less than the depth that is achievable by bottom-up proteomics.</p>
<p>In this tutorial we focus on the bottom-up proteomics workflow.
The most common version of this workflow is generally comprised of the following steps.
First, proteins in a biological sample must be extracted.
Usually this is done by denaturing and solubilizing the proteins while disrupting DNA and tissue.
Next, proteins are hydrolyzed into peptides, usually using a protease like trypsin.
Peptides from proteome hydrolysis must be purified.
Most often this is done with reversed phase chromatography cartridges or tips.
The peptides are then almost always separated by liquid chromatography before they are ionized and introduced into a mass spectrometer.
The mass spectrometer then collects precursor and fragment ion data from those peptides.
The data analysis is usually the rate limiting step.
Peptides must be identified, and proteins are inferred and quantities are assigned.
Changes in proteins across conditions are determined with statistical tests, and results must be interpreted in the context of the relevant biology.</p>
<p>There are many variations on this workflow. The wide variety of experimental goals that are achievable with proteomics technology leads to a wide variety of potential proteomics workflows. Even choice is important and every choice will affect the results. In this tutorial, we cover all of the required steps in detail to serve as a tutorial for new proteomics practioners.</p>
<ol type="1">
<li>Types of experiments enabled by proteomics</li>
<li>Protein extraction</li>
<li>proteolysis</li>
<li>Isotopic Labeling</li>
<li>Enrichments</li>
<li>Peptide purification</li>
<li>Mass Spectrometry</li>
<li>Peptide Ionization</li>
<li>Data Acquisition</li>
<li>Basic Data Analysis</li>
<li>Biological Interpretation</li>
<li>Experimental considerations and design</li>
</ol>
<h2 class="page_break_before" id="types-of-experiments">Types of Experiments</h2>
<p>A wide range of questions are addressable with proteomics technology, which translates to a wide range of variations of proteomics workflows.
Sometimes identifying what proteins are present is desired, and sometimes the quantities of as many proteins as possible are desired.
Proteomics experiments can be both qualitative and quantitative.</p>
<h3 id="qualitative-experiments">Qualitative experiments</h3>
<ul>
<li>Identifying proteins</li>
<li>Identifying post translational modifications</li>
<li>Identifying protein isoforms</li>
</ul>
<h3 id="quantitative-experiments">Quantitative experiments</h3>
<ul>
<li>Protein abundance changes</li>
<li>Phosphoproteomics</li>
<li>Glycoproteomics</li>
<li>Structural techniques (XL-MS, HDX-MS, FPOP, protein-painting, LiP-MS, radical footprinting, ion mobility)</li>
<li>Protein stability and small molecule binding (Thermal proteome profiling, TPP, or cellular thermal shift assay, CETSA)</li>
<li>Protein-protein interactions (PPIs): AP-MS, APEX, BioID</li>
</ul>
<h2 class="page_break_before" id="protein-extraction">Protein Extraction</h2>
<p>Protein extraction from the sample of interest is the initial phase of any mass spectrometry-based proteomics experiment.
Thought should be given to any planned downstream assays, specific needs of proteolysis (LiP-MS, post translational modification enrichments, enzymatic reactions, glycan purification or hydrogen-deuterium exchange experiments) long term project goals (reproducibility, multiple sample types, low abundance samples) as well as to the initial experimental question (coverage of a specific protein, subcellular proteomics, global proteomics, protein-protein interactions or immune or affinity enrichment of a specific classes of modifications.)
The 2009 version of Methods in Enzymology: guide to Protein Purification <span class="citation" data-cites="emz5eXnO">[<a href="#ref-emz5eXnO" role="doc-biblioref">10</a>]</span> serves as a deep dive into how molecular biologists and biochemists traditionally thought about protein extraction.
The Protein Protocols handbook <span class="citation" data-cites="LvSVXQeu">[<a href="#ref-LvSVXQeu" role="doc-biblioref">11</a>]</span> and the excellent review by Linn <span class="citation" data-cites="YFivV9n5">[<a href="#ref-YFivV9n5" role="doc-biblioref">12</a>]</span> are good sources of general proteomics protocols for a scientist new to the field.
Any change in extraction conditions should be expected to create potential changes in downstream results.
Be sure to think about and optimize the protein extraction step first and stick with a protocol that works for your needs.
If a collaborator is attempting to reproduce your results, make sure they begin with the same extraction protocols.</p>
<h3 id="buffer-choice">Buffer choice</h3>
<h4 id="general-proteomics">General proteomics</h4>
<p>A common question to proteomics core facilities is, “What is the best buffer for protein extraction?”
Unfortunately, there is no one correct answer.
For global proteomics experiments where maximizing the number of protein or peptide identifications is a goal, a buffer of neutral pH (50-100 mM PBS, Tris, HEPES, Ammonium Bicarbonate, triethanolamine bicarbonate; pH 7.5-8.5) is used in conjunction with a chaotrope or surfactant to denature and solubilize proteins (e.g., 8 M urea, 6 M guanidine, 5% SDS) <span class="citation" data-cites="OZvOVLht bpTXhTR5">[<a href="#ref-OZvOVLht" role="doc-biblioref">13</a>,<a href="#ref-bpTXhTR5" role="doc-biblioref">14</a>]</span>.
Often other salts like 50-150 mM NaCl are also added.
Complete denaturation of the proteins in the sample in a timely fashion is an advantage as it generally prevents changes to protein status by endogenous proteases, kinases, phosphotases, and other enzymes.
If intact protein separations are planned (based on size or isoelectric point) choose a denaturant compatible with those methods, such as SDS<span class="citation" data-cites="Igj2yRI0">[<a href="#ref-Igj2yRI0" role="doc-biblioref">15</a>]</span>.
Compatibility with protease (typically trypsin) and peptide cleanup steps will need to be considered.
8 M urea must be diluted to 2 M or less for trypsin and chymotrypsin digestions, while guanidine and SDS should be removed either through protein precipitation, through filter-assisted sample preparation (FASP), or similar solid phase digestion techniques.
Note that some buffers can potentially introduce modifications onto proteins such as carbamylation from urea at high temperatures <span class="citation" data-cites="B8hLhFif">[<a href="#ref-B8hLhFif" role="doc-biblioref">16</a>]</span>.</p>
<h4 id="protein-protein-interactions">Protein-protein interactions</h4>
<p>Denaturing conditions will efficiently extract proteins – but they will denature/disrupt most protein-protein interactions.
If you are working on an immune- or affinity purification of a specific protein and expect to analyze enzymatic activity, structural features, and/or protein-protein interactions, a non-denaturing lysis buffer should be utilized <span class="citation" data-cites="G9HWYdpG 15gFWnWy1">[<a href="#ref-G9HWYdpG" role="doc-biblioref">17</a>,<a href="#ref-15gFWnWy1" role="doc-biblioref">18</a>]</span>.
Check the calculated pI and hydrophobicity (the Expasy.org resource ProtParam is useful fo this) for a good idea of starting pH/conductivity, but you may need to perform a stability screen.
In general, a good starting point for the buffer will still be close to neutral pH with 50-250 mM NaCl, but specific proteins may require pH as low as 2 or as high as 9 for stable extraction.
A low percent of mass spec compatible detergent may also be used.
Newer mass spectrometry compatible detergents are also useful for protein extraction and ease of downstream processing – including Rapigest® (Waters), N-octyl-β-glucopyranoside, Azo <span class="citation" data-cites="1FdkzkZPe">[<a href="#ref-1FdkzkZPe" role="doc-biblioref">19</a>]</span>, PPS silent surfactant <span class="citation" data-cites="KbqdsNwY">[<a href="#ref-KbqdsNwY" role="doc-biblioref">20</a>]</span>, sodium laurate <span class="citation" data-cites="1FX8gYsTy">[<a href="#ref-1FX8gYsTy" role="doc-biblioref">21</a>]</span>, and sodium deoxycholate<span class="citation" data-cites="7nXVfPN7">[<a href="#ref-7nXVfPN7" role="doc-biblioref">22</a>]</span>.
AVOID the use of tween-20, triton-X, NP-40, and PEGs as these compounds are challenging to remove after digestion <span class="citation" data-cites="EZCh7C0x">[<a href="#ref-EZCh7C0x" role="doc-biblioref">23</a>]</span>.</p>
<h4 id="optional-additives">Optional additives</h4>
<p>For non-denaturing buffer conditions, which preserve tertiary and quaternary protein structures, additional additives may not be necessary for successful extraction and to prevent proteolysis or PTM modifications throughout the extraction process.
Protease, phosphatase and deubiquitinase inhibitors are optional additives in less denaturing conditions or in experiments focused on specific post-translational modifications.
Keep in mind that protease inhibitors may impact digestion conditions and will need to be diluted or removed prior to trypsin addition.
For extraction of DNA or RNA binding proteins, addition of a small amount of nuclease or benzonase might be useful for degradation of any bound nucleic acids and result in a more consistent digestion <span class="citation" data-cites="EK9zCZWn">[<a href="#ref-EK9zCZWn" role="doc-biblioref">24</a>]</span>.</p>
<h3 id="mechanical-or-sonic-disruption">Mechanical or Sonic Disruption</h3>
<h4 id="cell-lysis">Cell lysis</h4>
<p>One typical lysis buffer is 8 M urea in 100 mM Tris, pH 8.5; the pH based on optimum trypsin activity <span class="citation" data-cites="Cj5ZGyu8">[<a href="#ref-Cj5ZGyu8" role="doc-biblioref">25</a>]</span>
Small mammalian cell pellets and exosomes will lyse almost instantly upon addition denaturing buffer.
Efficiency of extraction and degradation of nucleic acids can be improved using various sonication methods: 1) probe sonicator with ice; 2) water bath sonicator with ice or cooling; 3) bioruptor® sonication device 4) Adaptive focused acoustics (AFA®) <span class="citation" data-cites="195QiKJpv">[<a href="#ref-195QiKJpv" role="doc-biblioref">26</a>]</span>.
Key to these additional lysis techniques are to keep the temperature of the sample from rising significantly which can cause proteins to aggregate or degrade.
Some cell types may require additional force for effective lysis (see below).
For bacteria with cell wells, lysozyme is often added in the lysis buffer.
Any added protein will be present in downstream results, however, so excessive addition of lysozyme is to be avoided unless tagged protein purification will occur.</p>
<h4 id="tissueother-lysis">Tissue/other lysis</h4>
<p>Although small pieces of soft tissue can often be successfully extracted with the probe and sonication methods described above, larger/harder tissues as well as plants/yeast/fungi are better extracted with some form of additional mechanical force.
If proteins are too be extracted from a large amount of sample, such as soil, feces, or other diffuse input, one option is to use a dedicated blender and filter the sample after followed by centrifugation.
If samples are smaller, such as tissue, tumors, etc., cryo-homogenization is recommended.
The simplest form of this is grinding the sample with liquid nitrogen and a mortar and pestle.
Tools such as bead beaters are also used, where the sample is placed in a tube with appropriately sized beads and shaken rapidly.
Cryo-mills are chambers where liquid nitrogen is applied around a vessel and large bead or beads.
Cryo-fractionators homogenize samples in special bags that are frozen in liquid nitrogen and smashed with various degrees of force <span class="citation" data-cites="4EGQHHdM">[<a href="#ref-4EGQHHdM" role="doc-biblioref">27</a>]</span>.
After homogenization, samples can be sonicated by one of the methods above to fragment DNA and increase solubilization of proteins.</p>
<h3 id="measuring-the-efficiency-of-protein-extraction">Measuring the efficiency of protein extraction</h3>
<p>Following protein extraction, samples should be centrifuged (10-14,000 g for 10-30 min depending on sample type) to remove debris prior to calculating protein concentration.
The amount of insoluble material remaining should be noted throughout an experiment as a large change may indicate protein extraction issues.
Protein concentration can be calculated using a number of assays or tools <span class="citation" data-cites="wXcRYG3x DRVtai1B">[<a href="#ref-wXcRYG3x" role="doc-biblioref">28</a>,<a href="#ref-DRVtai1B" role="doc-biblioref">29</a>]</span>; generally UV absorption methods are facile and affordable, such as Bradford or BCA assays.
Protein can also be estimated by tryptophan fluorescence, which has the benefit of not consuming sample <span class="citation" data-cites="7YHVbaDH">[<a href="#ref-7YHVbaDH" role="doc-biblioref">30</a>]</span>.
A nanodrop UV spectrophotometer may be used to measure absorbance at UV280.
Consistency in this method is important as each method will have inherent bias and error <span class="citation" data-cites="qFKCAO2X jQLWiiId">[<a href="#ref-qFKCAO2X" role="doc-biblioref">31</a>,<a href="#ref-jQLWiiId" role="doc-biblioref">32</a>]</span>.
Extraction buffer components will need to be compatible with any assay chosen; alternatively, buffer may be removed (see below) prior to protein concentration calculation.</p>
<h3 id="reduction-and-alkylation">Reduction and alkylation</h3>
<p>Typically, disulfide bonds in proteins are reduced and alkylated prior to proteolysis in order to disrupt structures and simplify peptide analysis.
This allows better access to all residues during proteolysis and removes the crosslinked peptides created by S-S inter peptide linkages.
There are a variety of reagent options for these steps.
For reduction, the typical agents used are 5-15 mM concentration of TCEP/DTT/2BME.
For the following alkylation step, a slightly higher 10-20mM concentration, alkylating agent such as Chloroacetamide/iodoacetamide, n-ethyl maleimide can be used [PMID:29019370; <span class="citation" data-cites="2cCnZ6ft">[<a href="#ref-2cCnZ6ft" role="doc-biblioref">33</a>]</span>; <span class="citation" data-cites="9vICnnYM">[<a href="#ref-9vICnnYM" role="doc-biblioref">34</a>]</span>].
It is also possible to alklylate free cysteines with one reagent, reduce di-sulfide bonds (or other Cysteine modifications) and alkylate with a different reagent in order to monitor which residues are linked/modified in a protein <span class="citation" data-cites="GkFAexwC Z6nk0TFI s4OLOlUG">[<a href="#ref-GkFAexwC" role="doc-biblioref">35</a>,<a href="#ref-Z6nk0TFI" role="doc-biblioref">36</a>,<a href="#ref-s4OLOlUG" role="doc-biblioref">37</a>]</span>.
Reduction reactions are generally carried out in the dark at room temperature to avoid excessive off target alklation.</p>
<h3 id="removal-of-bufferinterfering-small-molecules">Removal of buffer/interfering small molecules</h3>
<p>If extraction must take place in a buffer which is incompatible for efficient proteolysis (check the guidelines for the protease of choice), then protein cleanup should occur prior to digestion.
This is generally performed through precipitation of proteins.
The most common types are 1) acetone, 2) trichloroacetic acid (TCA), and 3) methanol/chloroform/water <span class="citation" data-cites="xvenBhjQ OMzPHtgd">[<a href="#ref-xvenBhjQ" role="doc-biblioref">38</a>,<a href="#ref-OMzPHtgd" role="doc-biblioref">39</a>]</span>.
Proteins are generally insoluble in most pure organic solvents, so cold ethanol or methanol are sometimes used.
Pellets should be washed with organic solvent for complete removal especially of detergents.
Alternatively, solid phase based digestion methods such as S-trap <span class="citation" data-cites="Llj72YHx">[<a href="#ref-Llj72YHx" role="doc-biblioref">40</a>]</span>, FASP <span class="citation" data-cites="rWTbDNu2 bkON7ZR2">[<a href="#ref-rWTbDNu2" role="doc-biblioref">41</a>,<a href="#ref-bkON7ZR2" role="doc-biblioref">42</a>]</span>, SP3 <span class="citation" data-cites="29YpOcx3 R9HFU5Rw">[<a href="#ref-29YpOcx3" role="doc-biblioref">43</a>,<a href="#ref-R9HFU5Rw" role="doc-biblioref">44</a>]</span> and on column/bead can allow for proteins to be applied to a solid phase and buffers removed prior to proteolysis <span class="citation" data-cites="PMID">[<a href="#ref-PMID" role="doc-biblioref"><strong>PMID?</strong></a> 29754492]</span>.
Specialty detergent removal columns exist (Pierce/Thermo Fisher Scientific) but add expense and time consuming steps to the process.
Additionally these should be checked for efficiency prior to implementing in a workflow with many samples as avoiding detergent contamination in the LC/MS is very important.</p>
<h3 id="summary">Summary</h3>
<p>Often you will be given protein extraction conditions from molecular biologists or biochemistry which you will have to make work with downstream mass spectrometry applications.
For bottom-up proteomics, the overreaching goal is efficient and consistent extraction and digestion.</p>
<h2 class="page_break_before" id="proteolysis">Proteolysis</h2>
<p>Proteolysis is the defining step that differentiates bottom-up or shotgun proteomics from top-down proteomics.
Hydrolysis of proteins is extremely important because it defines the population of potentially identifyable peptides.
Generally peptides between a length of 7-35 amino acids are considered useful for mass spectrometry analysis.
Peptides that are too long are difficult to identify by tandem mass spectrometry, or may be lost during sample preparation due to irreversible binding with solid-phase extraction sorbents.
Peptides that are too short are also not useful because they may match to many proteins during protein inference.
There are many choices of enzymes and chemicals that hydrolyze proteins into peptides.
This section summarizes potential choices and their strengths and weaknesses.</p>
<p>Trypsin is the most common choice of protease for proteome hydrolysis <span class="citation" data-cites="vpBg0CP7">[<a href="#ref-vpBg0CP7" role="doc-biblioref">45</a>]</span>.
Trypsin is favorable because of its specificity, availability, efficiency and low cost.
Trypsin cleaves at the C-terminus of basic amino acids, Arg and Lys.
Many of the peptides generated from trypsin are short in length (less than ~ 20 amino acids), which is ideal for chromatographic separation, MS-based peptide fragmentation and identification by database search.
The main drawback of trypsin is that majority (56%) of the tryptic peptides are ≤ 6 amino acids, and hence using trypsin alone limits the observable proteome <span class="citation" data-cites="16FpKYZCe lSFMVWD1 10vHwy5gN">[<a href="#ref-16FpKYZCe" role="doc-biblioref">46</a>,<a href="#ref-lSFMVWD1" role="doc-biblioref">47</a>,<a href="#ref-10vHwy5gN" role="doc-biblioref">48</a>]</span>.
This limits the number of identifiable protein isoforms and post-translational modifications.</p>
<ol start="3" type="1">
<li>theoretical studies of proteolysis enzymes <span class="citation" data-cites="y9KaXde4">[<a href="#ref-y9KaXde4" role="doc-biblioref">49</a>]</span></li>
<li>Challenges associated with alternative enzyme choices (non-specific and semi-specific enzymes)</li>
</ol>
<p>Many alternative proteases are available with different specificities that complement trypsin to reveal different proteomic sequences <span class="citation" data-cites="7aKTZT1i 16FpKYZCe">[<a href="#ref-16FpKYZCe" role="doc-biblioref">46</a>,<a href="#ref-7aKTZT1i" role="doc-biblioref">50</a>]</span>, which can help distinguish protein isoforms <span class="citation" data-cites="gXhdeFOF">[<a href="#ref-gXhdeFOF" role="doc-biblioref">51</a>]</span>.
The enzyme choice mostly depends on the application.
In general, for a mere protein identification mostly trypsin is the choice due to the reasons aforementioned.
However, alternative enzymes can facilitate <em>de novo</em> assembly when the genomic data information is limited in the public database repositories <span class="citation" data-cites="14bc70ELO zmo81rNN IZw9sVjg 19KCAADV7 7BuX2TBR">[<a href="#ref-14bc70ELO" role="doc-biblioref">52</a>,<a href="#ref-zmo81rNN" role="doc-biblioref">53</a>,<a href="#ref-IZw9sVjg" role="doc-biblioref">54</a>,<a href="#ref-19KCAADV7" role="doc-biblioref">55</a>,<a href="#ref-7BuX2TBR" role="doc-biblioref">56</a>]</span>.
Use of multiple proteases for proteome digestion also can improve the sensitivity and accuracy of protein quantification <span class="citation" data-cites="1GLf6FYas">[<a href="#ref-1GLf6FYas" role="doc-biblioref">57</a>]</span>.
Moreover, by providing an increased peptide diversity, the use of multiple proteases can expand sequence coverage and increase the probability of finding peptides which are unique to single proteins <span class="citation" data-cites="WubQ7yFP yeQvImXS y9KaXde4">[<a href="#ref-y9KaXde4" role="doc-biblioref">49</a>,<a href="#ref-WubQ7yFP" role="doc-biblioref">58</a>,<a href="#ref-yeQvImXS" role="doc-biblioref">59</a>]</span>.
A multi-protease approach can also improve the identification of N-Termini and signal peptides for small proteins <span class="citation" data-cites="Ac0S3PrF">[<a href="#ref-Ac0S3PrF" role="doc-biblioref">60</a>]</span>.
Overall, integrating multiple-protease data can increase the number of proteins identified <span class="citation" data-cites="P861T4mW 14eCb7Jfc">[<a href="#ref-P861T4mW" role="doc-biblioref">61</a>,<a href="#ref-14eCb7Jfc" role="doc-biblioref">62</a>]</span>, the number of identified post-translational modifications detected <span class="citation" data-cites="WubQ7yFP 18s9UXUGf yeQvImXS">[<a href="#ref-WubQ7yFP" role="doc-biblioref">58</a>,<a href="#ref-yeQvImXS" role="doc-biblioref">59</a>,<a href="#ref-18s9UXUGf" role="doc-biblioref">63</a>]</span> and decrease the ambiguity of the protein group list <span class="citation" data-cites="WubQ7yFP">[<a href="#ref-WubQ7yFP" role="doc-biblioref">58</a>]</span>.</p>
<p>Lysyl endopeptidase (Lys-C) obtained from <em>Lysobacter enzymogenesis</em> is a serine protease involved in cleaving carboxyl terminus of Lys <span class="citation" data-cites="ZgqlEERS">[<a href="#ref-lSFMVWD1" role="doc-biblioref">47</a>]</span>.
Like trypsin, the optimum pH range required for its activity is from 7 to 9.
A major advantage of Lys-C is its resistance to denaturing agents, including 8 M urea - a chaotrope commonly used to denature proteins <em>prior</em> to digestion <span class="citation" data-cites="gXhdeFOF">[<a href="#ref-gXhdeFOF" role="doc-biblioref">51</a>]</span>.
Trypsin is less efficient at cleaving Lys than Arg, which could limit the quality of quantitation from tryptic peptides.
Hence, to achieve complete protein digestion with minimal missed cleavages, Lys-C is often used simultaneously with trypsin digestion <span class="citation" data-cites="nobcST81">[<a href="#ref-nobcST81" role="doc-biblioref">65</a>]</span>.</p>
<p>Alpha-lytic protease (aLP) is also secreted by the soil bacterial <em>Lysobacter enzymogenesis</em> <span class="citation" data-cites="oDNy6K9V">[<a href="#ref-oDNy6K9V" role="doc-biblioref">66</a>]</span>.
Wild-type aLP (WaLP) and an active site mutant of aLP, M190A (MaLP), have been used to expand proteome coverage <span class="citation" data-cites="yeQvImXS">[<a href="#ref-yeQvImXS" role="doc-biblioref">59</a>]</span>.
Based on observed peptide sequences from yeast proteome digestion, WaLP showed a specificity for small aliphatic amino acids like alanine, valine, and glycine, but also threonine and serine.
MaLP showed specificity for slightly larger amino acids like methionine, phenylalanine, and surprisingly, a preference for leucine over isoleucine.
The specificity of WaLP for threonine enabled the first method for mapping endogenous human SUMO sites <span class="citation" data-cites="4iAess79">[<a href="#ref-4iAess79" role="doc-biblioref">67</a>]</span>.</p>
<p>Glutamyl peptidase I, commonly known as Glu-C or V8 protease, is a serine protease obtained from <em>Staphyloccous aureus</em> <span class="citation" data-cites="UeP5oN6q">[<a href="#ref-UeP5oN6q" role="doc-biblioref">68</a>]</span>.
Glu-C cleaves at the C-terminus of glutamate, but also after aspartate <span class="citation" data-cites="UeP5oN6q FcovvAwu">[<a href="#ref-UeP5oN6q" role="doc-biblioref">68</a>,<a href="#ref-FcovvAwu" role="doc-biblioref">69</a>]</span>.</p>
<p>Peptidyl-Asp metallopeptidase, commonly known as Asp-N, is a metalloprotease obtained from <em>Pseudomonas fragi</em> <span class="citation" data-cites="Z8khRFem">[<a href="#ref-Z8khRFem" role="doc-biblioref">70</a>]</span>.
Asp-N catalyzes the hydrolysis of peptide bonds at the N-terminal of aspartate residues.
The optimum activity of this enzyme occurs at a pH range between 4 and 9.<br />
As with any metalloprotease, chelators like EDTA should be avoided for digestion buffers when using Asp-N.
Studies also suggest that Asp-N cleaves at the amino terminus of glutamate when a detergent is present in the proteolysis buffer <span class="citation" data-cites="Z8khRFem">[<a href="#ref-Z8khRFem" role="doc-biblioref">70</a>]</span>.
Asp-N often leaves many missed cleavages <span class="citation" data-cites="gXhdeFOF">[<a href="#ref-gXhdeFOF" role="doc-biblioref">51</a>]</span>.</p>
<p>Chymotrypsin or chymotrypsinogen A is a serine protease obtained from porcine or bovine pancreas with an optimum pH range from 7.8 to 8.0 <span class="citation" data-cites="129jHkJ7J">[<a href="#ref-129jHkJ7J" role="doc-biblioref">71</a>]</span>.
It cleaves at the C-terminus of hydrphobic amino acids Phe, Trp, Tyr and barely Met and Leu residues.
Since the transmembrane region of membrane proteins commonly lacks tryptic cleavage sites, this enzyme works well with membrane proteins having more hydrophobic residues <span class="citation" data-cites="n8tCUNeh Ddlp2JQj gXhdeFOF">[<a href="#ref-gXhdeFOF" role="doc-biblioref">51</a>,<a href="#ref-n8tCUNeh" role="doc-biblioref">72</a>,<a href="#ref-Ddlp2JQj" role="doc-biblioref">73</a>]</span>.
The chymotryptic peptides generated after proteolysis will cover the proteome space orthogonal to that of tryptic peptides both in a quantitative and qualitative manner <span class="citation" data-cites="VbXzZ784 1BDWCCTHa Ddlp2JQj">[<a href="#ref-Ddlp2JQj" role="doc-biblioref">73</a>,<a href="#ref-VbXzZ784" role="doc-biblioref">74</a>,<a href="#ref-1BDWCCTHa" role="doc-biblioref">75</a>]</span></p>
<p>Clostripain, commonly known as Arg-C, is a cysteine protease obtained from <em>Clostridium histolyticum</em> <span class="citation" data-cites="qJADzs4">[<a href="#ref-qJADzs4" role="doc-biblioref">76</a>]</span>.
It hydrolyses mostly the C-terminal Arg residues and sometimes Lys residues, but with less efficiency.
The peptides generated are generally longer than that of tryptic peptides.
Arg-C is often used with other proteases for improving qualitative proteome data and also for investigating PTMs <span class="citation" data-cites="lSFMVWD1">[<a href="#ref-lSFMVWD1" role="doc-biblioref">47</a>]</span>.</p>
<p>LysargiNase, also known as Ulilysin, is a recently discovered protease belonging to the metalloprotease family.
It is a thermophilic protease derived from <em>Methanosarcina acetivorans</em> that specifically cleaves at the N-terminus of Lys and Arg residues <span class="citation" data-cites="MqKkSQNK">[<a href="#ref-MqKkSQNK" role="doc-biblioref">77</a>]</span>.
Hence, it enabled discovery of C-terminal peptides that were not observed using trypsin.
In addition, it can also cleave modified amino acids such as methylated or dimethylated Arg and Lys <span class="citation" data-cites="MqKkSQNK">[<a href="#ref-MqKkSQNK" role="doc-biblioref">77</a>]</span>.</p>
<p>Peptidyl-Lys metalloendopeptidase, or Lys-N, is an metalloprotease obtained from <em>Grifola frondosa</em> <span class="citation" data-cites="3xdcLBaR">[<a href="#ref-3xdcLBaR" role="doc-biblioref">78</a>]</span>.
It cleaves N-terminally of Lys and has an optimal activity at pH 9.0.
Unlike trypsin, Lys-N is more resistant to denaturing agents and can be heated up to 70 °C <span class="citation" data-cites="lSFMVWD1">[<a href="#ref-lSFMVWD1" role="doc-biblioref">47</a>]</span>.
Reports suggest that the peptides generated after Lys-N digestion produces more of c-type ions in a ETD-based mass spectrometer <span class="citation" data-cites="O6S1GvPn">[<a href="#ref-O6S1GvPn" role="doc-biblioref">79</a>]</span>.
Hence this can be used for analysing PTMs, identification of C-terminal peptides and also for <em>de novo</em> sequencing strategies <span class="citation" data-cites="O6S1GvPn 8DGqkN3D">[<a href="#ref-O6S1GvPn" role="doc-biblioref">79</a>,<a href="#ref-8DGqkN3D" role="doc-biblioref">80</a>]</span>.</p>
<p>Pepsin A, commonly known as pepsin, is an aspartic protease obtained from bovine or porcine pancreas <span class="citation" data-cites="eMC9mqfj">[<a href="#ref-eMC9mqfj" role="doc-biblioref">81</a>]</span>.
Pepsin was one of several proteins crystalized by John Northrop, who shared the 1946 Nobel prize in chemistry for this work <span class="citation" data-cites="pnU7apE 9Yf3fEMC 10CHY6r2E 1EtaxGZW0">[<a href="#ref-pnU7apE" role="doc-biblioref">82</a>,<a href="#ref-9Yf3fEMC" role="doc-biblioref">83</a>,<a href="#ref-10CHY6r2E" role="doc-biblioref">84</a>,<a href="#ref-1EtaxGZW0" role="doc-biblioref">85</a>/]</span>.
Pepsin works at an optimum pH range from 1 to 4 and specifically cleaves Trp, Phe, Tyr and Leu <span class="citation" data-cites="lSFMVWD1">[<a href="#ref-lSFMVWD1" role="doc-biblioref">47</a>]</span>.
Since it possess high enzyme activity and broad specificity at lower pH, it is preferred over other proteases for MS-based disulphide mapping <span class="citation" data-cites="itW7kK3s Vske1oHJ">[<a href="#ref-itW7kK3s" role="doc-biblioref">86</a>,<a href="#ref-Vske1oHJ" role="doc-biblioref">87</a>]</span>.
Pepsin is also used extensively for structural mass spectrometry studies with hydrogen-deuterium exchange (HDX) because the rate of back exchange of the amide deuteron is minimized at low pH <span class="citation" data-cites="Azi1X1R4 1nMCmYmb">[<a href="#ref-Azi1X1R4" role="doc-biblioref">88</a>,<a href="#ref-1nMCmYmb" role="doc-biblioref">89</a>]</span>.</p>
<p>Proteinase K was first isolated from the mold <em>Tritirachium album</em> Limber <span class="citation" data-cites="YOXX6An6">[<a href="#ref-YOXX6An6" role="doc-biblioref">90</a>]</span>.
The epithet ‘K’ is derived from its ability to efficiently hydrolyse keratin <span class="citation" data-cites="YOXX6An6">[<a href="#ref-YOXX6An6" role="doc-biblioref">90</a>]</span>.
It is a member of the subtilisin family of proteases and is relatively unspecific with a preference for proteolysis at hydrophobic and aromatic amino acid residues <span class="citation" data-cites="1CgjQBy2S">[<a href="#ref-1CgjQBy2S" role="doc-biblioref">91</a>]</span>.
The optimal enzyme activity is between pH 7.5 and 12.
Proteinase K is used at low concentrations for limited proteolysis (LiP) and the detection of protein structural changes in the eponymous technique LiP-MS <span class="citation" data-cites="1BhNuatKH">[<a href="#ref-1BhNuatKH" role="doc-biblioref">92</a>]</span>.</p>
<div id="fig:Multiple-protease-proteolysis" class="fignos">
<figure>
<img src="images/Proteolysis_v03.png" data-tag="1" style="width:100.0%" alt="Figure 1: Multiple protease proteolysis improves protein inference The use of other proteases beyond Trypsin such as Lysyl endopeptidase (Lys-C), Peptidyl-Asp metallopeptidase (Asp-N), Glutamyl peptidase I, (Glu-C), Chymotrypsin, Clostripain (Arg-C) or Peptidyl-Lys metalloendopeptidase (Lys-N) can generate a greater diversity of peptides. This improves protein sequence coverage and allows for the correct identification of their N-termini. Increasing the number of complimentary enzymes used will increase the number of proteins identified by single peptides and decreases the ambiguity of the assignment of protein groups. Therefore, this will allow more protein isoforms and post-translational modifications to be identified than using Trypsin alone." />
<figcaption aria-hidden="true"><span>Figure 1:</span> <strong>Multiple protease proteolysis improves protein inference</strong>
The use of other proteases beyond Trypsin such as Lysyl endopeptidase (Lys-C), Peptidyl-Asp metallopeptidase (Asp-N), Glutamyl peptidase I, (Glu-C), Chymotrypsin, Clostripain (Arg-C) or Peptidyl-Lys metalloendopeptidase (Lys-N) can generate a greater diversity of peptides.
This improves protein sequence coverage and allows for the correct identification of their N-termini.
Increasing the number of complimentary enzymes used will increase the number of proteins identified by single peptides and decreases the ambiguity of the assignment of protein groups.
Therefore, this will allow more protein isoforms and post-translational modifications to be identified than using Trypsin alone.</figcaption>
</figure>
</div>
<h2 class="page_break_before" id="peptide-quantification">Peptide Quantification</h2>
<p>Discussion of methods to isotopically label peptides or proteins that enable quantification</p>
<ol type="1">
<li>SILAC/SILAM</li>
<li>iTRAQ</li>
<li>TMT</li>
<li>dimethyl labeling</li>
</ol>
<!-- todo: not sure if this is the best place for this figure about quantification strategies, do we need a whole section on quant? -->
<div id="fig:quant-summary" class="fignos">
<figure>
<img src="images/Summay_peptide-protein-labeling_.svg" data-tag="1" style="width:100.0%" alt="Figure 1: Quantitative strategies commonly used in proteomics. A) Label-free quantitation. Proteins are extracted from samples, enzymatically hydrolyzed into peptides and analyzed by mass spectrometry. Chromatographic peak areas from peptides are compared across samples that are analyzed sequentially. B) Metabolic labelling. Stable isotope labeling with amino acids in cell culture (SILAC) is based on feeding cells stable isotope labeled amino acids (“light” or “heavy”). Samples grown with heavy or light amino acids are mixed before cell lysis. The relative intensities of the heavy and light peptide are used to compute protein changes between samples. C) Isobaric or chemical labelling. Proteins are isolated separately from samples, enzymatically hydrolyzed into peptides, and then chemically tagged with isobaric stable isotope labels. These isobaric tags produce unique reporter mass-to-charge (m/z) signals that are produced upon fragmentation with MS/MS. Peptide fragment ions are used to identify peptides, and the relative reporter ion signals are used for quantification." />
<figcaption aria-hidden="true"><span>Figure 1:</span> <strong>Quantitative strategies commonly used in proteomics.</strong>
A) Label-free quantitation.
Proteins are extracted from samples, enzymatically hydrolyzed into peptides and analyzed by mass spectrometry.
Chromatographic peak areas from peptides are compared across samples that are analyzed sequentially.
B) Metabolic labelling.
Stable isotope labeling with amino acids in cell culture (SILAC) is based on feeding cells stable isotope labeled amino acids (“light” or “heavy”).
Samples grown with heavy or light amino acids are mixed before cell lysis.
The relative intensities of the heavy and light peptide are used to compute protein changes between samples.
C) Isobaric or chemical labelling.
Proteins are isolated separately from samples, enzymatically hydrolyzed into peptides, and then chemically tagged with isobaric stable isotope labels.
These isobaric tags produce unique reporter mass-to-charge (m/z) signals that are produced upon fragmentation with MS/MS.
Peptide fragment ions are used to identify peptides, and the relative reporter ion signals are used for quantification.</figcaption>
</figure>
</div>
<h2 id="peptide-or-protein-enrichment">Peptide or Protein Enrichment</h2>
<h3 id="protein-enrichment-e.g.-for-protein-protein-interactions">Protein enrichment (e.g. for protein protein interactions)</h3>
<ul>
<li>coIP</li>
<li>APEX</li>
<li>bioID</li>
<li>bioplex</li>
</ul>
<h3 id="peptide-enrichment">Peptide enrichment</h3>
<ul>
<li>antibody enrichments of modifications, e.g. lysine acetylation <span class="citation" data-cites="58knZUn0">[<a href="#ref-58knZUn0" role="doc-biblioref">93</a>]</span>.</li>
<li>TiO2 and Fe enrichment of phosphorylation</li>
<li>Glycosylation</li>
<li>SISCAPA</li>
</ul>
<h2 class="page_break_before" id="methods-for-peptide-purification">Methods for Peptide Purification</h2>
<ol type="1">
<li>Reverse phase including tips and cartridges</li>
<li>stage tips</li>
<li>in stage tip (iST)</li>
<li>SP2, SP3</li>
<li>s traps</li>
</ol>
<h2 class="page_break_before" id="types-of-mass-spectrometers-used-for-proteomics">Types of Mass Spectrometers used for Proteomics</h2>
<ol type="1">
<li>QQQ</li>
<li>Q-TOF</li>
<li>Q-Orbitrap</li>
<li>LTQ-Orbitrap</li>
<li>TOF/TOF</li>
<li>FT-ICR</li>
<li>types of ion mobility</li>
</ol>
<ul>
<li>SLIM</li>
<li>FAIMS</li>
<li>traveling wave</li>
<li>tims</li>
</ul>
<h2 class="page_break_before" id="peptide-ionization">Peptide Ionization</h2>
<p>Until the early 1990s, peptides analysis by mass spectrometry was challenging.
Hard ionization techniques in use at the time, like fast atom bombardment, were not directly applicable to peptides without destroying or breaking them.
The soft ionization tehniques however, revolutionized the proteomics field and it became possible to routinely ionize and analyze peptides using MALDI and ESI techniques at high-throughput scale.
These two techniques were so impactful that the 2002 Nobel Prize in Chemistry was co-awarded to John Fenn (ESI) and Koichi Tanaka (MALDI) “for their development of soft desorption ionisation methods for mass spectrometric analyses of biological macromolecules” <span class="citation" data-cites="YdLSljVE">[<a href="#ref-YdLSljVE" role="doc-biblioref">94</a>/]</span>.</p>
<h3 id="maldi">MALDI</h3>
<p>The term, Matrix-assisted LASER desorption ionization (MALDI), was coined by Hillenkamp and Karas in 1985<span class="citation" data-cites="zfiYj2co">[<a href="#ref-zfiYj2co" role="doc-biblioref">95</a>]</span>.
Karas and Hillenkamp discovered the MALDI technique first, although a similar ionization method was shown by Koichi Tanaka in 1988 <span class="citation" data-cites="15Ye4Za7Y">[<a href="#ref-15Ye4Za7Y" role="doc-biblioref">2</a>]</span>.
A few months later, Karas and Hillenkamp also demonstrated MALDI applied to protein ionization <span class="citation" data-cites="tnQHCFV7">[<a href="#ref-tnQHCFV7" role="doc-biblioref">96</a>]</span>.
It also created a controversy that the widely used method of MALDI from these two people had been overlooked, and the Nobel prize was awarded to Tanaka, whose system was rarely used<span class="citation" data-cites="1HBJs2tAB">[<a href="#ref-1HBJs2tAB" role="doc-biblioref">97</a>]</span>.</p>
<p>MALDI first requires the peptide sample to be co-crystallized with a matrix molecule, which is usually a volatile, low molecular-weight, organic aromatic compound.
Some examples of such compounds are cyno-hydroxycinnamic acid, dihyrobenzic acid, sinapinic acid, alpha-hydroxycinnamic acid, ferulic acid etc <span class="citation" data-cites="w7xaaHV0">[<a href="#ref-w7xaaHV0" role="doc-biblioref">98</a>/]</span>.
Subsequently, the analyte is placed in a vacuum chamber in which it is irradiated with a LASER, usually at 337nm <span class="citation" data-cites="wGHAVD9r">[<a href="#ref-wGHAVD9r" role="doc-biblioref">99</a>]</span>.
This laser energy is absorbed by the matrix, which then transfers that energy along with its free protons to the co-crystalized peptides without significantly breaking them.
The matrix and co-crystallized sample generate plumes, and the volatile matrix imparts its protons to the peptides as it gets ionized first.
The weak acidic conditions used as well as the acidic nature of the matrix allows easy exchange of protons for the peptides to get ionized and fly under the electrical field in the mass spectrometer.
These ionized peptides generally form the metastable ions, most of them will fragment quickly <span class="citation" data-cites="DEnAe0bs">[<a href="#ref-DEnAe0bs" role="doc-biblioref">100</a>]</span>.
However, it can take several milliseconds and the mass spectrometry analysis can be performed before this time.
Peptides ionized by MALDI almost always take up a single charge and thus observed and detected as [M+H]+ species.</p>
<h4 id="maldi-mechanism">MALDI Mechanism</h4>
<div id="fig:MALDI-mechanism" class="fignos">
<figure>
<img src="images/MALDI.png" data-tag="1" style="width:100.0%" alt="Figure 1: MALDI The analyte-matrix mixture is irradiated by a laser source, leading to ablation. Desorption and proton transfer ionize the analyte molecules that can then be accelerated into a mass spectrometer." />
<figcaption aria-hidden="true"><span>Figure 1:</span> <strong>MALDI</strong>
The analyte-matrix mixture is irradiated by a laser source, leading to ablation.
Desorption and proton transfer ionize the analyte molecules that can then be accelerated into a mass spectrometer.</figcaption>
</figure>
</div>
<h3 id="electrospray-ionization">Electrospray Ionization</h3>
<p>ESI was first applied to peptides by John Fenn and coworkers in 1989 <span class="citation" data-cites="4Ajjx5fo">[<a href="#ref-4Ajjx5fo" role="doc-biblioref">1</a>]</span>.
Concepts related to electrospray Ionization (ESI) were published at least as early as 1882, when Lord Rayleigh described the number of charges that could assemble on the surface of a droplet <span class="citation" data-cites="BslEV2vj">[<a href="#ref-BslEV2vj" role="doc-biblioref">101</a>]</span>.
ESI is usually coupled with reverse-phase liquid-chromatography of peptides directly interfaced to a mass spectrometer.
A high voltage (~ 2 kV) is applied between the spray needle and the mass spectrometer.
As solvent exits the needle, it forms droplets that take on charge at the surface, and through a debated mechanism, those charges are imparted to peptide ions.
The liquid phase is generally kept acidic to help impart protons easily to the analytes.</p>
<p>Tryptic peptides ionized by ESI usually carry one charge one the side chain of their c-terminal residue (Arg or Lys) and one charge at their n-terminal amine.
Peptides can have more than one charge if they have a longer peptide backbone, have histidine residues, or have missed cleavages leaving extra Arg and Lys.
In most cases, peptides ionized by ESI are observed at more than one charge state.
Evidence suggests that the distribution of peptide charge states can be maniupulated through chemical additives <span class="citation" data-cites="11vyuKWyw">[<a href="#ref-11vyuKWyw" role="doc-biblioref">102</a>]</span>.</p>
<h4 id="electrospray-mechanism">Electrospray Mechanism</h4>
<div id="fig:ESI-mechanism" class="fignos">
<figure>
<img src="images/ESI.png" data-tag="1" style="width:100.0%" alt="Figure 1: Electrospray Ionization Charged droplets are formed, their size is reduced due to evaporation until charge repulsion leads to Coulomb fission and results in charged analyte molecules." />
<figcaption aria-hidden="true"><span>Figure 1:</span> <strong>Electrospray Ionization</strong>
Charged droplets are formed, their size is reduced due to evaporation until charge repulsion leads to Coulomb fission and results in charged analyte molecules.</figcaption>
</figure>
</div>
<p>The main goal of ESI is the production of gas-phase ions from electrolyte ions in solution.
During the process of ionization, the solution emerging from the electrospray needle or capillary is distorted into a Taylor cone and charged droplets are formed.
The charged droplets subsequently decrease in size due to solvent evaporation.
As the droplets shrink, the charge density and Coulombic repulsion increase.
This process destabilizes the droplets until the repulsion between the charges is higher than the surface tension and they fission (Coulomb explosion) <span class="citation" data-cites="4mpHldYP">[<a href="#ref-4mpHldYP" role="doc-biblioref">103</a>]</span> <span class="citation" data-cites="93IPP3bH">[<a href="#ref-93IPP3bH" role="doc-biblioref">104</a>]</span>.
Typical bottom-up proteomics experiments make use of acidic analyte solutions which leads to the formation of positively charged analyte molecules due to an excess presence of protons.</p>
<!-- TODO, explicitly describe the taylor cone and coulombic explosion -->
<!-- TODO: discussion of ionizaation mechanisms, ion evaporation versus charge residue model -->
<h2 class="page_break_before" id="data-acquisition">Data Acquisition</h2>
<!-- Figures:
1. show a plot of the polularity of MRM/DIA/DDA/ over time? 
-->
<p>Hybrid mass spectrometers used for modern proteome analysis offer the flexibility to collect data in many different ways.
Data acquisition strategies differ in the sequence of precursor scans and fragment ion scans, and in how analytes are chosen for MS/MS.
Constant innovation to develop better data collection methods improves our view of the proteome, but many method options may confuse newcomers.
This section provides an overview of the general classes of data collection methods.</p>
<p>Data acquisition strategies for proteomics fall into one of two groups.</p>
<ol type="1">
<li>Data dependent acquisition (DDA), in which the exact scan sequence in each analysis depends on the data that the mass spectrometer observes.</li>
<li>Data independent acquisition (DIA), in which the exact scan sequence in each analysis DOES NOT depend on the data; the collected scans are the same whether you inject yeast peptides, human peptides, or a solvent blank.</li>
</ol>
<p>DDA and DIA can both be futher subdivided in to targeted and untargeted methods.</p>
<!-- to do: figure showing the four basic types of data acquisition and their names, like four squares? -->
<h3 id="dda">DDA</h3>
<p>In most cases, the peptide masses that will be observed are not known before doing the experiment.
Data collection methods must account for this.
DDA was invented in the early 1990s, which enabled collecting MS/MS spectra for observed peptides as they eluting from the LC column <span class="citation" data-cites="JpKe7ymh byV7q4rP 1BU9KEnwO">[<a href="#ref-JpKe7ymh" role="doc-biblioref">105</a>,<a href="#ref-byV7q4rP" role="doc-biblioref">106</a>,<a href="#ref-1BU9KEnwO" role="doc-biblioref">107</a>]</span>.</p>
<h4 id="untargeted-dda">Untargeted DDA</h4>
<p>A common method currently used in modern proteomics is untargeted DDA.
The MS collects precursor (MS1) scans iteratively until precursor mass envelopes meeting certain criteria are detected.
Criteria for selection are usually specific charge states and a minimum signal intensity.
When those ions meet these criteria, the MS selects those masses for fragmentation.</p>
<p>Because ions are selected as they are observed, repeated DDA of the same sample will produce a different set of identifications.
This stochasticity is the main drawback of DDA.
To ameliorate this issue, often strategies are used to transfer identifications between multiple sample analyses.
This transfer of IDs across runs is known as “match between runs”, which was originally made famous by the processing software MaxQaunt <span class="citation" data-cites="vO5QsPvL Gs7HWzAD">[<a href="#ref-vO5QsPvL" role="doc-biblioref">108</a>,<a href="#ref-Gs7HWzAD" role="doc-biblioref">109</a>]</span>.
There are several other similar tools and strategies, including the accurate mass and time approach <span class="citation" data-cites="66LthRpU">[<a href="#ref-66LthRpU" role="doc-biblioref">110</a>]</span>, Q-MEND <span class="citation" data-cites="MaoP0MGM">[<a href="#ref-MaoP0MGM" role="doc-biblioref">111</a>]</span>, IDEAL-Q <span class="citation" data-cites="5H13CEWA">[<a href="#ref-5H13CEWA" role="doc-biblioref">112</a>]</span> and superHIRN <span class="citation" data-cites="rRs3KNfz">[<a href="#ref-rRs3KNfz" role="doc-biblioref">113</a>]</span>.
More recent work has introducted statistical assessment of MBR methods using a two-proteome model <span class="citation" data-cites="1EHOiWEPV">[<a href="#ref-1EHOiWEPV" role="doc-biblioref">114</a>]</span>.
Statistically controlled MBR is currently available in the IonQuant tool <span class="citation" data-cites="XGf3EEUa">[<a href="#ref-XGf3EEUa" role="doc-biblioref">115</a>]</span>.</p>
<p>Because DDA is required for quantification of proteins using isobaric tags like TMT, this stochasiticity of DDA limits the ability to compare quantities across batches.
For example, if you have 3o samples, you can use two sets of the 16-plex kit to label 15 samples in each set with one channel labeled by a pooled sample to enable comparison across the groups.
When you collect DDA data from each of those sets, each set will have MS/MS data from an overlapping but different set of peptides.
If one set has MS/MS from a peptide but the other set does not, then that peptide cannot be quantified in the whole sample group.
This limits the number of quantified proteins in large TMT experiments with multiple batches.
<!-- mention intelligent DDA and real time search? --></p>
<h4 id="targeted-dda">Targeted DDA</h4>
<p>Targeted DDA is not common in modern proteomics.
In targeted DDA, in addition to general criteria like a minimum intensity and a certain charge state, the mass spectrometer looks for specific masses.
These masses might be previously observed signals that were previously missed by MS/MS <span class="citation" data-cites="2D6WnK0w yqW0xFtN">[<a href="#ref-2D6WnK0w" role="doc-biblioref">116</a>,<a href="#ref-yqW0xFtN" role="doc-biblioref">117</a>]</span>.
In these studies, the sample is first analyzed by LC-MS to detect precursor ion features with some software, and then subsequent analyses target those masses for fragmentation with inclusion lists until they are all fragmeted.
This was shown to increase proteome converage.</p>
<h4 id="dda-methods-for-modifications">DDA methods for modifications</h4>
<p>There are also DDA methods that look for specific fragment or neutral loss ions in the resulting spectra.
For example, when linear ion traps were the main proteomics workhorses, CID analysis of phosphopeptides would result in predominantly neutral loss of the phosphate with limited sequence ion information.
To gain sequence ions in these experiemnts, instruments could be set to isolate a loss of 98 Thompsons for MS3 <span class="citation" data-cites="SngtJdsy WtrZNDoq">[<a href="#ref-SngtJdsy" role="doc-biblioref">118</a>,<a href="#ref-WtrZNDoq" role="doc-biblioref">119</a>]</span>.
The newer collisional dissociation technique HCD significantly improves the detection of peptide fragments with the phosphorylation intact on fragment ions, and thus, this neutral loss scanning technique is no longer common.</p>
<p>A similar strategy was introduced for N-linked glycopeptides <span class="citation" data-cites="GL3PYaNa">[<a href="#ref-GL3PYaNa" role="doc-biblioref">120</a>]</span>.
Collisional dissociation of glycosylated peptides produces oxonium ions at 204.09 (HexNAc) or 366.14 (HexHexNAc).
In oxonium ions from the glycosylation were detected in the top 20 fragments of the HCD spectra, then an ETD scan was triggered.
This ETD scan provides information about the peptide sequence, while the original HCD scan provides glycan structure information.</p>
<h3 id="dia">DIA</h3>
<p>The simplest method to operate a mass spectrometer is to have predefined scans that are collected for each sample analysis.
This is DIA; the scans that are collected do not depend on the data that the instrument observes.
Thus, the scan sequence is the same every time.
Although simple in terms of data collection, when the scan sequence includes MS/MS, sophisticated software is required to analyze the data.
Like DDA, DIA can also be either targeted or untargeted <span class="citation" data-cites="wAKbWLXc">[<a href="#ref-wAKbWLXc" role="doc-biblioref">121</a>]</span>:
The two targeted DIA methods are selected reaction monitoring (SRM) or multiple reaction monitoring (MRM), and untargeted DIA (uDIA) is often referred to simply as “DIA” or SWATH (Figure 4).</p>
<div id="fig:DIA-types" class="fignos">
<figure>
<img src="images/data-acquisition-DIA-types.svg" data-tag="4" style="width:100.0%" alt="Figure 4: Types of DIA. A) SRM/MRM. Peptides are ionized by ESI and although there are many peptides entering the mass spectrometer at any time, the first quadruople (Q1) isolates one mass, which is then fragmented by HCD. Fragment masses from the peptide are then selected in the third quadruople (Q3). This leads to very low noise and high sensitivity. B) PRM. Like MRM, peptides are selected in the first quadruople, but this analysis is done on a high resolution instrument like an Orbitrap or TOF. Selectivity is gained by exploiting the high mass accuracy and resolution to monitor multiple fragment ions. C) uDIA/SWATH. Like MRM and PRM, peptides are isolated with Q1, but in this case a much wider isolation window is used. This usually results in co-isolation of many peptides simultaneously. Fragments from many peptides are measured with high resolution and high mass accuracy. Special software is used to get peptide identities and quantities from the fragment ions." />
<figcaption aria-hidden="true"><span>Figure 4:</span> <strong>Types of DIA.</strong>
A) SRM/MRM.
Peptides are ionized by ESI and although there are many peptides entering the mass spectrometer at any time, the first quadruople (Q1) isolates one mass, which is then fragmented by HCD.
Fragment masses from the peptide are then selected in the third quadruople (Q3).
This leads to very low noise and high sensitivity.
B) PRM.
Like MRM, peptides are selected in the first quadruople, but this analysis is done on a high resolution instrument like an Orbitrap or TOF.
Selectivity is gained by exploiting the high mass accuracy and resolution to monitor multiple fragment ions.
C) uDIA/SWATH.
Like MRM and PRM, peptides are isolated with Q1, but in this case a much wider isolation window is used.
This usually results in co-isolation of many peptides simultaneously.
Fragments from many peptides are measured with high resolution and high mass accuracy.
Special software is used to get peptide identities and quantities from the fragment ions.</figcaption>
</figure>
</div>
<h4 id="targeted-dia">Targeted DIA</h4>
<p>The first type of targeted DIA is called SRM or MRM <span class="citation" data-cites="NQj9Wees">[<a href="#ref-NQj9Wees" role="doc-biblioref">122</a>]</span>.
The popularity of this method in the literature peaked in 2014, with just under 1,500 documents on pubmed that year resulting from a search for “MRM”.
In this strategy, the QQQ MS is set so that the first quadrupole selects the precursor mass of the peptide(s) of interest, the second quadrupole fragments the peptide, and the third quadruople monitors the product of specific fragments from that peptide.
This strategy is very sensitive and has the benefit of very low noise.
The fragments monitored in Q3 are chosen such that it is unlikely these fragments could arise from another peptide.
Usually at least a few transitions are monitored for each peptide in order to get multiple measures for that peptide.</p>
<p>An early example of MRM applied to quantify c-reactive protein was in 2004 <span class="citation" data-cites="1FAMk6Qey">[<a href="#ref-1FAMk6Qey" role="doc-biblioref">123</a>]</span>.
Around the same time, SRM was combined with antibody enrichment of peptides from target proteins <span class="citation" data-cites="63Pqc9VE">[<a href="#ref-63Pqc9VE" role="doc-biblioref">124</a>]</span>.
This approach was popular for analysis of plasma proteins <span class="citation" data-cites="1B9P7OPqR">[<a href="#ref-1B9P7OPqR" role="doc-biblioref">125</a>]</span>.
These early examples led to many more studies that used QQQ MS instruments to get accurate quantitation of many proteins in one injection <span class="citation" data-cites="iJJwYdQ1 ba7F2Epn">[<a href="#ref-iJJwYdQ1" role="doc-biblioref">126</a>,<a href="#ref-ba7F2Epn" role="doc-biblioref">127</a>]</span>.
Scheduling MRM measurement when chromatography is stable additionally enabled better utilization of instrument duty cycle and therefore monitoring of more peptides per injection <span class="citation" data-cites="jcu6ZT0x">[<a href="#ref-jcu6ZT0x" role="doc-biblioref">128</a>]</span>.
Efforts even developed libraries of transitions that allow quantification of any protein in model organisms <span class="citation" data-cites="eFcH0xbv">[<a href="#ref-eFcH0xbv" role="doc-biblioref">129</a>]</span>.</p>
<p>Another similar targeted DIA method is called parallel reaction monitoring (PRM) <span class="citation" data-cites="1Bt8NAhNo">[<a href="#ref-1Bt8NAhNo" role="doc-biblioref">130</a>]</span>.
Instead of using a QQQ instrument, PRM uses a hybrid MS with a quadrupole and a high resolution mass analyzer, such as an Q-TOF or Q-Exactive.
The idea is that instead of monitoring specific fragments in Q3, the high mass accuracy can be used to filter peptide fragments for high selectivity and accurate quantification.
Studies have found that PRM and MRM/SRM have comparable dynamic range and linearity <span class="citation" data-cites="HB3XvRF2">[<a href="#ref-HB3XvRF2" role="doc-biblioref">131</a>]</span>.</p>
<h4 id="untargeted-dia">Untargeted DIA</h4>
<p>There were many implementations of uDIA over the years, starting in 2003 by Purvine et al from the Goodlett lab <span class="citation" data-cites="dWzeUq6d">[<a href="#ref-dWzeUq6d" role="doc-biblioref">132</a>]</span>.
In this first work they demonstrated uDIA using a Q-TOF with in source fragmentation, and showed that extracted ion chromatograms of precursor and fragment ions matched in shape suggesting that this could be used to identify and quantify peptides.
The following year, Venable et al from the Yates lab introduced uDIA with an ion trap <span class="citation" data-cites="yn40ByaQ">[<a href="#ref-yn40ByaQ" role="doc-biblioref">133</a>]</span>.
Subsequent methods include MSE <span class="citation" data-cites="Z3Hfnak1">[<a href="#ref-Z3Hfnak1" role="doc-biblioref">134</a>]</span>, PAcIFIC <span class="citation" data-cites="Wb9egjJg">[<a href="#ref-Wb9egjJg" role="doc-biblioref">135</a>]</span>, all ions fragmentation (AIF) <span class="citation" data-cites="OubW7j0g">[<a href="#ref-OubW7j0g" role="doc-biblioref">136</a>]</span>.
Computational methods were also developed to automate interpretation of this data, such as DeMux <span class="citation" data-cites="OubW7j0g">[<a href="#ref-OubW7j0g" role="doc-biblioref">136</a>]</span>, XDIA <span class="citation" data-cites="y71ZDsTJ">[<a href="#ref-y71ZDsTJ" role="doc-biblioref">137</a>]</span>, and ETISEQ <span class="citation" data-cites="ckj0XYOM">[<a href="#ref-ckj0XYOM" role="doc-biblioref">138</a>]</span>.</p>
<p>The paper that is often cited for uDIA that led to widespread adoption was by Gillet et al. from the Aebersold group in 2012 <span class="citation" data-cites="11ccXv5xm">[<a href="#ref-11ccXv5xm" role="doc-biblioref">139</a>]</span>.
In this paper they branded the idea as SWATH.
Widespread adoption may have been facilitated by the co-marketing of this idea by ABSciex as a proteomics solution on their new 5600 Q-TOF (called “tripleTOF” despite containing only one TOF, likely a portmanteau of “triple quadrupole” and “Q-TOF”).
Importantly, in the Gillet et al. paper the authors described a computational method to extract information from SWATH where peptides of interest were queried against the data.
They also demonstrated the application of SWATH to measure proteomic changes that happen in diauxic shift, and showed that SWATH can reveal modified peptides, in this case a methionine oxidation.</p>
<p>There are also many papers describing uDIA with orbitraps.
One early example described combining random isolation windows together and then demultiplexing the chimeric spectra <span class="citation" data-cites="1GCjdtGKY">[<a href="#ref-1GCjdtGKY" role="doc-biblioref">140</a>]</span>.
In another landmark paper, over 6,000 proteins were identified from mouse tissue by at least 2 peptides <span class="citation" data-cites="1GvzNjeV3">[<a href="#ref-1GvzNjeV3" role="doc-biblioref">141</a>]</span>.
In 2018, the new model orbitrap at that time (HF-X) enabled identification of nearly 6,000 human proteins in only 30 minutes.
Currently orbitraps have all but replaced the Sciex Q-TOFs for DIA data collection.</p>
<p>A new direction in uDIA is the addition of ion separation by ion mobility.
This has appeared in two forms.
On the timsTOF, diaPASEF makes use of the trapped ion mobility to increase speed and sensitivity of analysis <span class="citation" data-cites="YKRUqViz">[<a href="#ref-YKRUqViz" role="doc-biblioref">142</a>]</span>.
On the orbitrap, the combination of FAIMS and DIA has enabled the identification of over 10,000 proteins from one sample, which is a major milestone <span class="citation" data-cites="44oI1g68">[<a href="#ref-44oI1g68" role="doc-biblioref">143</a>]</span>.</p>
<h2 class="page_break_before" id="analysis-of-raw-data">Analysis of Raw Data</h2>
<p>The goal of basic data analysis is to convert raw spectral data into identities and quantities of peptides and proteins that can be used for biologically-focused analysis.
This step may often include measures of quality control, cross-run data normalization, quantification on different levels (precursor, peptide, protein), protein inference, PTM (post translational modification) localization and also first steps of data analysis, such as statistical hypothesis tests.</p>
<p>In typical bottom-up proteomics experiments, proteins are digested into peptides and further analyzed with LC-MS/MS systems.
Peptides can have different PTMs and ionize differently depending on their length and amino acid distributions.
Therefore, mass spectrometers often record different charge and modification states of one single peptide.
The entity that is recorded on a mass spectrometer is usually referred to as a precursor ion (peptide with its modification and charge state).
This precursor ion is fragmented and the precursor or peptide sequences are obtained though spectral matching.
The quantity of a precursor is estimated with various methods.
The measured precursor quantities are combined to generate a peptide quantity.
Peptides are also often combined into a protein group through protein inference, which combines multiple peptide identifications into a single protein identification <span class="citation" data-cites="imR1dFei">[<a href="#ref-imR1dFei" role="doc-biblioref">144</a>]</span> <span class="citation" data-cites="wtH5a4Ax">[<a href="#ref-wtH5a4Ax" role="doc-biblioref">145</a>]</span>.
Protein inference is still a challenge in bottom-up proteomics.</p>
<p>Due to the inherent differences in the data structures of DDA and DIA measurements, there exist different types of software that can facilitate the steps mentioned above.
The existing software for DDA and DIA analysis can be further divided into freeware and non-freeware:</p>
<h4 id="dda-freeware">DDA freeware</h4>
<table>
<colgroup>
<col style="width: 13%" />
<col style="width: 40%" />
<col style="width: 46%" />
</colgroup>
<thead>
<tr class="header">
<th style="text-align: center;">Name</th>
<th style="text-align: center;">Publication</th>
<th style="text-align: center;">Website</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: center;">MaxQuant</td>
<td style="text-align: center;">Cox and Mann, 2008<span class="citation" data-cites="18PIzg2Co">[<a href="#ref-18PIzg2Co" role="doc-biblioref">146</a>]</span></td>
<td style="text-align: center;"><a href="https://www.maxquant.org/">MaxQuant</a></td>
</tr>
<tr class="even">
<td style="text-align: center;">MSFragger</td>
<td style="text-align: center;">Kong et al., 2017<span class="citation" data-cites="bKpYR6PB">[<a href="#ref-bKpYR6PB" role="doc-biblioref">147</a>]</span></td>
<td style="text-align: center;"><a href="https://msfragger.nesvilab.org/">MSFragger</a></td>
</tr>
<tr class="odd">
<td style="text-align: center;">Mascot</td>
<td style="text-align: center;">Perkins et al., 1999<span class="citation" data-cites="rKtsv6G">[<a href="#ref-rKtsv6G" role="doc-biblioref">148</a>]</span></td>
<td style="text-align: center;"><a href="https://www.matrixscience.com/">Mascot</a></td>
</tr>
<tr class="even">
<td style="text-align: center;">MS-GF+</td>
<td style="text-align: center;">Kim et al., <span class="citation" data-cites="1B2RVFOKK">[<a href="#ref-1B2RVFOKK" role="doc-biblioref">149</a>]</span></td>
<td style="text-align: center;"><a href="https://github.com/MSGFPlus/msgfplus">MS-GF+</a></td>
</tr>
<tr class="odd">
<td style="text-align: center;">X!Tandem</td>
<td style="text-align: center;">Craig et al., <span class="citation" data-cites="IcvYCrHi iK4IJ8ED">[<a href="#ref-IcvYCrHi" role="doc-biblioref">150</a>,<a href="#ref-iK4IJ8ED" role="doc-biblioref">151</a>]</span></td>
<td style="text-align: center;"><a href="https://www.thegpm.org/tandem/">GPMDB</a></td>
</tr>
</tbody>
</table>
<h4 id="dia-freeware">DIA freeware:</h4>
<table>
<colgroup>
<col style="width: 13%" />
<col style="width: 40%" />
<col style="width: 46%" />
</colgroup>
<thead>
<tr class="header">
<th style="text-align: center;">Name</th>
<th style="text-align: center;">Publication</th>
<th style="text-align: center;">Website</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: center;">MaxDIA</td>
<td style="text-align: center;">Cox and Mann, 2008<span class="citation" data-cites="18PIzg2Co">[<a href="#ref-18PIzg2Co" role="doc-biblioref">146</a>]</span></td>
<td style="text-align: center;"><a href="https://www.maxquant.org/">MaxQuant</a></td>
</tr>
<tr class="even">
<td style="text-align: center;">Skyline</td>
<td style="text-align: center;">MacLean et al., 2010<span class="citation" data-cites="CObAggjA">[<a href="#ref-CObAggjA" role="doc-biblioref">152</a>]</span></td>
<td style="text-align: center;"><a href="https://skyline.ms/project/home/software/Skyline/begin.view">Skyline</a></td>
</tr>
<tr class="odd">
<td style="text-align: center;">DIA-NN</td>
<td style="text-align: center;">Demichev et al., 2019<span class="citation" data-cites="1HLH2DWkK">[<a href="#ref-1HLH2DWkK" role="doc-biblioref">153</a>]</span></td>
<td style="text-align: center;"><a href="https://github.com/vdemichev/DiaNN">DIA-NN</a></td>
</tr>
</tbody>
</table>
<h4 id="targeted-proteomics-freeware">Targeted proteomics freeware:</h4>
<table>
<colgroup>
<col style="width: 13%" />
<col style="width: 40%" />
<col style="width: 46%" />
</colgroup>
<thead>
<tr class="header">
<th style="text-align: center;">Name</th>
<th style="text-align: center;">Publication</th>
<th style="text-align: center;">Website</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: center;">Skyline</td>
<td style="text-align: center;">MacLean et al., 2010<span class="citation" data-cites="CObAggjA">[<a href="#ref-CObAggjA" role="doc-biblioref">152</a>]</span></td>
<td style="text-align: center;"><a href="https://skyline.ms/project/home/software/Skyline/begin.view">Skyline</a></td>
</tr>
</tbody>
</table>
<h4 id="dda-non-freeware">DDA non-freeware:</h4>
<table>
<colgroup>
<col style="width: 13%" />
<col style="width: 40%" />
<col style="width: 46%" />
</colgroup>
<thead>
<tr class="header">
<th style="text-align: center;">Name</th>
<th style="text-align: center;">Publication</th>
<th style="text-align: center;">Website</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: center;">ProteomeDiscoverer</td>
<td style="text-align: center;"></td>
<td style="text-align: center;"><a href="https://www.thermofisher.com/ch/en/home/industrial/mass-spectrometry/liquid-chromatography-mass-spectrometry-lc-ms/lc-ms-software/multi-omics-data-analysis/proteome-discoverer-software.html">ProteomeDiscoverer</a></td>
</tr>
<tr class="even">
<td style="text-align: center;">Mascot</td>
<td style="text-align: center;">Perkins et al., 1999<span class="citation" data-cites="rKtsv6G">[<a href="#ref-rKtsv6G" role="doc-biblioref">148</a>]</span></td>
<td style="text-align: center;"><a href="https://www.matrixscience.com/">Mascot</a></td>
</tr>
<tr class="odd">
<td style="text-align: center;">Spectromine</td>
<td style="text-align: center;"></td>
<td style="text-align: center;"><a href="ttps://biognosys.com/software/spectromine/?gclid=Cj0KCQiAoY-PBhCNARIsABcz770mjUz6iavBr9Ql7RPUdMvaHu9RYgPNrEfZco1wExEeoFwnQXuCHscaAlgBEALw_wcB">Spectromine</a></td>
</tr>
<tr class="even">
<td style="text-align: center;">PEAKS</td>
<td style="text-align: center;">Tran et al., 2018<span class="citation" data-cites="IUqTUA43">[<a href="#ref-IUqTUA43" role="doc-biblioref">154</a>]</span></td>
<td style="text-align: center;"><a href="https://www.bioinfor.com/peaks-studio/">PEAKS</a></td>
</tr>
</tbody>
</table>
<h4 id="dia-non-freeware">DIA non-freeware:</h4>
<table>
<colgroup>
<col style="width: 13%" />
<col style="width: 40%" />
<col style="width: 46%" />
</colgroup>
<thead>
<tr class="header">
<th style="text-align: center;">Name</th>
<th style="text-align: center;">Publication</th>
<th style="text-align: center;">Website</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: center;">Spectronaut</td>
<td style="text-align: center;">Bruderer et al., 2015<span class="citation" data-cites="UKcaDUJ7">[<a href="#ref-UKcaDUJ7" role="doc-biblioref">155</a>]</span></td>
<td style="text-align: center;"><a href="https://biognosys.com/software/spectronaut/">Spectronaut</a></td>
</tr>
<tr class="even">
<td style="text-align: center;">PEAKS</td>
<td style="text-align: center;">Tran et al., 2018<span class="citation" data-cites="IUqTUA43">[<a href="#ref-IUqTUA43" role="doc-biblioref">154</a>]</span></td>
<td style="text-align: center;"><a href="https://www.bioinfor.com/peaks-studio/">PEAKS</a></td>
</tr>
</tbody>
</table>
<h4 id="data-summary-and-interpretation">Data Summary and Interpretation</h4>
<table>
<colgroup>
<col style="width: 15%" />
<col style="width: 35%" />
<col style="width: 48%" />
</colgroup>
<thead>
<tr class="header">
<th style="text-align: center;">Name</th>
<th style="text-align: center;">Publication</th>
<th style="text-align: center;">Website</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td style="text-align: center;">Peptide Shaker</td>
<td style="text-align: center;">Vaudel <em>et al.</em>, 2015<span class="citation" data-cites="B10jxZjh tZ6JV4UF">[<a href="#ref-B10jxZjh" role="doc-biblioref">156</a>,<a href="#ref-tZ6JV4UF" role="doc-biblioref">157</a>]</span></td>
<td style="text-align: center;"><a href="http://compomics.github.io/projects/peptide-shaker.html">PeptideShaker</a>, <a href="https://github.com/barsnes-group/peptide-shaker-online">Peptide Shaker Online</a></td>
</tr>
</tbody>
</table>
<h3 id="analysis-of-dda-data">Analysis of DDA data</h3>
<p>DDA data analysis either directly uses the vendor proprietary data format directly with a proprietary search engine like Mascot, Sequest (through Proteome Discoverer), Paragon (through Protein Pilot), or it can be processed through one of the many freely available search engines or pipelines, for example, MaxQuant, MSGF+, X!Tandem, Morpheus, MSFragger, and OMSSA.
Tables 1 and 4 give weblinks and citations for these software tools.
<!-- todo: need to make sure all these are listed in the table and cited -->
<!-- todo: need a paragraph discussing intergration environments like the transproteomics pipeline (TPP) and IDpicker -->
For analysis with freeware, raw data is converted to either text-based MGF (mascot generic format) or into a standard open XML format like mzML <span class="citation" data-cites="zsmAmqZJ">[<a href="#ref-zsmAmqZJ" role="doc-biblioref">158</a>]</span> [<span class="citation" data-cites="fQHXeOG1">[<a href="#ref-fQHXeOG1" role="doc-biblioref">159</a>]</span>]<span class="citation" data-cites="r2hPariY">[<a href="#ref-r2hPariY" role="doc-biblioref">160</a>]</span>.
The appropriate FASTA file containing proteins predicted from that organism’s genome is chosen as a reference database to search the experimental spectra.
All search parameters like peptide and fragment mass errors (i.e. MS1 and MS2 tolerances), enzyme specificity, number of missed cleavages, chemical artefacts (fixed modifications) and potential biological modifications (variable/dynamic modifications) are specified before executing the search.
The search algorithm scores each query spectrum against its possible peptide matches <span class="citation" data-cites="1GxDrwPIW">[<a href="#ref-1GxDrwPIW" role="doc-biblioref">161</a>]</span>.
A spectrum and its best scoring candidate peptide are called a peptide spectrum match (PSM).
The scores reflect a <em>goodness-of-fit</em> between an experimental spectrum and a theoretical one and do not necessarily depict the correctness of the peptide assignment.</p>
<p>For evaluating the matches, a decoy database is preferred as a null model for peptide matching.
A randomized or reversed version of target database is used as a nonparametric null model.
The decoy database can be searched separate from the target database (Kall’s method)<span class="citation" data-cites="bjANWlBN">[<a href="#ref-bjANWlBN" role="doc-biblioref">162</a>]</span> or it can be combined with the target database before search (Elias and Gygi method)<span class="citation" data-cites="BrkRiRe8">[<a href="#ref-BrkRiRe8" role="doc-biblioref">163</a>]</span>.
Using either separate method or concatenated database search method, an estimate of false hits can be calculated which is used to estimate the false discovery rate (FDR) <span class="citation" data-cites="5mQtZh5o">[<a href="#ref-5mQtZh5o" role="doc-biblioref">164</a>]</span>.
The FDR denotes the proportion of false hits in the population accepted as true.
For Kall’s method: the false hits are estimated to be the number of decoys above a given threshold.
It is assumed that the number of decoy hits that pass a threshold are the false hits.
A similar number of target population may also be false.
Therefore, the FDR is calculated as:</p>
<p><span class="math display">\[FDR = \frac{Decoy PSMs}{Target PSMs}\]</span></p>
<p>For Elias and Gygi Method, the target population in which FDR is estimated changes.
The target and decoy hits coming from a joint database compete against each other.
For any spectrum, either a target or a decoy peptide can be the best hit.
It is argued that the joint target-decoy population has decoy hits as confirmed false hits.
However, due to the joint database search, the target database may also have equal number of false hits.
Thus, the number of false hits is multiplied by two for FDR estimation.</p>
<p><span class="math display">\[FDR = \frac{2 * Decoy PSMs}{Target + Decoy PSMs}\]</span></p>
<h3 id="strategies-for-analysis-of-dia-data">Strategies for analysis of DIA data</h3>
<h3 id="targeted-proteomics-data-analysis">Targeted proteomics data analysis</h3>
<h3 id="quality-control">Quality control</h3>
<h3 id="statistical-hypothesis-testing">Statistical hypothesis testing</h3>
<h2 class="page_break_before" id="databases">Databases</h2>
<h3 id="what-are-they-and-where-do-you-get-them">What are they and where do you get them?</h3>
<h4 id="protein-database-sources-and-types">Protein Database Sources and Types</h4>
<p>Many mass spectrometry-based proteomic techniques use search algorithms that require a defined theoretical search space to identify peptide sequences based on precursor mass and fragmentation patterns, which are then used to infer the presence and abundance of a protein.
The search space is calculated from the potential proteins in a sample, which includes the proteome (often a single species) and expected contaminants.
This is called database searching and the flat file of protein sequences in FASTA format acts as a protein database.
In this section, we will describe major resources for proteome FASTA files (protein sequence collections), how to retrieve them, and suggested best practices for preserving FASTA file provenance to improve reproducibility.</p>
<p>In general, FASTA sequence collections can be retrieved from three central clearing houses: UniProt, RefSeq, and Ensembl.
These will be discussed separately below as they each have specific design goals, data products, and unique characteristics.
It is important to learn the following three points for each resource: the source of the underlying data, canonical versus non-canonical sequences, and how versioning works.
These points, along with general best practices, such as using a taxonomic identifier, are essential to understand and communicate search settings used in analyses of proteomic datasets.
Finally, it is critical to understand that sequence collections from these three resources are not the same, nor do they offer the same sets of species.</p>
<p>Key terminology may vary between resources, so these terms are defined here.
The term “taxon identifier” is used across resources and is based on the NCBI taxonomy database.
Every taxonomic node has a number, e.g., Homo sapiens (genus species) is 9606 and Mammalia (class) is 40674.
This can be useful when retrieving and describing protein sequence collections.
Another term used is “annotation”, which has different meanings in different contexts.
Broadly, a “genome annotation” is the result of an annotation pipeline to predict coding sequences, and often a gene name/symbol if possible.
Two examples are MAKER <span class="citation" data-cites="KDnDuk25">[<a href="#ref-KDnDuk25" role="doc-biblioref">165</a>]</span> and the RefSeq annotation pipeline <span class="citation" data-cites="12mAW7BXf">[<a href="#ref-12mAW7BXf" role="doc-biblioref">166</a>]</span>.
Alternatively, “protein annotation” (or gene annotation) often refers to the annotation of proteins (gene products) using names and ontology (i.e., protein names, gene names/symbols, functional domains, gene onotology, keywords, etc.).
Protein annotation is termed “biocuration” and described in detail by UniProt <span class="citation" data-cites="BRlMNUOW">[<a href="#ref-BRlMNUOW" role="doc-biblioref">167</a>]</span>.
Lastly, there are established minimum reporting guidelines for referring to FASTA files established in MIAPE: Mass Spectrometry Informatics that are taxon identifier and number of sequences <span class="citation" data-cites="1EbCD9r1v paTtvUb8">[<a href="#ref-1EbCD9r1v" role="doc-biblioref">168</a>,<a href="#ref-paTtvUb8" role="doc-biblioref">169</a>]</span>.
The FASTA file naming suggestions below are not official but are suggested as a best practice.</p>
<h5 id="uniprot">UniProt</h5>
<p>The Universal Protein Resource (UniProt) <span class="citation" data-cites="16uefmiYk">[<a href="#ref-16uefmiYk" role="doc-biblioref">170</a>]</span>, has three different products: UniProt Knowledgebase (UniProtKB), the UniProt Reference Clusters (UniRef), and the UniProt Archive (UniParc).
The numerous resources and capabilities associated with the UniProt are not explored in this section, but these are well described on UniProt’s website.
UniProtKB is the source of proteomes across the Tree of Life and is the resource we will be describing herein.
There are broadly two types of proteome sequence collections: Swiss-Prot/TrEMBL and designated proteomes.
The Swiss-Prot/TrEMBL type can be understood by discussing how data is integrated into UniProt.
Most protein sequences in UniProt are derived from coding sequences submitted to EMBL-Bank, GenBank and DDBJ.
These translated sequences are initially imported into TrEMBL database, which is why TrEMBL is also termed “unreviewed”.
There are other sources of protein sequences, as described by UniProt <span class="citation" data-cites="1BIF8v4YE">[<a href="#ref-1BIF8v4YE" role="doc-biblioref">171</a>]</span>.
These include the Protein Data Bank (PDB), direct protein sequencing, sequences derived from the literature, gene prediction (from sources such as Ensembl) or in-house prediction by UniProt itself.
Protein sequences can then be manually curated into the Swiss-Prot database using multiple outlined steps (described in detail by UniProt here <span class="citation" data-cites="Q2scu90J">[<a href="#ref-Q2scu90J" role="doc-biblioref">172</a>]</span>) and is why Swiss-Prot is also termed “reviewed”.
Note that more than one TrEMBL entry may be removed and replaced by a single Swiss-Prot entry during curation.
A search of “organism:9606” at UniProtKB will retrieve both the Swiss-Prot/reviewed and TrEMBL/unreviewed sequences for Homo sapiens.
The entries do not overlap, so users often either use just Swiss-Prot or Swiss-Prot combined with TrEMBL, the latter being the most exhaustive option.
With ever-increasing numbers of high-quality genome assemblies processed with robust automated annotation pipelines, TrEMBL entries will contain higher quality protein sequences than in the past.
In other words, if a mammal species has 20 000 to 40 000 entries in UniProtKB and many of these are TrEMBL, users should be comfortable using all the protein entries to define their search space (more on this later when discussing proteomes at UniProtKB).
Determining the expected size of a well-annotated proteome requires additional knowledge, but tools to answer these questions continue to improve.
As more and more genome annotations are generated, the backlog of manual curation continues to increase.
However, automated genome annotations are also rapidly improving, blurring the line between Swiss-Prot and TrEMBL utility.</p>
<p>The second type of protein sequence collections available at UniProtKB are designated proteomes, with subclasses of “proteome”, “reference proteome” or “pan-proteome”.
As defined by UniProt, a proteome is the set of proteins derived from the annotation of a completely sequenced genome assembly (one proteome per genome assembly).
This means that a proteome will include both Swiss-Prot and TrEMBL entries present in a single genome annotation, and that all entries in the proteome can be traced to a single complete genome assembly.
This aids in tracking provenance as assemblies change, and metrics of these assemblies are available.
These metrics include Benchmarking Universal Single-Copy Ortholog (BUSCO) score, and “Completeness” as Standard, Close Standard or Outlier based on the Complete Proteome Detector (CPD).
Given the quality of genome annotation pipelines, using a proteome as a FASTA file for a species is the preferred method of defining search spaces now.
Outside of humans, no higher eukaryotic Swiss-Prot sequence collections are complete enough for use in proteomics analyses, but this does not mean that the available Swiss-Prot plus TrEMBL protein sequence collection precludes accurate proteomic data analysis.
Lastly, the difference between reference proteome and proteome is used to highlight model organisms or organisms of interest, but not to imply improved quality.
UniProt also has support for the concept of “pan proteomes” (consensus proteomes for a closely related set of organisms) but this is mostly used for bacteria (e.g., strains of a given species will share a pan proteome).</p>
<p>When retrieving protein sequence collections as Swiss-Prot/TrEMBL or designated proteomes, there is an option of downloading “FASTA (canonical)” or “FASTA (canonical &amp; isoform)”.
The later includes additional manually annotated isoforms for Swiss-Prot sequences.
Each Swiss-Prot entry has one canonical sequence chosen by the manual curator.
Any additional sequence variants (mostly from alternative slicing) are annotated as differences with respect to the canonical sequence.
Specifying “canonical” will select only one protein sequence per Swiss-Prot entry while specifying “canonical &amp; isoforms” will download additional protein sequences by including isoforms for Swiss-Prot entries.
Recently, an option to “download one protein sequence per gene (FASTA)” has been added.
These FASTA files include Swiss-Prot and TrEMBL sequences to number about 20 000 protein sequences for a wide range of higher eukaryotic organisms.</p>
<p>The number of additional isoforms varies considerably by species.
In the human, mouse, and rat proteomes of the total number of entries, 26 %, 40 % and 72 % are canonical, respectively.
The choice of including isoforms is related to the search algorithm and experimental goals.
For instance, if differentiating isoforms is relevant, they should be included otherwise they will not be detected.
In cases where isoforms are present in the FASTA (evident by shared protein names) but these cannot be removed prior to downloading (e.g., California sea lion, Zalophus californianus, proteome UP000515165, release 2022_01), non-redundant FASTA files can be manually generated (i.e., “remove_duplicates.py” via <span class="citation" data-cites="16BHXvDGS">[<a href="#ref-16BHXvDGS" role="doc-biblioref">173</a>]</span>).
If possible, retrieving canonical protein sequences via proteomes is the most straight forward approach and in general appropriate for most search algorithms, versus the method of searching and downloading Swiss-Prot and/or TrEMBL entries.</p>
<p>Though FASTA files are the typical input of many search algorithms, UniProt also offers an XML and GFF format download.
In contrast to the flat FASTA file format, the XML format includes sequence information as well as associated information like PTMs, which is used in some search algorithms like MetaMorpheus <span class="citation" data-cites="1GfGl6YGi">[<a href="#ref-1GfGl6YGi" role="doc-biblioref">174</a>]</span>.</p>
<p>Once a protein sequence collection has been selected and retrieved, there is the evergreen question of how to name and report this to others in a way that allows them to reproduce the retrieval. The minimum reporting information is the taxon identified and number of sequences used <span class="citation" data-cites="1EbCD9r1v paTtvUb8">[<a href="#ref-1EbCD9r1v" role="doc-biblioref">168</a>,<a href="#ref-paTtvUb8" role="doc-biblioref">169</a>]</span>. The following naming format (and those below) augments this and is suggested for UniProtKB FASTA files (the use of underscores or hyphens is not critical):
[common or scientific name]-[taxon id]-uniprot-[swiss-prot/trembl/proteome]-[UP# if used]-[canonical/canonical plus isoform]-[release]
example of a Homo sapiens (human) protein fasta from UniProtKB:</p>
<p>Human-9606-uniprot-proteome-UP000005640-canonical-2022_01.fasta</p>
<p>The importance of the taxon identifier has already been described above and is a consistent identifier across time and shared across resources. The choices of Swiss-Prot and TrEMBL in some combination was discussed above, and Proteome can be “proteome”, “reference proteome” or “pan-proteome”. The proteome identifier (‘UP’ followed by 9 digits) is conserved across releases, and release information should also be included. A confusing issue to newcomers is what the term “release” means. This is a year_month format (e.g., 2022_01), but it is not the date a FASTA file was downloaded or created, nor does it imply there are monthly updates. This release “date” is a traceable release identifier that is listed on UniProt’s website. Including all this information ensures that the exact provenance of a FASTA file is known and allows the FASTA file to be regenerated.</p>
<h5 id="refseq">RefSeq</h5>
<p>NCBI is a clearing house of numerous types of data and databases.
Specific to protein sequence collections, NCBI Reference Sequence Database (RefSeq) provides annotated genomes across the Tree of Life.
The newly developed NCBI Datasets portal <span class="citation" data-cites="uPHDl2Vm">[<a href="#ref-uPHDl2Vm" role="doc-biblioref">175</a>]</span> is the preferred method for accessing the myriad of NCBI data products, though protein sequence collections can also be retrieved from RefSeq directly<span class="citation" data-cites="tg8zlLqI xpZWsuGI">[<a href="#ref-tg8zlLqI" role="doc-biblioref">176</a>,<a href="#ref-xpZWsuGI" role="doc-biblioref">177</a>]</span>.
Like UniProt described above, most of the additional functionality and information available through NCBI Datasets and RefSeq will not be described here, although the Eukaryotic RefSeq annotation dashboard <span class="citation" data-cites="BfmLo0uT">[<a href="#ref-BfmLo0uT" role="doc-biblioref">178</a>]</span> is a noteworthy resource to monitor the progress of new or re-annotations.
We recommend exploring the resources available from NCBI <span class="citation" data-cites="fWjGnKWy">[<a href="#ref-fWjGnKWy" role="doc-biblioref">179</a>]</span>, utilizing their tutorials and help requests.</p>
<p>RefSeq is akin to the “proteome” sequence collection from UniProtKB, where a release is based on a single genome assembly.
If a more complete genome assembly is deposited or additional secondary evidence (e.g., RNA sequencing) is deposited, RefSeq can update the annotation with a new annotation release.
Every annotation release will have an annotation report that contains information on the underlying genome assembly, the new genome annotation, secondary evidence used, and various statistics about what was updated.
The current annotation release is referred to as the “reference annotation”, but each annotation is numbered sequentially starting at 100 (the first release).
Certain species are on scheduled re-annotation, like human and mouse, while other species are updated as needed based on new data and community feedback (ex. release 100 of taxon 9704 was in 2018, but a more contiguous genome assembly resulted in re-annotation to release 101 in 2020).
This general process for new and existing species is described in Heck and Neely <span class="citation" data-cites="17gB3bs6x">[<a href="#ref-17gB3bs6x" role="doc-biblioref">180</a>]</span>.</p>
<p>Since RefSeq is genome assembly-centric, its protein sequence collections are retrieved for each species.
This contrasts with being able to use a higher-level taxon identifier like 40674 (Mammalia) in UniProt to retrieve a single FASTA.
To accomplish this same search in NCBI Datasets requires a Mammalia search, followed by browsing all 2083 genomes and then filtering the results to reference genomes with annotations, and those resulting 188 could be bulk downloaded, though this will still be 188 individual FASTA files.
It is possible to download a single FASTA from an upper-level taxon identifier using the NCBI Taxonomy Browser, though this service may be redundant with the new NCBI Datasets portal.
Given the constant development of NCBI Datasets, these functionalities may change, but the general RefSeq philosophy of single species FASTA should be kept in mind.
Likewise, when retrieving genome annotations there is no ability to specify canonical entries only, but it is possible to use computational tools to remove redundant entries (“remove_duplicates.py” from <span class="citation" data-cites="16BHXvDGS">[<a href="#ref-16BHXvDGS" role="doc-biblioref">173</a>]</span>).</p>
<p>Similar to the UniProtKB FASTA file naming suggestion, the following naming format is suggested for RefSeq protein sequence collection FASTA (the use of underscores or hyphens is not critical):
[common or scientific name]-[taxon id]-refseq-[release number]
example of a Equus caballus (horse) protein FASTA from RefSeq:
Equus_caballus-9796-refseq-103.fasta
The release number starts at 100 and is consecutively numbered.
Note, the human releases only recently began following this consecutive numbering for Release 110, and previously had a much longer number to be included (e.g., NCBI Release 109.20211119).
Also, in a few species (Human and Chinese hamster, currently), there is a reference and an alternate assembly, both with an available annotation.
In these cases, including the underlying assembly identifier would be needed. Note that when you retrieve the protein FASTA from NCBI it will include two more identifiers that aren’t required in the file name since it can be determined from the taxon identifier and release number.
These are the genome assembly used (this is generated by the depositor and follows no naming scheme) and the RefSeq identifier (GCF followed by a number string).
These aren’t essential for FASTA naming, but are for comparing between UniProt, RefSeq and Ensembl when the same underlying assembly is used (or not, indicating how up to date one is versus the other).</p>
<h5 id="ensembl">Ensembl</h5>
<p>There are two main web portals for Ensembl sequence collections: the Ensembl genome browser <span class="citation" data-cites="69obv795">[<a href="#ref-69obv795" role="doc-biblioref">181</a>]</span> has vertebrate organisms and the Ensemble Genome project <span class="citation" data-cites="1EGyqGdLr">[<a href="#ref-1EGyqGdLr" role="doc-biblioref">182</a>]</span> has specific web portals for different non-vertebrate branches of the Tree of Life.
This contrasts with NCBI and UniProt where all branches are centrally available.
Recently, Ensembl has created a new portal “Rapid Release” focusing on quickly making annotations available (replacing the “Pre-Ensemble” portal), albeit without the full functionality of the primary Ensembl resources.
Overall, Ensembl provides diverse comparative and genomic tools that should be explored, but, specific to this discussion, they provide species-specific genome annotation products similar to RefSeq.</p>
<p>To retrieve a protein sequence collection from Ensemble at any of the portals, a species can be searched using a name, which will then have taxon identifier displayed (but searching by identifier is not readily apparent).
From the results you can select your species and follow links for genome annotation.
Caution should be used when browsing the annotation products since the protein coding sequence (abbreviated “cds”) annotations are nucleic acid sequences (a useable via 3-frame translation if using certain software), while actual translated peptide sequences are in the “pep” folders.
The pep folders contain file names with “ab initio” and “all” in the FASTA file names (file extensions are “fa” for FASTA and “gz” indicating gzip compression algorithm), while there may only be one pep product for certain species in the “Rapid Release” portal.
The “ab initio” FASTA files contain mostly predicted gene products.
The “all” FASTA files are the usable protein sequence collections.
Ensembl FASTA files usually have some protein sequence redundancy.</p>
<p>Ensembl provides a release number for all the databases within each portal.
Similar to the UniProt file naming suggestion, the following naming format is suggested for Ensembl protein sequence collection FASTA (the use of underscores or hyphens is not critical):</p>
<p>[common or scientific name]-[taxon id]-ensembl-[abinitio/all]-[rapid]-[release number]</p>
<p>example of a Sus scrofa (pig) protein FASTA from Ensembl:</p>
<p>Pig-9823-ensembl-all-106.fasta</p>
<p>Similar to the FASTA download from RefSeq, the downloaded file name can include additional identifying information related to the underlying genome assembly.
Again, this is not required for labeling, but is useful to easily compare assembly versions.</p>
<p>Since much of the data from Ensembl is also regularly processed into UniProt, using UniProt sequence collections instead may be preferred.
That said, they are not on the same release schedule nor will the FASTA files contain the same proteins.
Ensembl sequences still must go through the established protein sequence pipeline at UniProt to remove redundancy and conform to UniProt accession and FASTA header formats.
Moreover, the gene-centric and comparative tools built into Ensembl may be more experimentally appropriate and using an Ensembl protein sequence collection can better leverage those tools.</p>
<h4 id="other-resources">Other resources</h4>
<p>There are other locations of protein sequence collections, and these will likewise have different FASTA file formatting; sequences may have unusual characters, and formats of accessions and FASTA header lines may need to be reformatted to be compatible with search software.
These alternatives include institutes like the Joint Genome Institute’s microbial genome clearing house, species-specific community resource (e.g., PomBase, FlyBase, WormBase, TryTrypDB, etc.), and one-off websites tenuously hosting in-house annotations.
It is preferred to use protein sequence collection from the main three sources described here, since provenance can be tracked, and versions maintained.
It is beyond the scope of this discussion to address other genome annotation resources, how they are versioned, or the best way to describe FASTA files retrieved from those sources.
In these cases, defaulting to the minimum requirements of listing number of entries and supplying the FASTA along with data are necessary.</p>
<h4 id="contaminants">Contaminants</h4>
<p>Samples are rarely comprised of only proteins from the species of interest.
There can be protein contamination during sample collection or processing.
This may include proteins from human skin, wool from clothing, particles from latex, or even porcine trypsin itself, all of which contain proteins that can be digested along with the intended sample and analyzed in the mass spectrometer.
Avoiding unwanted matching of mass spectra originating from contaminant proteins to the cellular proteins due to sequence similarities is important to the identification and quantitation of as many cellular proteins as possible.
To avoid random matching, repositories of supplementary sequences for contaminant proteins have been added to a reference database for MS data searches.
Appending a contaminants database to the reference database allows the identification of peptides that are not exclusive to one species.
Peptides that are exclusive to the organism of interest are used to calculate abundance to avoid inflated quantitative results due to potential contaminant peptides.</p>
<p>As early as 2004, The Global Proteome Machine was providing a protein sequence collection of these common Repository of Adventitious Proteins (cRAP), while another contaminant list was published in 2008 <span class="citation" data-cites="OeXy2hrh">[<a href="#ref-OeXy2hrh" role="doc-biblioref">183</a>]</span>.
The current cRAP version (v1.0) was described in 2012 <span class="citation" data-cites="jwgBjKOg">[<a href="#ref-jwgBjKOg" role="doc-biblioref">184</a>]</span> and is still widely in use today.
cRAP is the contaminant protein list used in nearly all modern database searching software, though the documentation, versioning or updating of many of these “built-in” contaminant sequence collections is difficult to follow.
There is also another contaminant sequence collection distributed with MaxQuant.
Together, the cRAP and MaxQuant contaminant protein sequence collections are found in some form across most software, including MetaMorpheus and Philosopher (available in FragPipe) <span class="citation" data-cites="WdwXXrRx">[<a href="#ref-WdwXXrRx" role="doc-biblioref">185</a>]</span>.
This list of known frequently contaminating proteins can either be automatically included by the software or can be retrieved as a FASTA to be used along with the primary search FASTA(s).
Recently the Hao Lab has revisited these common contaminant sequences in an effort to update the protein sequences, test their utility on experimental data, and add or remove entries <span class="citation" data-cites="m8bjbzJ5">[<a href="#ref-m8bjbzJ5" role="doc-biblioref">186</a>]</span>.</p>
<p>In addition to these environmentally unintended contaminants, there are known contaminants that also have available protein sequence collections (or can be generated using the steps above) and should be included in the search space.
These can include the media cells were grown in (e.g., fetal bovine serum <span class="citation" data-cites="CnwgtVyY 1CIFGLtYQ">[<a href="#ref-CnwgtVyY" role="doc-biblioref">187</a>,<a href="#ref-1CIFGLtYQ" role="doc-biblioref">188</a>]</span>, food fed to cells/animals (e.g., Caenorhabditis elegans grown on Escherichia coli) or known non-specific binders in affinity purification (i.e., CRAPome <span class="citation" data-cites="wUllSafI">[<a href="#ref-wUllSafI" role="doc-biblioref">189</a>]</span>).
The common Repository of Fetal Bovine Serum Proteins (cRFP)<span class="citation" data-cites="EyM6Y0Xm">[<a href="#ref-EyM6Y0Xm" role="doc-biblioref">190</a>]</span> are protein lists of common protein contaminants and fetal serum bovine sequences used to reduced the number of falsely identified proteins in cell culture experiments.
Cells washed or cultured in contaminant free media before harvest or the collection of secreted proteins depletes most high abundance contaminant proteins but the sequence similarity between contaminant and secreted proteins can cause false identifications and overestimation of the true protein abundance leading to wasted resources and time on validating false leads.
As emphasized throughout this section, accurately defining the search space is essential for accurate results and, especially in the case of contaminants, requires knowledge of the experiment and sample processing to adequately define possible background proteins.</p>
<h3 id="choosing-the-right-database">Choosing the right database</h3>
<!-- add greater exploration of how to choose a database -->
<p>Proteomics data analysis requires carefully matching the search space (defined by the database choice) with the expected proteins.
A properly chosen database will minimize false positives and false negatives.
Choosing a database that is too large will increase the number of false positives, or decoy hits, which in turn will reduce the total number of identifiable proteins.
For this reason it is ill advised to search against all possible protein sequences ever predicted from any genomic sequence.
On the other hand, choosing a database that is too small may increase false negatives, or missed protein identifications, because in order for a protein to be identified it must be present in the database.
Thus, proteomics practitioners must do their best to predict the proteins that might be in their sample before they analyze their data.</p>
<p>Proteomics data analysis requires carefully aligning the search space with the expected proteome and the statistical approach of the search algorithm.
Search algorithms can self-correct when a database is overly large such that higher identity thresholds are required for identification to minimize false positives (e.g., Mascot), while smaller experiment-specific search spaces (also referred to as “subsets”) can have unintended effects on false positives if not managed appropriately <span class="citation" data-cites="qgZxt4hu 12o52Fhcj Lovg2liS">[<a href="#ref-qgZxt4hu" role="doc-biblioref">191</a>,<a href="#ref-12o52Fhcj" role="doc-biblioref">192</a>,<a href="#ref-Lovg2liS" role="doc-biblioref">193</a>]</span> or may even improve protein identifications <span class="citation" data-cites="nMWQwf9T">[<a href="#ref-nMWQwf9T" role="doc-biblioref">194</a>]</span>.
Whether to employ a search space that is sample-specific (i.e., subset), species-specific (with only canonical proteins, described below), exhaustive species-specific (including all isoforms), or even larger clade-level protein sequence set (e.g., the over 14 million protein sequences associated with Fungi, taxon identifier 4751) is a complex issue that is experiment and software dependent.
Moreover, in cases where no species-specific protein sequence collection exists, homology-based searching can be used (as described in <span class="citation" data-cites="17gB3bs6x">[<a href="#ref-17gB3bs6x" role="doc-biblioref">180</a>]</span>).
In each of these cases, proteomics practitioners must understand their specific experimental sample and search algorithm in order to know how to best define the search space, which is essential to yielding accurate results. See more discussion of database choice in the following section.</p>
<h2 class="page_break_before" id="biological-interpretation">Biological Interpretation</h2>
<ol type="1">
<li>term enrichment analysis (KEGG, GO)</li>
<li>network analysis methods</li>
<li>structure analysis</li>
<li>isoform analysis</li>
<li>follow-up experiments</li>
</ol>
<h2 class="page_break_before" id="methods-for-protein-or-peptide-fractionation">Methods for protein or peptide fractionation</h2>
<p>Protein fractionation
* SDS-PAGE</p>
<p>Peptide fractionation
* bRP</p>
<h2 class="page_break_before" id="tandem-mass-spectrometry-and-peptide-fragmentation">Tandem Mass Spectrometry and Peptide Fragmentation</h2>
<h3 id="tandem-mass-spectrometry">Tandem Mass Spectrometry</h3>
<p>Why do we need tandem mass spectra data for proteomics?
As mass accuracy and resolution increase, the number of potential peptide matches decrease.
Still, there are many potential matches to a human peptide with a 5 ppm mass tolerance window (citation).
Additional information is required to uniquely assign a signal to a peptide.
Early approaches described use of accurate mass and time data to uniquely identify peptides <span class="citation" data-cites="JmTKP2r8">[<a href="#ref-JmTKP2r8" role="doc-biblioref">195</a>]</span>.
Most current research, however, has pursued using the pattern of peptide fragment masses helps narrow down the number of potential peptides.</p>
<h3 id="peptide-fragmentation">Peptide Fragmentation</h3>
<p><em>how it works</em>
Modern proteomics utilizes hybrid mass spectrometers, where hybrid means there are multiple mass analyzers working in harmony.
Most hybrid mass spectrometers combine a quadrople for mass selection before a high resolution analyzer, usually a TOF or an orbitrap.
In these hybrid mass spectrometers, mass selection happens using the first quadrupole to isolate a small mass range around that mass (usually less than 1 thompson).
The selected ions then pass to the collision cell where fragmentation happens.</p>
<!-- linear ion traps don't need a quadrupole first -->
<ol type="1">
<li>Nomenclature of backbone fragments (abc, xyz, internal, losses).</li>
<li>Mechanisms of peptide fragmentation, collision based (CID and HCD, PQD?) versus electron based (ECD and ETD).</li>
</ol>
<h3 id="basic-spectral-interpretation">Basic Spectral Interpretation</h3>
<p>Segway into the section on raw data analysis.</p>
<h2 id="experiment-design">Experiment Design</h2>
<p>This section should discuss trade offs and balancing them to design an experiment.
1. constraints: Each experiment will have different constraints, which may include the number of samples needed for analysis, or desire to quantify a specific subset of proteins within a sample.
2. sample size
3. statistics
4. costs</p>
<h2 class="page_break_before" id="acknowledgements">Acknowledgements</h2>
<p>The authors thank Phil Wilmarth for helpful input.
Identification of certain commercial equipment, instruments, software, or materials does not imply recommendation or endorsement by the National Institute of Standards and Technology, nor does it imply that the products identified are necessarily the best available for the purpose.</p>
<h2 class="page_break_before" id="references">References</h2>
<!-- Explicitly insert bibliography here -->
<div id="refs" class="references csl-bib-body" role="doc-bibliography">
<div id="ref-4Ajjx5fo" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1. </div><div class="csl-right-inline"><strong>Electrospray Ionization for Mass Spectrometry of Large Biomolecules</strong> <div class="csl-block">John B Fenn, Matthias Mann, Chin Kai Meng, Shek Fu Wong, Craig M Whitehouse</div> <em>Science</em> (1989-10-06) <a href="https://doi.org/cq2q43">https://doi.org/cq2q43</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1126/science.2675315">10.1126/science.2675315</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/2675315">2675315</a></div></div>
</div>
<div id="ref-15Ye4Za7Y" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">2. </div><div class="csl-right-inline"><strong>Protein and polymer analyses up tom/z 100 000 by laser ionization time-of-flight mass spectrometry</strong> <div class="csl-block">Koichi Tanaka, Hiroaki Waki, Yutaka Ido, Satoshi Akita, Yoshikazu Yoshida, Tamio Yoshida, T Matsuo</div> <em>Rapid Communications in Mass Spectrometry</em> (1988-08) <a href="https://doi.org/ffbwwr">https://doi.org/ffbwwr</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1002/rcm.1290020802">10.1002/rcm.1290020802</a></div></div>
</div>
<div id="ref-pG7zq91Q" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">3. </div><div class="csl-right-inline"><strong>An approach to correlate tandem mass spectral data of peptides with amino acid sequences in a protein database.</strong> <div class="csl-block">JK Eng, AL McCormack, JR Yates</div> <em>Journal of the American Society for Mass Spectrometry</em> (1994-11) <a href="https://www.ncbi.nlm.nih.gov/pubmed/24226387">https://www.ncbi.nlm.nih.gov/pubmed/24226387</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1016/1044-0305(94)80016-2">10.1016/1044-0305(94)80016-2</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/24226387">24226387</a></div></div>
</div>
<div id="ref-ujahSo3G" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">4. </div><div class="csl-right-inline"><strong>An Automated Multidimensional Protein Identification Technology for Shotgun Proteomics</strong> <div class="csl-block">Dirk A Wolters, Michael P Washburn, John R Yates</div> <em>Analytical Chemistry</em> (2001-10-25) <a href="https://doi.org/bn4kq6">https://doi.org/bn4kq6</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1021/ac010617e">10.1021/ac010617e</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/11774908">11774908</a></div></div>
</div>
<div id="ref-mw5jxASm" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">5. </div><div class="csl-right-inline"><strong>A Statistical Model for Identifying Proteins by Tandem Mass Spectrometry</strong> <div class="csl-block">Alexey I Nesvizhskii, Andrew Keller, Eugene Kolker, Ruedi Aebersold</div> <em>Analytical Chemistry</em> (2003-07-15) <a href="https://doi.org/b2xv45">https://doi.org/b2xv45</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1021/ac0341261">10.1021/ac0341261</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/14632076">14632076</a></div></div>
</div>
<div id="ref-y9Bki24q" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">6. </div><div class="csl-right-inline"><strong>Target-decoy search strategy for increased confidence in large-scale protein identifications by mass spectrometry</strong> <div class="csl-block">Joshua E Elias, Steven P Gygi</div> <em>Nature Methods</em> (2007-03) <a href="https://doi.org/djz7fz">https://doi.org/djz7fz</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1038/nmeth1019">https://doi.org/10.1038/nmeth1019</a></div></div>
</div>
<div id="ref-16QSCuG38" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">7. </div><div class="csl-right-inline"><strong>Mass-spectrometric exploration of proteome structure and function</strong> <div class="csl-block">Ruedi Aebersold, Matthias Mann</div> <em>Nature</em> (2016-09) <a href="https://doi.org/f83zqm">https://doi.org/f83zqm</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1038/nature19949">10.1038/nature19949</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/27629641">27629641</a></div></div>
</div>
<div id="ref-a6Z8FNxW" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">8. </div><div class="csl-right-inline"><strong>High-throughput quantitative top-down proteomics</strong> <div class="csl-block">Kellye A Cupp-Sutton, Si Wu</div> <em>Molecular Omics</em> (2020) <a href="https://doi.org/gnx98p">https://doi.org/gnx98p</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1039/c9mo00154a">10.1039/c9mo00154a</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/31932818">31932818</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7529119">PMC7529119</a></div></div>
</div>
<div id="ref-h00Trxkt" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">9. </div><div class="csl-right-inline"><strong>Proteoforms as the next proteomics currency</strong> <div class="csl-block">Lloyd M Smith, Neil L Kelleher</div> <em>Science</em> (2018-03-09) <a href="https://doi.org/gn6p4x">https://doi.org/gn6p4x</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1126/science.aat1884">10.1126/science.aat1884</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/29590032">29590032</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5944612">PMC5944612</a></div></div>
</div>
<div id="ref-emz5eXnO" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">10. </div><div class="csl-right-inline"><strong>Guide to protein purification</strong> <div class="csl-block">Richard R Burgess, Murray P Deutscher</div> <em>Elsevier/Academic Press</em> (2009) <div class="csl-block">ISBN: 9780123745361</div></div>
</div>
<div id="ref-LvSVXQeu" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">11. </div><div class="csl-right-inline"><strong>The protein protocols handbook</strong> <div class="csl-block">John M Walker (editor)</div> <em>Humana Press</em> (2009) <div class="csl-block">ISBN: 9781597451987</div></div>
</div>
<div id="ref-YFivV9n5" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">12. </div><div class="csl-right-inline"><strong>Strategies and considerations for protein purifications.</strong> <div class="csl-block">Stuart Linn</div> <em>Methods in enzymology</em> (2009) <a href="https://www.ncbi.nlm.nih.gov/pubmed/19892162">https://www.ncbi.nlm.nih.gov/pubmed/19892162</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1016/s0076-6879(09)63002-0">10.1016/s0076-6879(09)63002-0</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/19892162">19892162</a></div></div>
</div>
<div id="ref-OZvOVLht" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">13. </div><div class="csl-right-inline"><strong>Effective interactions between chaotropic agents and proteins.</strong> <div class="csl-block">Giovanni Salvi, Paolo De Los Rios, Michele Vendruscolo</div> <em>Proteins</em> (2005-11-15) <a href="https://www.ncbi.nlm.nih.gov/pubmed/16152629">https://www.ncbi.nlm.nih.gov/pubmed/16152629</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1002/prot.20626">10.1002/prot.20626</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/16152629">16152629</a></div></div>
</div>
<div id="ref-bpTXhTR5" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">14. </div><div class="csl-right-inline"><strong>A quantitative study of the effects of chaotropic agents, surfactants, and solvents on the digestion efficiency of human plasma proteins by trypsin.</strong> <div class="csl-block">Jennifer L Proc, Michael A Kuzyk, Darryl B Hardie, Juncong Yang, Derek S Smith, Angela M Jackson, Carol E Parker, Christoph H Borchers</div> <em>Journal of proteome research</em> (2010-10-01) <a href="https://www.ncbi.nlm.nih.gov/pubmed/20722421">https://www.ncbi.nlm.nih.gov/pubmed/20722421</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1021/pr100656u">10.1021/pr100656u</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/20722421">20722421</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2996461">PMC2996461</a></div></div>
</div>
<div id="ref-Igj2yRI0" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">15. </div><div class="csl-right-inline"><strong>Best practices and benchmarks for intact protein analysis for top-down mass spectrometry.</strong> <div class="csl-block">Daniel P Donnelly, Catherine M Rawlins, Caroline J DeHart, Luca Fornelli, Luis F Schachner, Ziqing Lin, Jennifer L Lippens, Krishna C Aluri, Richa Sarin, Bifan Chen, … Jeffrey N Agar</div> <em>Nature methods</em> (2019-06-27) <a href="https://www.ncbi.nlm.nih.gov/pubmed/31249407">https://www.ncbi.nlm.nih.gov/pubmed/31249407</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1038/s41592-019-0457-0">10.1038/s41592-019-0457-0</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/31249407">31249407</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6719561">PMC6719561</a></div></div>
</div>
<div id="ref-B8hLhFif" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">16. </div><div class="csl-right-inline"><strong>Inhibition of protein carbamylation in urea solution using ammonium-containing buffers.</strong> <div class="csl-block">Shisheng Sun, Jian-Ying Zhou, Weiming Yang, Hui Zhang</div> <em>Analytical biochemistry</em> (2013-10-23) <a href="https://www.ncbi.nlm.nih.gov/pubmed/24161613">https://www.ncbi.nlm.nih.gov/pubmed/24161613</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1016/j.ab.2013.10.024">10.1016/j.ab.2013.10.024</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/24161613">24161613</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4072244">PMC4072244</a></div></div>
</div>
<div id="ref-G9HWYdpG" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">17. </div><div class="csl-right-inline"><strong>Integrated analysis of gene expression, CpG island methylation, and gene copy number in breast cancer cells by deep sequencing.</strong> <div class="csl-block">Zhifu Sun, Yan W Asmann, Krishna R Kalari, Brian Bot, Jeanette E Eckel-Passow, Tiffany R Baker, Jennifer M Carr, Irina Khrebtukova, Shujun Luo, Lu Zhang, … EAubrey Thompson</div> <em>PloS one</em> (2011-02-25) <a href="https://www.ncbi.nlm.nih.gov/pubmed/21364760">https://www.ncbi.nlm.nih.gov/pubmed/21364760</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1371/journal.pone.0017490">10.1371/journal.pone.0017490</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/21364760">21364760</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3045451">PMC3045451</a></div></div>
</div>
<div id="ref-15gFWnWy1" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">18. </div><div class="csl-right-inline"><strong>A generic protein purification method for protein complex characterization and proteome exploration.</strong> <div class="csl-block">G Rigaut, A Shevchenko, B Rutz, M Wilm, M Mann, B Séraphin</div> <em>Nature biotechnology</em> (1999-10) <a href="https://www.ncbi.nlm.nih.gov/pubmed/10504710">https://www.ncbi.nlm.nih.gov/pubmed/10504710</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1038/13732">10.1038/13732</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/10504710">10504710</a></div></div>
</div>
<div id="ref-1FdkzkZPe" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">19. </div><div class="csl-right-inline"><strong>Photocleavable Surfactant-Enabled Extracellular Matrix Proteomics.</strong> <div class="csl-block">Samantha J Knott, Kyle A Brown, Harini Josyer, Austin Carr, David Inman, Song Jin, Andreas Friedl, Suzanne M Ponik, Ying Ge</div> <em>Analytical chemistry</em> (2020-11-24) <a href="https://www.ncbi.nlm.nih.gov/pubmed/33232116">https://www.ncbi.nlm.nih.gov/pubmed/33232116</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1021/acs.analchem.0c03104">10.1021/acs.analchem.0c03104</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/33232116">33232116</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7961849">PMC7961849</a></div></div>
</div>
<div id="ref-KbqdsNwY" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">20. </div><div class="csl-right-inline"><strong>Investigation of solubilization and digestion methods for microsomal membrane proteome analysis using data-independent LC-MSE.</strong> <div class="csl-block">Flaubert Mbeunkui, Michael B Goshe</div> <em>Proteomics</em> (2011-01-31) <a href="https://www.ncbi.nlm.nih.gov/pubmed/21280217">https://www.ncbi.nlm.nih.gov/pubmed/21280217</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1002/pmic.200900698">10.1002/pmic.200900698</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/21280217">21280217</a></div></div>
</div>
<div id="ref-1FX8gYsTy" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">21. </div><div class="csl-right-inline"><strong>Sodium laurate, a novel protease- and mass spectrometry-compatible detergent for mass spectrometry-based membrane proteomics.</strong> <div class="csl-block">Yong Lin, Linju Huo, Zhonghua Liu, Jianglin Li, Yi Liu, Quanze He, Xianchun Wang, Songping Liang</div> <em>PloS one</em> (2013-03-28) <a href="https://www.ncbi.nlm.nih.gov/pubmed/23555778">https://www.ncbi.nlm.nih.gov/pubmed/23555778</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1371/journal.pone.0059779">10.1371/journal.pone.0059779</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/23555778">23555778</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3610932">PMC3610932</a></div></div>
</div>
<div id="ref-7nXVfPN7" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">22. </div><div class="csl-right-inline"><strong>Evaluation of the application of sodium deoxycholate to proteomic analysis of rat hippocampal plasma membrane.</strong> <div class="csl-block">Jian Zhou, Tieyang Zhou, Rui Cao, Zhen Liu, Jianying Shen, Ping Chen, Xianchun Wang, Songping Liang</div> <em>Journal of proteome research</em> (2006-10) <a href="https://www.ncbi.nlm.nih.gov/pubmed/17022626">https://www.ncbi.nlm.nih.gov/pubmed/17022626</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1021/pr060112a">10.1021/pr060112a</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/17022626">17022626</a></div></div>
</div>
<div id="ref-EZCh7C0x" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">23. </div><div class="csl-right-inline"><strong>Identification of Poly(ethylene glycol) and Poly(ethylene glycol)-Based Detergents Using Peptide Search Engines.</strong> <div class="csl-block">Shiva Ahmadi, Dominic Winter</div> <em>Analytical chemistry</em> (2018-05-15) <a href="https://www.ncbi.nlm.nih.gov/pubmed/29726681">https://www.ncbi.nlm.nih.gov/pubmed/29726681</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1021/acs.analchem.8b00365">10.1021/acs.analchem.8b00365</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/29726681">29726681</a></div></div>
</div>
<div id="ref-EK9zCZWn" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">24. </div><div class="csl-right-inline"><strong>Quantitative assessment of in-solution digestion efficiency identifies optimal protocols for unbiased protein analysis.</strong> <div class="csl-block">Ileana R León, Veit Schwämmle, Ole N Jensen, Richard R Sprenger</div> <em>Molecular &amp; cellular proteomics : MCP</em> (2013-06-21) <a href="https://www.ncbi.nlm.nih.gov/pubmed/23792921">https://www.ncbi.nlm.nih.gov/pubmed/23792921</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1074/mcp.m112.025585">10.1074/mcp.m112.025585</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/23792921">23792921</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3790306">PMC3790306</a></div></div>
</div>
<div id="ref-Cj5ZGyu8" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">25. </div><div class="csl-right-inline"><strong>Proteomic challenges: sample preparation techniques for microgram-quantity protein analysis from biological samples.</strong> <div class="csl-block">Peter Feist, Amanda B Hummon</div> <em>International journal of molecular sciences</em> (2015-02-05) <a href="https://www.ncbi.nlm.nih.gov/pubmed/25664860">https://www.ncbi.nlm.nih.gov/pubmed/25664860</a> <div class="csl-block">DOI: <a href="https://doi.org/10.3390/ijms16023537">10.3390/ijms16023537</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/25664860">25664860</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4346912">PMC4346912</a></div></div>
</div>
<div id="ref-195QiKJpv" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">26. </div><div class="csl-right-inline"><strong>Mechanical forces used for cell fractionation can create hybrid membrane vesicles.</strong> <div class="csl-block">Izhar Salomon, Hans Janssen, Jacques Neefjes</div> <em>International journal of biological sciences</em> (2010-10-20) <a href="https://www.ncbi.nlm.nih.gov/pubmed/21060726">https://www.ncbi.nlm.nih.gov/pubmed/21060726</a> <div class="csl-block">DOI: <a href="https://doi.org/10.7150/ijbs.6.649">10.7150/ijbs.6.649</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/21060726">21060726</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2974167">PMC2974167</a></div></div>
</div>
<div id="ref-4EGQHHdM" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">27. </div><div class="csl-right-inline"><strong>Lyophilization and homogenization of biological samples improves reproducibility and reduces standard deviation in molecular biology techniques.</strong> <div class="csl-block">Agnes Molnar, Tamas Lakat, Adam Hosszu, Beata Szebeni, Anna Balogh, Laszlo Orfi, Attila J Szabo, Andrea Fekete, Judit Hodrea</div> <em>Amino acids</em> (2021-05-17) <a href="https://www.ncbi.nlm.nih.gov/pubmed/34002278">https://www.ncbi.nlm.nih.gov/pubmed/34002278</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1007/s00726-021-02994-w">10.1007/s00726-021-02994-w</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/34002278">34002278</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8128086">PMC8128086</a></div></div>
</div>
<div id="ref-wXcRYG3x" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">28. </div><div class="csl-right-inline"><strong>Assays for determination of protein concentration.</strong> <div class="csl-block">Bradley JSC Olson, John Markwell</div> <em>Current protocols in protein science</em> (2007-05) <a href="https://www.ncbi.nlm.nih.gov/pubmed/18429326">https://www.ncbi.nlm.nih.gov/pubmed/18429326</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1002/0471140864.ps0304s48">10.1002/0471140864.ps0304s48</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/18429326">18429326</a></div></div>
</div>
<div id="ref-DRVtai1B" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">29. </div><div class="csl-right-inline"><strong>Development and characterization of the NanoOrange protein quantitation assay: a fluorescence-based assay of proteins in solution.</strong> <div class="csl-block">Laurie J Jones, Richard P Haugland, Victoria L Singer</div> <em>BioTechniques</em> (2003-04) <a href="https://www.ncbi.nlm.nih.gov/pubmed/12703310">https://www.ncbi.nlm.nih.gov/pubmed/12703310</a> <div class="csl-block">DOI: <a href="https://doi.org/10.2144/03344pt03">10.2144/03344pt03</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/12703310">12703310</a></div></div>
</div>
<div id="ref-7YHVbaDH" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">30. </div><div class="csl-right-inline"><strong>Fast and Sensitive Total Protein and Peptide Assays for Proteomic Analysis</strong> <div class="csl-block">Jacek R Wiśniewski, Fabienne Z Gaugaz</div> <em>Analytical Chemistry</em> (2015-04-09) <a href="https://doi.org/f3nsk2">https://doi.org/f3nsk2</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1021/ac504689z">10.1021/ac504689z</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/25837572">25837572</a></div></div>
</div>
<div id="ref-qFKCAO2X" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">31. </div><div class="csl-right-inline"><strong>Evaluation of colorimetric assays for analyzing reductively methylated proteins: Biases and mechanistic insights.</strong> <div class="csl-block">Pamlea N Brady, Megan A Macnaughtan</div> <em>Analytical biochemistry</em> (2015-09-03) <a href="https://www.ncbi.nlm.nih.gov/pubmed/26342307">https://www.ncbi.nlm.nih.gov/pubmed/26342307</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1016/j.ab.2015.08.027">10.1016/j.ab.2015.08.027</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/26342307">26342307</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4631703">PMC4631703</a></div></div>
</div>
<div id="ref-jQLWiiId" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">32. </div><div class="csl-right-inline"><strong>Quantification of Intrinsically Disordered Proteins: A Problem Not Fully Appreciated.</strong> <div class="csl-block">Sara Contreras-Martos, Hung H Nguyen, Phuong N Nguyen, Nevena Hristozova, Mauricio Macossay-Castillo, Denes Kovacs, Angela Bekesi, Jesper S Oemig, Dominique Maes, Kris Pauwels, … Pierre Lebrun</div> <em>Frontiers in molecular biosciences</em> (2018-09-04) <a href="https://www.ncbi.nlm.nih.gov/pubmed/30234128">https://www.ncbi.nlm.nih.gov/pubmed/30234128</a> <div class="csl-block">DOI: <a href="https://doi.org/10.3389/fmolb.2018.00083">10.3389/fmolb.2018.00083</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/30234128">30234128</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6131523">PMC6131523</a></div></div>
</div>
<div id="ref-2cCnZ6ft" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">33. </div><div class="csl-right-inline"><strong>A simplified procedure for the reduction and alkylation of cysteine residues in proteins prior to proteolytic digestion and mass spectral analysis.</strong> <div class="csl-block">John E Hale, Jon P Butler, Valentina Gelfanova, Jin-Sam You, Michael D Knierman</div> <em>Analytical biochemistry</em> (2004-10-01) <a href="https://www.ncbi.nlm.nih.gov/pubmed/15351294">https://www.ncbi.nlm.nih.gov/pubmed/15351294</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1016/j.ab.2004.04.013">10.1016/j.ab.2004.04.013</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/15351294">15351294</a></div></div>
</div>
<div id="ref-9vICnnYM" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">34. </div><div class="csl-right-inline"><strong>Systematic Evaluation of Protein Reduction and Alkylation Reveals Massive Unspecific Side Effects by Iodine-containing Reagents.</strong> <div class="csl-block">Torsten Müller, Dominic Winter</div> <em>Molecular &amp; cellular proteomics : MCP</em> (2017-05-24) <a href="https://www.ncbi.nlm.nih.gov/pubmed/28539326">https://www.ncbi.nlm.nih.gov/pubmed/28539326</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1074/mcp.m116.064048">10.1074/mcp.m116.064048</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/28539326">28539326</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5500753">PMC5500753</a></div></div>
</div>
<div id="ref-GkFAexwC" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">35. </div><div class="csl-right-inline"><strong>Discovery of a Redox Thiol Switch: Implications for Cellular Energy Metabolism.</strong> <div class="csl-block">Xing-Huang Gao, Ling Li, Marc Parisien, Jing Wu, Ilya Bederman, Zhaofeng Gao, Dawid Krokowski, Steven M Chirieleison, Derek Abbott, Benlian Wang, … Maria Hatzoglou</div> <em>Molecular &amp; cellular proteomics : MCP</em> (2020-03-04) <a href="https://www.ncbi.nlm.nih.gov/pubmed/32132231">https://www.ncbi.nlm.nih.gov/pubmed/32132231</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1074/mcp.ra119.001910">10.1074/mcp.ra119.001910</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32132231">32132231</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7196587">PMC7196587</a></div></div>
</div>
<div id="ref-Z6nk0TFI" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">36. </div><div class="csl-right-inline"><strong>Residue Modification and Mass Spectrometry for the Investigation of Structural and Metalation Properties of Metallothionein and Cysteine-Rich Proteins.</strong> <div class="csl-block">Gordon W Irvine, Martin J Stillman</div> <em>International journal of molecular sciences</em> (2017-04-26) <a href="https://www.ncbi.nlm.nih.gov/pubmed/28445428">https://www.ncbi.nlm.nih.gov/pubmed/28445428</a> <div class="csl-block">DOI: <a href="https://doi.org/10.3390/ijms18050913">10.3390/ijms18050913</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/28445428">28445428</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5454826">PMC5454826</a></div></div>
</div>
<div id="ref-s4OLOlUG" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">37. </div><div class="csl-right-inline"><strong>Chasing cysteine oxidative modifications: proteomic tools for characterizing cysteine redox status.</strong> <div class="csl-block">Christopher I Murray, Jennifer E Van Eyk</div> <em>Circulation. Cardiovascular genetics</em> (2012-10-01) <a href="https://www.ncbi.nlm.nih.gov/pubmed/23074338">https://www.ncbi.nlm.nih.gov/pubmed/23074338</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1161/circgenetics.111.961425">10.1161/circgenetics.111.961425</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/23074338">23074338</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3500588">PMC3500588</a></div></div>
</div>
<div id="ref-xvenBhjQ" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">38. </div><div class="csl-right-inline"><strong>Comparison of protein precipitation methods for sample preparation prior to proteomic analysis.</strong> <div class="csl-block">Lei Jiang, Lin He, Michael Fountoulakis</div> <em>Journal of chromatography. A</em> (2004-01-16) <a href="https://www.ncbi.nlm.nih.gov/pubmed/14753699">https://www.ncbi.nlm.nih.gov/pubmed/14753699</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1016/j.chroma.2003.10.029">10.1016/j.chroma.2003.10.029</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/14753699">14753699</a></div></div>
</div>
<div id="ref-OMzPHtgd" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">39. </div><div class="csl-right-inline"><strong>Protein precipitation techniques.</strong> <div class="csl-block">Richard R Burgess</div> <em>Methods in enzymology</em> (2009) <a href="https://www.ncbi.nlm.nih.gov/pubmed/19892180">https://www.ncbi.nlm.nih.gov/pubmed/19892180</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1016/s0076-6879(09)63020-2">10.1016/s0076-6879(09)63020-2</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/19892180">19892180</a></div></div>
</div>
<div id="ref-Llj72YHx" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">40. </div><div class="csl-right-inline"><strong>Facile Preparation of Peptides for Mass Spectrometry Analysis in Bottom-Up Proteomics Workflows.</strong> <div class="csl-block">Melinda Wojtkiewicz, Linda Berg Luecke, Maia I Kelly, Rebekah L Gundry</div> <em>Current protocols</em> (2021-03) <a href="https://www.ncbi.nlm.nih.gov/pubmed/33750040">https://www.ncbi.nlm.nih.gov/pubmed/33750040</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1002/cpz1.85">10.1002/cpz1.85</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/33750040">33750040</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8011989">PMC8011989</a></div></div>
</div>
<div id="ref-rWTbDNu2" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">41. </div><div class="csl-right-inline"><strong>Universal sample preparation method for proteome analysis.</strong> <div class="csl-block">Jacek R Wiśniewski, Alexandre Zougman, Nagarjuna Nagaraj, Matthias Mann</div> <em>Nature methods</em> (2009-04-19) <a href="https://www.ncbi.nlm.nih.gov/pubmed/19377485">https://www.ncbi.nlm.nih.gov/pubmed/19377485</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1038/nmeth.1322">10.1038/nmeth.1322</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/19377485">19377485</a></div></div>
</div>
<div id="ref-bkON7ZR2" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">42. </div><div class="csl-right-inline"><strong>Filter-Aided Sample Preparation for Proteome Analysis.</strong> <div class="csl-block">Jacek R Wiśniewski</div> <em>Methods in molecular biology (Clifton, N.J.)</em> (2018) <a href="https://www.ncbi.nlm.nih.gov/pubmed/30259475">https://www.ncbi.nlm.nih.gov/pubmed/30259475</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1007/978-1-4939-8695-8_1">10.1007/978-1-4939-8695-8_1</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/30259475">30259475</a></div></div>
</div>
<div id="ref-29YpOcx3" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">43. </div><div class="csl-right-inline"><strong>[Group and type distribution of hemolytic streptococci isolated from clinical specimens--prevalence of group A type 3 isolates in 1985 in Toyama Prefecture].</strong> <div class="csl-block">H Kodama, N Tokuman, I Yasui, Y Gyobu, Y Kashiwagi</div> <em>Kansenshogaku zasshi. The Journal of the Japanese Association for Infectious Diseases</em> (1987-04) <a href="https://www.ncbi.nlm.nih.gov/pubmed/3117935">https://www.ncbi.nlm.nih.gov/pubmed/3117935</a> <div class="csl-block">DOI: <a href="https://doi.org/10.11150/kansenshogakuzasshi1970.61.482">10.11150/kansenshogakuzasshi1970.61.482</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/3117935">3117935</a></div></div>
</div>
<div id="ref-R9HFU5Rw" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">44. </div><div class="csl-right-inline"><strong>Evaluation of FASP, SP3, and iST Protocols for Proteomic Sample Preparation in the Low Microgram Range.</strong> <div class="csl-block">Malte Sielaff, Jörg Kuharev, Toszka Bohn, Jennifer Hahlbrock, Tobias Bopp, Stefan Tenzer, Ute Distler</div> <em>Journal of proteome research</em> (2017-10-11) <a href="https://www.ncbi.nlm.nih.gov/pubmed/28948796">https://www.ncbi.nlm.nih.gov/pubmed/28948796</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1021/acs.jproteome.7b00433">10.1021/acs.jproteome.7b00433</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/28948796">28948796</a></div></div>
</div>
<div id="ref-vpBg0CP7" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">45. </div><div class="csl-right-inline"><strong>Getting intimate with trypsin, the leading protease in proteomics</strong> <div class="csl-block">Elien Vandermarliere, Michael Mueller, Lennart Martens</div> <em>Mass Spectrometry Reviews</em> (2013-06-15) <a href="https://doi.org/gn64qb">https://doi.org/gn64qb</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1002/mas.21376">10.1002/mas.21376</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/23775586">23775586</a></div></div>
</div>
<div id="ref-16FpKYZCe" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">46. </div><div class="csl-right-inline"><strong>Value of using multiple proteases for large-scale mass spectrometry-based proteomics.</strong> <div class="csl-block">Danielle L Swaney, Craig D Wenger, Joshua J Coon</div> <em>Journal of proteome research</em> (2010-03-05) <a href="https://www.ncbi.nlm.nih.gov/pubmed/20113005">https://www.ncbi.nlm.nih.gov/pubmed/20113005</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1021/pr900863u">10.1021/pr900863u</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/20113005">20113005</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2833215">PMC2833215</a></div></div>
</div>
<div id="ref-lSFMVWD1" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">47. </div><div class="csl-right-inline"><strong>Proteomics beyond trypsin.</strong> <div class="csl-block">Liana Tsiatsiani, Albert JR Heck</div> <em>The FEBS journal</em> (2015-04-14) <a href="https://www.ncbi.nlm.nih.gov/pubmed/25823410">https://www.ncbi.nlm.nih.gov/pubmed/25823410</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1111/febs.13287">10.1111/febs.13287</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/25823410">25823410</a></div></div>
</div>
<div id="ref-10vHwy5gN" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">48. </div><div class="csl-right-inline"><div class="csl-block">Jesse G Meyer</div> <em>ISRN computational biology</em> (2014-04-22) <a href="https://www.ncbi.nlm.nih.gov/pubmed/30687733">https://www.ncbi.nlm.nih.gov/pubmed/30687733</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1155/2014/960902">10.1155/2014/960902</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/30687733">30687733</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6347401">PMC6347401</a></div></div>
</div>
<div id="ref-y9KaXde4" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">49. </div><div class="csl-right-inline"><strong>&lt;i&gt;In Silico&lt;/i&gt;Proteome Cleavage Reveals Iterative Digestion Strategy for High Sequence Coverage</strong> <div class="csl-block">Jesse G Meyer</div> <em>ISRN Computational Biology</em> (2014-04-22) <a href="https://doi.org/gb6s2r">https://doi.org/gb6s2r</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1155/2014/960902">10.1155/2014/960902</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/30687733">30687733</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6347401">PMC6347401</a></div></div>
</div>
<div id="ref-7aKTZT1i" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">50. </div><div class="csl-right-inline"><strong>Multiple enzymatic digestion for enhanced sequence coverage of proteins in complex proteomic mixtures using capillary LC with ion trap MS/MS.</strong> <div class="csl-block">Gargi Choudhary, Shiaw-Lin Wu, Paul Shieh, William S Hancock</div> <em>Journal of proteome research</em> <a href="https://www.ncbi.nlm.nih.gov/pubmed/12643544">https://www.ncbi.nlm.nih.gov/pubmed/12643544</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1021/pr025557n">10.1021/pr025557n</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/12643544">12643544</a></div></div>
</div>
<div id="ref-gXhdeFOF" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">51. </div><div class="csl-right-inline"><strong>Six alternative proteases for mass spectrometry-based proteomics beyond trypsin.</strong> <div class="csl-block">Piero Giansanti, Liana Tsiatsiani, Teck Yew Low, Albert JR Heck</div> <em>Nature protocols</em> (2016-04-28) <a href="https://www.ncbi.nlm.nih.gov/pubmed/27123950">https://www.ncbi.nlm.nih.gov/pubmed/27123950</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1038/nprot.2016.057">10.1038/nprot.2016.057</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/27123950">27123950</a></div></div>
</div>
<div id="ref-14bc70ELO" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">52. </div><div class="csl-right-inline"><strong>Combination of Proteogenomics with Peptide </strong> <div class="csl-block">B Blank-Landeshammer, I Teichert, R Märker, M Nowrousian, U Kück, A Sickmann</div> <em>mBio</em> (2019-10-15) <a href="https://www.ncbi.nlm.nih.gov/pubmed/31615963">https://www.ncbi.nlm.nih.gov/pubmed/31615963</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1128/mbio.02367-19">10.1128/mbio.02367-19</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/31615963">31615963</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6794485">PMC6794485</a></div></div>
</div>
<div id="ref-zmo81rNN" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">53. </div><div class="csl-right-inline"><strong>Precision </strong> <div class="csl-block">Hao Yang, Yan-Chang Li, Ming-Zhi Zhao, Fei-Lin Wu, Xi Wang, Wei-Di Xiao, Yi-Hao Wang, Jun-Ling Zhang, Fu-Qiang Wang, Feng Xu, … Ping Xu</div> <em>Molecular &amp; cellular proteomics : MCP</em> (2019-01-08) <a href="https://www.ncbi.nlm.nih.gov/pubmed/30622160">https://www.ncbi.nlm.nih.gov/pubmed/30622160</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1074/mcp.tir118.000918">10.1074/mcp.tir118.000918</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/30622160">30622160</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6442358">PMC6442358</a></div></div>
</div>
<div id="ref-IZw9sVjg" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">54. </div><div class="csl-right-inline"><strong>Mass spectrometry-assisted venom profiling of Hypnale hypnale found in the Western Ghats of India incorporating de novo sequencing approaches.</strong> <div class="csl-block">Muralidharan Vanuopadath, Nithin Sajeev, Athira Radhamony Murali, Nayana Sudish, Nithya Kangosseri, Ivy Rose Sebastian, Nidhi Dalpatraj Jain, Amit Pal, Dileepkumar Raveendran, Bipin Gopalakrishnan Nair, Sudarslal Sadasivan Nair</div> <em>International journal of biological macromolecules</em> (2018-07-07) <a href="https://www.ncbi.nlm.nih.gov/pubmed/29990557">https://www.ncbi.nlm.nih.gov/pubmed/29990557</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1016/j.ijbiomac.2018.07.016">10.1016/j.ijbiomac.2018.07.016</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/29990557">29990557</a></div></div>
</div>
<div id="ref-19KCAADV7" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">55. </div><div class="csl-right-inline"><strong>Venomics and antivenomics of Indian spectacled cobra (Naja naja) from the Western Ghats</strong> <div class="csl-block">Muralidharan Vanuopadath, Dileepkumar Raveendran, Bipin Gopalakrishnan Nair, Sudarslal Sadasivan Nair</div> <em>Acta Tropica</em> (2022-04) <a href="https://doi.org/gpbzf7">https://doi.org/gpbzf7</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1016/j.actatropica.2022.106324">10.1016/j.actatropica.2022.106324</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/35093326">35093326</a></div></div>
</div>
<div id="ref-7BuX2TBR" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">56. </div><div class="csl-right-inline"><strong>Sequencing-Grade &lt;i&gt;De novo&lt;/i&gt; Analysis of MS/MS Triplets (CID/HCD/ETD) From Overlapping Peptides</strong> <div class="csl-block">Adrian Guthals, Karl R Clauser, Ari M Frank, Nuno Bandeira</div> <em>Journal of Proteome Research</em> (2013-05-30) <a href="https://doi.org/f47kqd">https://doi.org/f47kqd</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1021/pr400173d">10.1021/pr400173d</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/23679345">23679345</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4591044">PMC4591044</a></div></div>
</div>
<div id="ref-1GLf6FYas" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">57. </div><div class="csl-right-inline"><strong>Multiple-Enzyme-Digestion Strategy Improves Accuracy and Sensitivity of Label- and Standard-Free Absolute Quantification to a Level That Is Achievable by Analysis with Stable Isotope-Labeled Standard Spiking.</strong> <div class="csl-block">Jacek R Wiśniewski, Christine Wegler, Per Artursson</div> <em>Journal of proteome research</em> (2018-10-30) <a href="https://www.ncbi.nlm.nih.gov/pubmed/30336047">https://www.ncbi.nlm.nih.gov/pubmed/30336047</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1021/acs.jproteome.8b00549">10.1021/acs.jproteome.8b00549</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/30336047">30336047</a></div></div>
</div>
<div id="ref-WubQ7yFP" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">58. </div><div class="csl-right-inline"><strong>Improved Protein Inference from Multiple Protease Bottom-Up Mass Spectrometry Data</strong> <div class="csl-block">Rachel M Miller, Robert J Millikin, Connor V Hoffmann, Stefan K Solntsev, Gloria M Sheynkman, Michael R Shortreed, Lloyd M Smith</div> <em>Journal of Proteome Research</em> (2019-08-04) <a href="https://doi.org/gpqmp2">https://doi.org/gpqmp2</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1021/acs.jproteome.9b00330">10.1021/acs.jproteome.9b00330</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/31378069">31378069</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6733628">PMC6733628</a></div></div>
</div>
<div id="ref-yeQvImXS" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">59. </div><div class="csl-right-inline"><strong>Expanding Proteome Coverage with Orthogonal-specificity α-Lytic Proteases</strong> <div class="csl-block">Jesse G Meyer, Sangtae Kim, David A Maltby, Majid Ghassemian, Nuno Bandeira, Elizabeth A Komives</div> <em>Molecular &amp;amp; Cellular Proteomics</em> (2014-03) <a href="https://doi.org/f5vgcg">https://doi.org/f5vgcg</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1074/mcp.m113.034710">10.1074/mcp.m113.034710</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/24425750">24425750</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3945911">PMC3945911</a></div></div>
</div>
<div id="ref-Ac0S3PrF" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">60. </div><div class="csl-right-inline"><strong>Multi-protease Approach for the Improved Identification and Molecular Characterization of Small Proteins and Short Open Reading Frame-Encoded Peptides</strong> <div class="csl-block">Philipp T Kaulich, Liam Cassidy, Jürgen Bartel, Ruth A Schmitz, Andreas Tholey</div> <em>Journal of Proteome Research</em> (2021-03-24) <a href="https://doi.org/gpqmpz">https://doi.org/gpqmpz</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1021/acs.jproteome.1c00115">10.1021/acs.jproteome.1c00115</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/33760615">33760615</a></div></div>
</div>
<div id="ref-P861T4mW" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">61. </div><div class="csl-right-inline"><strong>A Multiple Protease Strategy to Optimise the Shotgun Proteomics of Mature Medicinal Cannabis Buds</strong> <div class="csl-block">Delphine Vincent, Vilnis Ezernieks, Simone Rochfort, German Spangenberg</div> <em>International Journal of Molecular Sciences</em> (2019-11-11) <a href="https://doi.org/gpqmp3">https://doi.org/gpqmp3</a> <div class="csl-block">DOI: <a href="https://doi.org/10.3390/ijms20225630">10.3390/ijms20225630</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/31717952">31717952</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6888629">PMC6888629</a></div></div>
</div>
<div id="ref-14eCb7Jfc" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">62. </div><div class="csl-right-inline"><strong>Confetti: A Multiprotease Map of the HeLa Proteome for Comprehensive Proteomics</strong> <div class="csl-block">Xiaofeng Guo, David C Trudgian, Andrew Lemoff, Sivaramakrishna Yadavalli, Hamid Mirzaei</div> <em>Molecular &amp;amp; Cellular Proteomics</em> (2014-06) <a href="https://doi.org/f56bwx">https://doi.org/f56bwx</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1074/mcp.m113.035170">10.1074/mcp.m113.035170</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/24696503">24696503</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4047476">PMC4047476</a></div></div>
</div>
<div id="ref-18s9UXUGf" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">63. </div><div class="csl-right-inline"><strong>An Augmented Multiple-Protease-Based Human Phosphopeptide Atlas</strong> <div class="csl-block">Piero Giansanti, Thin Thin Aye, Henk van den Toorn, Mao Peng, Bas van Breukelen, Albert JR Heck</div> <em>Cell Reports</em> (2015-06) <a href="https://doi.org/gpqmpx">https://doi.org/gpqmpx</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1016/j.celrep.2015.05.029">10.1016/j.celrep.2015.05.029</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/26074081">26074081</a></div></div>
</div>
<div id="ref-ZgqlEERS" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">64. </div><div class="csl-right-inline"><strong>Use of endoproteinase Lys-C from Lysobacter enzymogenes in protein sequence analysis.</strong> <div class="csl-block">PA Jekel, WJ Weijer, JJ Beintema</div> <em>Analytical biochemistry</em> (1983-10-15) <a href="https://www.ncbi.nlm.nih.gov/pubmed/6359954">https://www.ncbi.nlm.nih.gov/pubmed/6359954</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1016/0003-2697(83)90308-1">10.1016/0003-2697(83)90308-1</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/6359954">6359954</a></div></div>
</div>
<div id="ref-nobcST81" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">65. </div><div class="csl-right-inline"><strong>Large-scale quantitative assessment of different in-solution protein digestion protocols reveals superior cleavage efficiency of tandem Lys-C/trypsin proteolysis over trypsin digestion.</strong> <div class="csl-block">Timo Glatter, Christina Ludwig, Erik Ahrné, Ruedi Aebersold, Albert JR Heck, Alexander Schmidt</div> <em>Journal of proteome research</em> (2012-10-16) <a href="https://www.ncbi.nlm.nih.gov/pubmed/23017020">https://www.ncbi.nlm.nih.gov/pubmed/23017020</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1021/pr300273g">10.1021/pr300273g</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/23017020">23017020</a></div></div>
</div>
<div id="ref-oDNy6K9V" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">66. </div><div class="csl-right-inline"><strong>The alpha-lytic protease gene of Lysobacter enzymogenes. The nucleotide sequence predicts a large prepro-peptide with homology to pro-peptides of other chymotrypsin-like enzymes.</strong> <div class="csl-block">DM Epstein, PC Wensink</div> <em>The Journal of biological chemistry</em> (1988-11-15) <a href="https://www.ncbi.nlm.nih.gov/pubmed/3053694">https://www.ncbi.nlm.nih.gov/pubmed/3053694</a> <div class="csl-block">PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/3053694">3053694</a></div></div>
</div>
<div id="ref-4iAess79" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">67. </div><div class="csl-right-inline"><strong>Site-specific identification and quantitation of endogenous SUMO modifications under native conditions.</strong> <div class="csl-block">Ryan J Lumpkin, Hongbo Gu, Yiying Zhu, Marilyn Leonard, Alla S Ahmad, Karl R Clauser, Jesse G Meyer, Eric J Bennett, Elizabeth A Komives</div> <em>Nature communications</em> (2017-10-27) <a href="https://www.ncbi.nlm.nih.gov/pubmed/29079793">https://www.ncbi.nlm.nih.gov/pubmed/29079793</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1038/s41467-017-01271-3">10.1038/s41467-017-01271-3</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/29079793">29079793</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5660086">PMC5660086</a></div></div>
</div>
<div id="ref-UeP5oN6q" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">68. </div><div class="csl-right-inline"><strong>Purification and properties of an extracellular protease of Staphylococcus aureus.</strong> <div class="csl-block">GR Drapeau, Y Boily, J Houmard</div> <em>The Journal of biological chemistry</em> (1972-10-25) <a href="https://www.ncbi.nlm.nih.gov/pubmed/4627743">https://www.ncbi.nlm.nih.gov/pubmed/4627743</a> <div class="csl-block">PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/4627743">4627743</a></div></div>
</div>
<div id="ref-FcovvAwu" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">69. </div><div class="csl-right-inline"><strong>Mildly acidic conditions eliminate deamidation artifact during proteolysis: digestion with endoprotease Glu-C at pH 4.5.</strong> <div class="csl-block">Shanshan Liu, Kevin Ryan Moulton, Jared Robert Auclair, Zhaohui Sunny Zhou</div> <em>Amino acids</em> (2016-01-09) <a href="https://www.ncbi.nlm.nih.gov/pubmed/26748652">https://www.ncbi.nlm.nih.gov/pubmed/26748652</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1007/s00726-015-2166-z">10.1007/s00726-015-2166-z</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/26748652">26748652</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4795971">PMC4795971</a></div></div>
</div>
<div id="ref-Z8khRFem" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">70. </div><div class="csl-right-inline"><strong>Specificity of endoproteinase Asp-N (Pseudomonas fragi): cleavage at glutamyl residues in two proteins.</strong> <div class="csl-block">D Ingrosso, AV Fowler, J Bleibaum, S Clarke</div> <em>Biochemical and biophysical research communications</em> (1989-08-15) <a href="https://www.ncbi.nlm.nih.gov/pubmed/2669754">https://www.ncbi.nlm.nih.gov/pubmed/2669754</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1016/0006-291x(89)90848-6">10.1016/0006-291x(89)90848-6</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/2669754">2669754</a></div></div>
</div>
<div id="ref-129jHkJ7J" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">71. </div><div class="csl-right-inline"><strong>Chymotrypsin: molecular and catalytic properties.</strong> <div class="csl-block">W Appel</div> <em>Clinical biochemistry</em> (1986-12) <a href="https://www.ncbi.nlm.nih.gov/pubmed/3555886">https://www.ncbi.nlm.nih.gov/pubmed/3555886</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1016/s0009-9120(86)80002-9">10.1016/s0009-9120(86)80002-9</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/3555886">3555886</a></div></div>
</div>
<div id="ref-n8tCUNeh" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">72. </div><div class="csl-right-inline"><strong>Mass-spectrometry-based draft of the human proteome.</strong> <div class="csl-block">Mathias Wilhelm, Judith Schlegl, Hannes Hahne, Amin Moghaddas Gholami, Marcus Lieberenz, Mikhail M Savitski, Emanuel Ziegler, Lars Butzmann, Siegfried Gessulat, Harald Marx, … Bernhard Kuster</div> <em>Nature</em> (2014-05-29) <a href="https://www.ncbi.nlm.nih.gov/pubmed/24870543">https://www.ncbi.nlm.nih.gov/pubmed/24870543</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1038/nature13319">10.1038/nature13319</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/24870543">24870543</a></div></div>
</div>
<div id="ref-Ddlp2JQj" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">73. </div><div class="csl-right-inline"><strong>Confetti: a multiprotease map of the HeLa proteome for comprehensive proteomics.</strong> <div class="csl-block">Xiaofeng Guo, David C Trudgian, Andrew Lemoff, Sivaramakrishna Yadavalli, Hamid Mirzaei</div> <em>Molecular &amp; cellular proteomics : MCP</em> (2014-04-02) <a href="https://www.ncbi.nlm.nih.gov/pubmed/24696503">https://www.ncbi.nlm.nih.gov/pubmed/24696503</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1074/mcp.m113.035170">10.1074/mcp.m113.035170</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/24696503">24696503</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4047476">PMC4047476</a></div></div>
</div>
<div id="ref-VbXzZ784" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">74. </div><div class="csl-right-inline"><strong>Quantitative and qualitative proteome characteristics extracted from in-depth integrated genomics and proteomics analysis.</strong> <div class="csl-block">Teck Yew Low, Sebastiaan van Heesch, Henk van den Toorn, Piero Giansanti, Alba Cristobal, Pim Toonen, Sebastian Schafer, Norbert Hübner, Bas van Breukelen, Shabaz Mohammed, … Victor Guryev</div> <em>Cell reports</em> (2013-11-27) <a href="https://www.ncbi.nlm.nih.gov/pubmed/24290761">https://www.ncbi.nlm.nih.gov/pubmed/24290761</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1016/j.celrep.2013.10.041">10.1016/j.celrep.2013.10.041</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/24290761">24290761</a></div></div>
</div>
<div id="ref-1BDWCCTHa" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">75. </div><div class="csl-right-inline"><strong>Protease bias in absolute protein quantitation.</strong> <div class="csl-block">Mao Peng, Nadia Taouatas, Salvatore Cappadona, Bas van Breukelen, Shabaz Mohammed, Arjen Scholten, Albert JR Heck</div> <em>Nature methods</em> (2012-05-30) <a href="https://www.ncbi.nlm.nih.gov/pubmed/22669647">https://www.ncbi.nlm.nih.gov/pubmed/22669647</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1038/nmeth.2031">10.1038/nmeth.2031</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/22669647">22669647</a></div></div>
</div>
<div id="ref-qJADzs4" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">76. </div><div class="csl-right-inline"><strong>Studies on the active site of clostripain. The specific inactivation by the chloromethyl ketone derived from -N-tosyl-L-lysine.</strong> <div class="csl-block">WH Porter, LW Cunningham, WM Mitchell</div> <em>The Journal of biological chemistry</em> (1971-12-25) <a href="https://www.ncbi.nlm.nih.gov/pubmed/4332560">https://www.ncbi.nlm.nih.gov/pubmed/4332560</a> <div class="csl-block">PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/4332560">4332560</a></div></div>
</div>
<div id="ref-MqKkSQNK" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">77. </div><div class="csl-right-inline"><strong>LysargiNase mirrors trypsin for protein C-terminal and methylation-site identification.</strong> <div class="csl-block">Pitter F Huesgen, Philipp F Lange, Lindsay D Rogers, Nestor Solis, Ulrich Eckhard, Oded Kleifeld, Theodoros Goulas, FXavier Gomis-Rüth, Christopher M Overall</div> <em>Nature methods</em> (2014-11-24) <a href="https://www.ncbi.nlm.nih.gov/pubmed/25419962">https://www.ncbi.nlm.nih.gov/pubmed/25419962</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1038/nmeth.3177">10.1038/nmeth.3177</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/25419962">25419962</a></div></div>
</div>
<div id="ref-3xdcLBaR" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">78. </div><div class="csl-right-inline"><strong>Proteomic analyses using Grifola frondosa metalloendoprotease Lys-N.</strong> <div class="csl-block">Laura Hohmann, Carly Sherwood, Ashley Eastham, Amelia Peterson, Jimmy K Eng, James S Eddes, David Shteynberg, Daniel B Martin</div> <em>Journal of proteome research</em> (2009-03) <a href="https://www.ncbi.nlm.nih.gov/pubmed/19195997">https://www.ncbi.nlm.nih.gov/pubmed/19195997</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1021/pr800774h">10.1021/pr800774h</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/19195997">19195997</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2798736">PMC2798736</a></div></div>
</div>
<div id="ref-O6S1GvPn" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">79. </div><div class="csl-right-inline"><strong>Straightforward ladder sequencing of peptides using a Lys-N metalloendopeptidase.</strong> <div class="csl-block">Nadia Taouatas, Madalina M Drugan, Albert JR Heck, Shabaz Mohammed</div> <em>Nature methods</em> (2008-04-20) <a href="https://www.ncbi.nlm.nih.gov/pubmed/18425140">https://www.ncbi.nlm.nih.gov/pubmed/18425140</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1038/nmeth.1204">10.1038/nmeth.1204</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/18425140">18425140</a></div></div>
</div>
<div id="ref-8DGqkN3D" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">80. </div><div class="csl-right-inline"><strong>Cleavage specificities of the brother and sister proteases Lys-C and Lys-N.</strong> <div class="csl-block">Reinout Raijmakers, Pieter Neerincx, Shabaz Mohammed, Albert JR Heck</div> <em>Chemical communications (Cambridge, England)</em> (2010-10-18) <a href="https://www.ncbi.nlm.nih.gov/pubmed/20953479">https://www.ncbi.nlm.nih.gov/pubmed/20953479</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1039/c0cc02523b">10.1039/c0cc02523b</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/20953479">20953479</a></div></div>
</div>
<div id="ref-eMC9mqfj" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">81. </div><div class="csl-right-inline"><strong>A history of pepsin and related enzymes.</strong> <div class="csl-block">Joseph S Fruton</div> <em>The Quarterly review of biology</em> (2002-06) <a href="https://www.ncbi.nlm.nih.gov/pubmed/12089768">https://www.ncbi.nlm.nih.gov/pubmed/12089768</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1086/340729">10.1086/340729</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/12089768">12089768</a></div></div>
</div>
<div id="ref-pnU7apE" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">82. </div><div class="csl-right-inline"><strong>CRYSTALLINE PEPSIN : I. ISOLATION AND TESTS OF PURITY.</strong> <div class="csl-block">JH Northrop</div> <em>The Journal of general physiology</em> (1930-07-20) <a href="https://www.ncbi.nlm.nih.gov/pubmed/19872561">https://www.ncbi.nlm.nih.gov/pubmed/19872561</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1085/jgp.13.6.739">10.1085/jgp.13.6.739</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/19872561">19872561</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2141071">PMC2141071</a></div></div>
</div>
<div id="ref-9Yf3fEMC" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">83. </div><div class="csl-right-inline"><strong>CRYSTALLINE PEPSIN : II. GENERAL PROPERTIES AND EXPERIMENTAL METHODS.</strong> <div class="csl-block">JH Northrop</div> <em>The Journal of general physiology</em> (1930-07-20) <a href="https://www.ncbi.nlm.nih.gov/pubmed/19872562">https://www.ncbi.nlm.nih.gov/pubmed/19872562</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1085/jgp.13.6.767">10.1085/jgp.13.6.767</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/19872562">19872562</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2141088">PMC2141088</a></div></div>
</div>
<div id="ref-10CHY6r2E" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">84. </div><div class="csl-right-inline"><strong>CRYSTALLINE PEPSIN.</strong> <div class="csl-block">JH Northrop</div> <em>Science (New York, N.Y.)</em> (1929-05-31) <a href="https://www.ncbi.nlm.nih.gov/pubmed/17758437">https://www.ncbi.nlm.nih.gov/pubmed/17758437</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1126/science.69.1796.580">10.1126/science.69.1796.580</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/17758437">17758437</a></div></div>
</div>
<div id="ref-1EtaxGZW0" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">85. </div><div class="csl-right-inline"><strong>The Nobel Prize in Chemistry 1946</strong> <div class="csl-block">NobelPrize.org</div> <a href="https://www.nobelprize.org/prizes/chemistry/1946/speedread/">https://www.nobelprize.org/prizes/chemistry/1946/speedread/</a></div>
</div>
<div id="ref-itW7kK3s" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">86. </div><div class="csl-right-inline"><strong>Protein disulfide bond determination by mass spectrometry.</strong> <div class="csl-block">Jeffrey J Gorman, Tristan P Wallis, James J Pitt</div> <em>Mass spectrometry reviews</em> <a href="https://www.ncbi.nlm.nih.gov/pubmed/12476442">https://www.ncbi.nlm.nih.gov/pubmed/12476442</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1002/mas.10025">10.1002/mas.10025</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/12476442">12476442</a></div></div>
</div>
<div id="ref-Vske1oHJ" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">87. </div><div class="csl-right-inline"><strong>Facilitating protein disulfide mapping by a combination of pepsin digestion, electron transfer higher energy dissociation (EThcD), and a dedicated search algorithm SlinkS.</strong> <div class="csl-block">Fan Liu, Bas van Breukelen, Albert JR Heck</div> <em>Molecular &amp; cellular proteomics : MCP</em> (2014-06-30) <a href="https://www.ncbi.nlm.nih.gov/pubmed/24980484">https://www.ncbi.nlm.nih.gov/pubmed/24980484</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1074/mcp.o114.039057">10.1074/mcp.o114.039057</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/24980484">24980484</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4189002">PMC4189002</a></div></div>
</div>
<div id="ref-Azi1X1R4" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">88. </div><div class="csl-right-inline"><strong>Online, High-Pressure Digestion System for Protein Characterization by Hydrogen/Deuterium Exchange and Mass Spectrometry</strong> <div class="csl-block">Lisa M Jones, Hao Zhang, Ilan Vidavsky, Michael L Gross</div> <em>Analytical Chemistry</em> (2010-01-22) <a href="https://doi.org/b993rm">https://doi.org/b993rm</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1021/ac902477u">10.1021/ac902477u</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/20095571">20095571</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2826105">PMC2826105</a></div></div>
</div>
<div id="ref-1nMCmYmb" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">89. </div><div class="csl-right-inline"><strong>Hydrogen/deuterium exchange in mass spectrometry</strong> <div class="csl-block">Yury Kostyukevich, Thamina Acter, Alexander Zherebker, Arif Ahmed, Sunghwan Kim, Eugene Nikolaev</div> <em>Mass Spectrometry Reviews</em> (2018-03-30) <a href="https://doi.org/gffzx8">https://doi.org/gffzx8</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1002/mas.21565">10.1002/mas.21565</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/29603316">29603316</a></div></div>
</div>
<div id="ref-YOXX6An6" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">90. </div><div class="csl-right-inline"><strong>Proteinase K from Tritirachium album Limber.</strong> <div class="csl-block">W Ebeling, N Hennrich, M Klockow, H Metz, HD Orth, H Lang</div> <em>European journal of biochemistry</em> (1974-08-15) <a href="https://www.ncbi.nlm.nih.gov/pubmed/4373242">https://www.ncbi.nlm.nih.gov/pubmed/4373242</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1111/j.1432-1033.1974.tb03671.x">10.1111/j.1432-1033.1974.tb03671.x</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/4373242">4373242</a></div></div>
</div>
<div id="ref-1CgjQBy2S" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">91. </div><div class="csl-right-inline"><strong>Proteinase K</strong> <div class="csl-block">W Saenger</div> <em>Handbook of Proteolytic Enzymes</em> (2013) <a href="https://doi.org/gkfkcz">https://doi.org/gkfkcz</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1016/b978-0-12-382219-2.00714-6">https://doi.org/10.1016/b978-0-12-382219-2.00714-6</a> · ISBN: 9780123822192</div></div>
</div>
<div id="ref-1BhNuatKH" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">92. </div><div class="csl-right-inline"><strong>Measuring protein structural changes on a proteome-wide scale using limited proteolysis-coupled mass spectrometry.</strong> <div class="csl-block">Simone Schopper, Abdullah Kahraman, Pascal Leuenberger, Yuehan Feng, Ilaria Piazza, Oliver Müller, Paul J Boersema, Paola Picotti</div> <em>Nature protocols</em> (2017-10-26) <a href="https://www.ncbi.nlm.nih.gov/pubmed/29072706">https://www.ncbi.nlm.nih.gov/pubmed/29072706</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1038/nprot.2017.100">10.1038/nprot.2017.100</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/29072706">29072706</a></div></div>
</div>
<div id="ref-58knZUn0" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">93. </div><div class="csl-right-inline"><strong>Simultaneous Quantification of the Acetylome and Succinylome by ‘One‐Pot’ Affinity Enrichment</strong> <div class="csl-block">Nathan Basisty, Jesse G Meyer, Lei Wei, Bradford W Gibson, Birgit Schilling</div> <em>PROTEOMICS</em> (2018-08-19) <a href="https://doi.org/gn4cmb">https://doi.org/gn4cmb</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1002/pmic.201800123">10.1002/pmic.201800123</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/30035354">30035354</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6175148">PMC6175148</a></div></div>
</div>
<div id="ref-YdLSljVE" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">94. </div><div class="csl-right-inline"><strong>The Nobel Prize in Chemistry 2002</strong> <div class="csl-block">NobelPrize.org</div> <a href="https://www.nobelprize.org/prizes/chemistry/2002/summary/">https://www.nobelprize.org/prizes/chemistry/2002/summary/</a></div>
</div>
<div id="ref-zfiYj2co" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">95. </div><div class="csl-right-inline"><strong>Influence of the wavelength in high-irradiance ultraviolet laser desorption mass spectrometry of organic molecules</strong> <div class="csl-block">Michael Karas, Doris Bachmann, Franz Hillenkamp</div> <em>Analytical Chemistry</em> (1985-12-01) <a href="https://pubs.acs.org/doi/abs/10.1021/ac00291a042">https://pubs.acs.org/doi/abs/10.1021/ac00291a042</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1021/ac00291a042">10.1021/ac00291a042</a></div></div>
</div>
<div id="ref-tnQHCFV7" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">96. </div><div class="csl-right-inline"><strong>Laser desorption ionization of proteins with molecular masses exceeding 10,000 daltons</strong> <div class="csl-block">Michael Karas, Franz Hillenkamp</div> <em>Analytical Chemistry</em> (1988-10-15) <a href="https://doi.org/d577jp">https://doi.org/d577jp</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1021/ac00171a028">10.1021/ac00171a028</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/3239801">3239801</a></div></div>
</div>
<div id="ref-1HBJs2tAB" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">97. </div><div class="csl-right-inline"><strong>The Scientist :: Nobel Prize controversy</strong> (2007-05-17) <a href="https://web.archive.org/web/20070517202246/http://cmbi.bjmu.edu.cn/news/0212/55.htm">https://web.archive.org/web/20070517202246/http://cmbi.bjmu.edu.cn/news/0212/55.htm</a></div>
</div>
<div id="ref-w7xaaHV0" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">98. </div><div class="csl-right-inline"><strong>α-Cyano-4-hydroxycinnamic acid, sinapinic acid, and ferulic acid as matrices and alkylating agents for matrix-assisted laser desorption/ionization time-of-flight mass spectrometric analysis of cysteine-containing peptides</strong> <div class="csl-block">Hongmei Yang, Debin Wan, Fengrui Song, Zhiqiang Liu, Shuying Liu</div> <em>Rapid communications in mass spectrometry: RCM</em> (2013-06-30) <a href="https://pubmed.ncbi.nlm.nih.gov/23681820">https://pubmed.ncbi.nlm.nih.gov/23681820</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1002/rcm.6587">10.1002/rcm.6587</a></div></div>
</div>
<div id="ref-wGHAVD9r" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">99. </div><div class="csl-right-inline"><strong>The Desorption Process in MALDI</strong> <div class="csl-block">Klaus Dreisewerd</div> <em>Chemical Reviews</em> (2003-01-24) <a href="https://doi.org/cpzqmq">https://doi.org/cpzqmq</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1021/cr010375i">10.1021/cr010375i</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/12580636">12580636</a></div></div>
</div>
<div id="ref-DEnAe0bs" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">100. </div><div class="csl-right-inline"><strong>Matrix Dependence of Metastable Fragmentation of Glycoproteins in MALDI TOF Mass Spectrometry</strong> <div class="csl-block">Michael Karas, Ute Bahr, Kerstin Strupat, Franz Hillenkamp, Anthony Tsarbopoulos, Birendra N Pramanik</div> <em>Analytical Chemistry</em> (1995-02-01) <a href="https://doi.org/b54gnt">https://doi.org/b54gnt</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1021/ac00099a029">10.1021/ac00099a029</a></div></div>
</div>
<div id="ref-BslEV2vj" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">101. </div><div class="csl-right-inline"><strong>XX. &lt;i&gt;On the equilibrium of liquid conducting masses charged with electricity&lt;/i&gt;</strong> <div class="csl-block">Lord Rayleigh</div> <em>The London, Edinburgh, and Dublin Philosophical Magazine and Journal of Science</em> (1882-09) <a href="https://doi.org/c6bp6h">https://doi.org/c6bp6h</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1080/14786448208628425">10.1080/14786448208628425</a></div></div>
</div>
<div id="ref-11vyuKWyw" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">102. </div><div class="csl-right-inline"><strong>Charge state coalescence during electrospray ionization improves peptide identification by tandem mass spectrometry.</strong> <div class="csl-block">Jesse G Meyer, Elizabeth A Komives</div> <em>Journal of the American Society for Mass Spectrometry</em> (2012-05-18) <a href="https://www.ncbi.nlm.nih.gov/pubmed/22610994">https://www.ncbi.nlm.nih.gov/pubmed/22610994</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1007/s13361-012-0404-0">10.1007/s13361-012-0404-0</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/22610994">22610994</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6345509">PMC6345509</a></div></div>
</div>
<div id="ref-4mpHldYP" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">103. </div><div class="csl-right-inline"><strong>Electrospray: from ions in solution to ions in the gas phase, what we know now.</strong> <div class="csl-block">Paul Kebarle, Udo H Verkerk</div> <em>Mass spectrometry reviews</em> <a href="https://www.ncbi.nlm.nih.gov/pubmed/19551695">https://www.ncbi.nlm.nih.gov/pubmed/19551695</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1002/mas.20247">10.1002/mas.20247</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/19551695">19551695</a></div></div>
</div>
<div id="ref-93IPP3bH" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">104. </div><div class="csl-right-inline"><strong>Unraveling the mechanism of electrospray ionization.</strong> <div class="csl-block">Lars Konermann, Elias Ahadi, Antony D Rodriguez, Siavash Vahidi</div> <em>Analytical chemistry</em> (2012-11-20) <a href="https://www.ncbi.nlm.nih.gov/pubmed/23134552">https://www.ncbi.nlm.nih.gov/pubmed/23134552</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1021/ac302789c">10.1021/ac302789c</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/23134552">23134552</a></div></div>
</div>
<div id="ref-JpKe7ymh" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">105. </div><div class="csl-right-inline"><strong>Capillary Liquid Chromatography/Mass Spectrometry for Peptide and Protein Characterization</strong> <div class="csl-block">MT Davis, DC Stahl, KM Swiderek, TD Lee</div> <em>Methods</em> (1994-09) <a href="https://doi.org/fmtw5k">https://doi.org/fmtw5k</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1006/meth.1994.1031">10.1006/meth.1994.1031</a></div></div>
</div>
<div id="ref-byV7q4rP" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">106. </div><div class="csl-right-inline"><strong>Method to Correlate Tandem Mass Spectra of Modified Peptides to Amino Acid Sequences in the Protein Database</strong> <div class="csl-block">John R Yates, Jimmy K Eng, Ashley L McCormack, David Schieltz</div> <em>Analytical Chemistry</em> (1995-04-15) <a href="https://doi.org/dtcrm9">https://doi.org/dtcrm9</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1021/ac00104a020">10.1021/ac00104a020</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/7741214">7741214</a></div></div>
</div>
<div id="ref-1BU9KEnwO" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">107. </div><div class="csl-right-inline"><strong>Data-controlled automation of liquid chromatography/tandem mass spectrometry analysis of peptide mixtures.</strong> <div class="csl-block">DC Stahl, KM Swiderek, MT Davis, TD Lee</div> <em>Journal of the American Society for Mass Spectrometry</em> (1996-06) <a href="https://www.ncbi.nlm.nih.gov/pubmed/24203425">https://www.ncbi.nlm.nih.gov/pubmed/24203425</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1016/1044-0305(96)00057-8">10.1016/1044-0305(96)00057-8</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/24203425">24203425</a></div></div>
</div>
<div id="ref-vO5QsPvL" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">108. </div><div class="csl-right-inline"><strong>Accurate Proteome-wide Label-free Quantification by Delayed Normalization and Maximal Peptide Ratio Extraction, Termed MaxLFQ</strong> <div class="csl-block">Jürgen Cox, Marco Y Hein, Christian A Luber, Igor Paron, Nagarjuna Nagaraj, Matthias Mann</div> <em>Molecular &amp;amp; Cellular Proteomics</em> (2014-09) <a href="https://doi.org/f6hmm5">https://doi.org/f6hmm5</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1074/mcp.m113.031591">10.1074/mcp.m113.031591</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/24942700">24942700</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4159666">PMC4159666</a></div></div>
</div>
<div id="ref-Gs7HWzAD" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">109. </div><div class="csl-right-inline"><strong>The MaxQuant computational platform for mass spectrometry-based shotgun proteomics</strong> <div class="csl-block">Stefka Tyanova, Tikira Temu, Juergen Cox</div> <em>Nature Protocols</em> (2016-10-27) <a href="https://doi.org/f89fwn">https://doi.org/f89fwn</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1038/nprot.2016.136">10.1038/nprot.2016.136</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/27809316">27809316</a></div></div>
</div>
<div id="ref-66LthRpU" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">110. </div><div class="csl-right-inline"><strong>Advances in proteomics data analysis and display using an accurate mass and time tag approach</strong> <div class="csl-block">Jennifer SD Zimmer, Matthew E Monroe, Wei-Jun Qian, Richard D Smith</div> <em>Mass Spectrometry Reviews</em> (2006) <a href="https://doi.org/bqv76w">https://doi.org/bqv76w</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1002/mas.20071">10.1002/mas.20071</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/16429408">16429408</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1829209">PMC1829209</a></div></div>
</div>
<div id="ref-MaoP0MGM" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">111. </div><div class="csl-right-inline"><strong>A New Algorithm Using Cross-Assignment for Label-Free Quantitation with LC−LTQ-FT MS</strong> <div class="csl-block">Victor P Andreev, Lingyun Li, Lei Cao, Ye Gu, Tomas Rejtar, Shiaw-Lin Wu, Barry L Karger</div> <em>Journal of Proteome Research</em> (2007-04-19) <a href="https://doi.org/d3fshp">https://doi.org/d3fshp</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1021/pr0606880">10.1021/pr0606880</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/17441747">17441747</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2563808">PMC2563808</a></div></div>
</div>
<div id="ref-5H13CEWA" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">112. </div><div class="csl-right-inline"><strong>IDEAL-Q, an Automated Tool for Label-free Quantitation Analysis Using an Efficient Peptide Alignment Approach and Spectral Data Validation</strong> <div class="csl-block">Chih-Chiang Tsou, Chia-Feng Tsai, Ying-Hao Tsui, Putty-Reddy Sudhir, Yi-Ting Wang, Yu-Ju Chen, Jeou-Yuan Chen, Ting-Yi Sung, Wen-Lian Hsu</div> <em>Molecular &amp;amp; Cellular Proteomics</em> (2010-01) <a href="https://doi.org/d9gxpm">https://doi.org/d9gxpm</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1074/mcp.m900177-mcp200">10.1074/mcp.m900177-mcp200</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/19752006">19752006</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2808259">PMC2808259</a></div></div>
</div>
<div id="ref-rRs3KNfz" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">113. </div><div class="csl-right-inline"><strong>&lt;b&gt;
&lt;i&gt;SuperHirn&lt;/i&gt;
&lt;/b&gt;
- a novel tool for high resolution LC-MS-based peptide/protein profiling</strong> <div class="csl-block">Lukas N Mueller, Oliver Rinner, Alexander Schmidt, Simon Letarte, Bernd Bodenmiller, Mi-Youn Brusniak, Olga Vitek, Ruedi Aebersold, Markus Müller</div> <em>PROTEOMICS</em> (2007-08-28) <a href="https://doi.org/c5zp29">https://doi.org/c5zp29</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1002/pmic.200700057">10.1002/pmic.200700057</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/17726677">17726677</a></div></div>
</div>
<div id="ref-1EHOiWEPV" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">114. </div><div class="csl-right-inline"><strong>Evaluating False Transfer Rates from the Match-between-Runs Algorithm with a Two-Proteome Model</strong> <div class="csl-block">Matthew Y Lim, João A Paulo, Steven P Gygi</div> <em>Journal of Proteome Research</em> (2019-09-23) <a href="https://doi.org/ggd6nh">https://doi.org/ggd6nh</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1021/acs.jproteome.9b00492">10.1021/acs.jproteome.9b00492</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/31547658">31547658</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7346880">PMC7346880</a></div></div>
</div>
<div id="ref-XGf3EEUa" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">115. </div><div class="csl-right-inline"><strong>IonQuant Enables Accurate and Sensitive Label-Free Quantification With FDR-Controlled Match-Between-Runs</strong> <div class="csl-block">Fengchao Yu, Sarah E Haynes, Alexey I Nesvizhskii</div> <em>Molecular &amp;amp; Cellular Proteomics</em> (2021) <a href="https://doi.org/gqfrh3">https://doi.org/gqfrh3</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1016/j.mcpro.2021.100077">10.1016/j.mcpro.2021.100077</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/33813065">33813065</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8131922">PMC8131922</a></div></div>
</div>
<div id="ref-2D6WnK0w" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">116. </div><div class="csl-right-inline"><strong>Post Analysis Data Acquisition for the Iterative MS/MS Sampling of Proteomics Mixtures</strong> <div class="csl-block">Michael R Hoopmann, Gennifer E Merrihew, Priska D von Haller, Michael J MacCoss</div> <em>Journal of Proteome Research</em> (2009-03-03) <a href="https://doi.org/fgtwdr">https://doi.org/fgtwdr</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1021/pr800828p">10.1021/pr800828p</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/19256536">19256536</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2671646">PMC2671646</a></div></div>
</div>
<div id="ref-yqW0xFtN" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">117. </div><div class="csl-right-inline"><strong>The Implications of Proteolytic Background for Shotgun Proteomics</strong> <div class="csl-block">Paola Picotti, Ruedi Aebersold, Bruno Domon</div> <em>Molecular &amp;amp; Cellular Proteomics</em> (2007-09) <a href="https://doi.org/ffrsx7">https://doi.org/ffrsx7</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1074/mcp.m700029-mcp200">10.1074/mcp.m700029-mcp200</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/17533221">17533221</a></div></div>
</div>
<div id="ref-SngtJdsy" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">118. </div><div class="csl-right-inline"><strong>Large-scale characterization of HeLa cell nuclear phosphoproteins</strong> <div class="csl-block">Sean A Beausoleil, Mark Jedrychowski, Daniel Schwartz, Joshua E Elias, Judit Villén, Jiaxu Li, Martin A Cohn, Lewis C Cantley, Steven P Gygi</div> <em>Proceedings of the National Academy of Sciences</em> (2004-08-09) <a href="https://doi.org/cjmg8h">https://doi.org/cjmg8h</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1073/pnas.0404720101">10.1073/pnas.0404720101</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/15302935">15302935</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC514446">PMC514446</a></div></div>
</div>
<div id="ref-WtrZNDoq" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">119. </div><div class="csl-right-inline"><strong>Evaluation of the utility of neutral-loss-dependent MS3 strategies in large-scale phosphorylation analysis</strong> <div class="csl-block">Judit Villén, Sean A Beausoleil, Steven P Gygi</div> <em>PROTEOMICS</em> (2008-11) <a href="https://doi.org/c52q48">https://doi.org/c52q48</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1002/pmic.200800283">10.1002/pmic.200800283</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/18972524">18972524</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2745099">PMC2745099</a></div></div>
</div>
<div id="ref-GL3PYaNa" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">120. </div><div class="csl-right-inline"><strong>Higher Energy Collision Dissociation (HCD) Product Ion-Triggered Electron Transfer Dissociation (ETD) Mass Spectrometry for the Analysis of &lt;i&gt;N&lt;/i&gt;-Linked Glycoproteins</strong> <div class="csl-block">Charandeep Singh, Cleidiane G Zampronio, Andrew J Creese, Helen J Cooper</div> <em>Journal of Proteome Research</em> (2012-08-06) <a href="https://doi.org/f36tfq">https://doi.org/f36tfq</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1021/pr300257c">10.1021/pr300257c</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/22800195">22800195</a></div></div>
</div>
<div id="ref-wAKbWLXc" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">121. </div><div class="csl-right-inline"><strong>Clinical applications of quantitative proteomics using targeted and untargeted data-independent acquisition techniques</strong> <div class="csl-block">Jesse G Meyer, Birgit Schilling</div> <em>Expert Review of Proteomics</em> (2017-05-04) <a href="https://doi.org/gk6gdn">https://doi.org/gk6gdn</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1080/14789450.2017.1322904">10.1080/14789450.2017.1322904</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/28436239">28436239</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5671767">PMC5671767</a></div></div>
</div>
<div id="ref-NQj9Wees" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">122. </div><div class="csl-right-inline"><strong>Review of software tools for design and analysis of large scale MRM proteomic datasets</strong> <div class="csl-block">Christopher M Colangelo, Lisa Chung, Can Bruce, Kei-Hoi Cheung</div> <em>Methods</em> (2013-06) <a href="https://doi.org/f449z8">https://doi.org/f449z8</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1016/j.ymeth.2013.05.004">10.1016/j.ymeth.2013.05.004</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/23702368">23702368</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3775261">PMC3775261</a></div></div>
</div>
<div id="ref-1FAMk6Qey" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">123. </div><div class="csl-right-inline"><strong>Quantification of C-reactive protein in the serum of patients with rheumatoid arthritis using multiple reaction monitoring mass spectrometry and13C-labeled peptide standards</strong> <div class="csl-block">Eric Kuhn, Jiang Wu, Johann Karl, Hua Liao, Werner Zolg, Brad Guild</div> <em>PROTEOMICS</em> (2004-04) <a href="https://doi.org/cxn7fw">https://doi.org/cxn7fw</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1002/pmic.200300670">10.1002/pmic.200300670</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/15048997">15048997</a></div></div>
</div>
<div id="ref-63Pqc9VE" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">124. </div><div class="csl-right-inline"><strong>Mass Spectrometric Quantitation of Peptides and Proteins Using Stable Isotope Standards and Capture by Anti-Peptide Antibodies (SISCAPA)</strong> <div class="csl-block">NLeigh Anderson, Norman G Anderson, Lee R Haines, Darryl B Hardie, Robert W Olafson, Terry W Pearson</div> <em>Journal of Proteome Research</em> (2004-02-06) <a href="https://doi.org/c494px">https://doi.org/c494px</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1021/pr034086h">10.1021/pr034086h</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/15113099">15113099</a></div></div>
</div>
<div id="ref-1B9P7OPqR" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">125. </div><div class="csl-right-inline"><strong>Quantitative Mass Spectrometric Multiple Reaction Monitoring Assays for Major Plasma Proteins</strong> <div class="csl-block">Leigh Anderson, Christie L Hunter</div> <em>Molecular &amp;amp; Cellular Proteomics</em> (2006-04) <a href="https://doi.org/c8f5g7">https://doi.org/c8f5g7</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1074/mcp.m500331-mcp200">10.1074/mcp.m500331-mcp200</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/16332733">16332733</a></div></div>
</div>
<div id="ref-iJJwYdQ1" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">126. </div><div class="csl-right-inline"><strong>A review on mass spectrometry-based quantitative proteomics: Targeted and data independent acquisition</strong> <div class="csl-block">Veronika Vidova, Zdenek Spacil</div> <em>Analytica Chimica Acta</em> (2017-04) <a href="https://doi.org/f93hpx">https://doi.org/f93hpx</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1016/j.aca.2017.01.059">10.1016/j.aca.2017.01.059</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/28351641">28351641</a></div></div>
</div>
<div id="ref-ba7F2Epn" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">127. </div><div class="csl-right-inline"><strong>The current status of clinical proteomics and the use of MRM and MRM&lt;sup&gt;3&lt;/sup&gt;for biomarker validation</strong> <div class="csl-block">Jérôme Lemoine, Tanguy Fortin, Arnaud Salvador, Aurore Jaffuel, Jean-Philippe Charrier, Geneviève Choquet-Kastylevsky</div> <em>Expert Review of Molecular Diagnostics</em> (2012-05) <a href="https://doi.org/f337q3">https://doi.org/f337q3</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1586/erm.12.32">10.1586/erm.12.32</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/22616699">22616699</a></div></div>
</div>
<div id="ref-jcu6ZT0x" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">128. </div><div class="csl-right-inline"><strong>High Sensitivity Detection of Plasma Proteins by Multiple Reaction Monitoring of N-Glycosites</strong> <div class="csl-block">Jianru Stahl-Zeng, Vinzenz Lange, Reto Ossola, Katrin Eckhardt, Wilhelm Krek, Ruedi Aebersold, Bruno Domon</div> <em>Molecular &amp;amp; Cellular Proteomics</em> (2007-10) <a href="https://doi.org/bwgf39">https://doi.org/bwgf39</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1074/mcp.m700132-mcp200">10.1074/mcp.m700132-mcp200</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/17644760">17644760</a></div></div>
</div>
<div id="ref-eFcH0xbv" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">129. </div><div class="csl-right-inline"><strong>A database of mass spectrometric assays for the yeast proteome</strong> <div class="csl-block">Paola Picotti, Henry Lam, David Campbell, Eric W Deutsch, Hamid Mirzaei, Jeff Ranish, Bruno Domon, Ruedi Aebersold</div> <em>Nature Methods</em> (2008-11) <a href="https://doi.org/dvs3c7">https://doi.org/dvs3c7</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1038/nmeth1108-913">10.1038/nmeth1108-913</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/18974732">18974732</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2770732">PMC2770732</a></div></div>
</div>
<div id="ref-1Bt8NAhNo" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">130. </div><div class="csl-right-inline"><strong>Parallel Reaction Monitoring for High Resolution and High Mass Accuracy Quantitative, Targeted Proteomics</strong> <div class="csl-block">Amelia C Peterson, Jason D Russell, Derek J Bailey, Michael S Westphall, Joshua J Coon</div> <em>Molecular &amp;amp; Cellular Proteomics</em> (2012-11) <a href="https://doi.org/f4gzw2">https://doi.org/f4gzw2</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1074/mcp.o112.020131">10.1074/mcp.o112.020131</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/22865924">22865924</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3494192">PMC3494192</a></div></div>
</div>
<div id="ref-HB3XvRF2" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">131. </div><div class="csl-right-inline"><strong>Parallel reaction monitoring (PRM) and selected reaction monitoring (SRM) exhibit comparable linearity, dynamic range and precision for targeted quantitative HDL proteomics</strong> <div class="csl-block">Graziella E Ronsein, Nathalie Pamir, Priska D von Haller, Daniel S Kim, Michael N Oda, Gail P Jarvik, Tomas Vaisar, Jay W Heinecke</div> <em>Journal of Proteomics</em> (2015-01) <a href="https://doi.org/f6wq8n">https://doi.org/f6wq8n</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1016/j.jprot.2014.10.017">10.1016/j.jprot.2014.10.017</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/25449833">25449833</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4259393">PMC4259393</a></div></div>
</div>
<div id="ref-dWzeUq6d" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">132. </div><div class="csl-right-inline"><strong>Shotgun collision-induced dissociation of peptides using a time of flight mass analyzer</strong> <div class="csl-block">Samuel Purvine, Jason-Thomas Eppel*, Eugene C Yi, David R Goodlett</div> <em>PROTEOMICS</em> (2003-06) <a href="https://doi.org/c37bqm">https://doi.org/c37bqm</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1002/pmic.200300362">10.1002/pmic.200300362</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/12833507">12833507</a></div></div>
</div>
<div id="ref-yn40ByaQ" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">133. </div><div class="csl-right-inline"><strong>Automated approach for quantitative analysis of complex peptide mixtures from tandem mass spectra</strong> <div class="csl-block">John D Venable, Meng-Qiu Dong, James Wohlschlegel, Andrew Dillin, John R Yates III</div> <em>Nature Methods</em> (2004-09-29) <a href="https://doi.org/dm8rm4">https://doi.org/dm8rm4</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1038/nmeth705">10.1038/nmeth705</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/15782151">15782151</a></div></div>
</div>
<div id="ref-Z3Hfnak1" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">134. </div><div class="csl-right-inline"><strong>UPLC/MSE; a new approach for generating molecular fragment information for biomarker structure elucidation</strong> <div class="csl-block">Robert S Plumb, Kelly A Johnson, Paul Rainville, Brian W Smith, Ian D Wilson, Jose M Castro-Perez, Jeremy K Nicholson</div> <em>Rapid Communications in Mass Spectrometry</em> (2006) <a href="https://doi.org/c8gx39">https://doi.org/c8gx39</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1002/rcm.2550">10.1002/rcm.2550</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/16755610">16755610</a></div></div>
</div>
<div id="ref-Wb9egjJg" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">135. </div><div class="csl-right-inline"><strong>Precursor Acquisition Independent From Ion Count: How to Dive Deeper into the Proteomics Ocean</strong> <div class="csl-block">Alexandre Panchaud, Alexander Scherl, Scott A Shaffer, Priska D von Haller, Hemantha D Kulasekara, Samuel I Miller, David R Goodlett</div> <em>Analytical Chemistry</em> (2009-07-02) <a href="https://doi.org/dwbrjn">https://doi.org/dwbrjn</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1021/ac900888s">10.1021/ac900888s</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/19572557">19572557</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3086478">PMC3086478</a></div></div>
</div>
<div id="ref-OubW7j0g" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">136. </div><div class="csl-right-inline"><strong>Proteomics on an Orbitrap Benchtop Mass Spectrometer Using All-ion Fragmentation</strong> <div class="csl-block">Tamar Geiger, Juergen Cox, Matthias Mann</div> <em>Molecular &amp;amp; Cellular Proteomics</em> (2010-10) <a href="https://doi.org/fqrb3c">https://doi.org/fqrb3c</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1074/mcp.m110.001537">10.1074/mcp.m110.001537</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/20610777">20610777</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2953918">PMC2953918</a></div></div>
</div>
<div id="ref-y71ZDsTJ" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">137. </div><div class="csl-right-inline"><strong>XDIA: improving on the label-free data-independent analysis</strong> <div class="csl-block">Paulo C Carvalho, Xuemei Han, Tao Xu, Daniel Cociorva, Maria da Gloria Carvalho, Valmir C Barbosa, John R Yates III</div> <em>Bioinformatics</em> (2010-01-26) <a href="https://doi.org/cn8x3r">https://doi.org/cn8x3r</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1093/bioinformatics/btq031">10.1093/bioinformatics/btq031</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/20106817">20106817</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2832823">PMC2832823</a></div></div>
</div>
<div id="ref-ckj0XYOM" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">138. </div><div class="csl-right-inline"><strong>ETISEQ – an algorithm for automated elution time ion sequencing of concurrently fragmented peptides for mass spectrometry-based proteomics</strong> <div class="csl-block">Jason WH Wong, Alexander B Schwahn, Kevin M Downard</div> <em>BMC Bioinformatics</em> (2009-08-10) <a href="https://doi.org/c3rshm">https://doi.org/c3rshm</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1186/1471-2105-10-244">10.1186/1471-2105-10-244</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/19664259">19664259</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2731054">PMC2731054</a></div></div>
</div>
<div id="ref-11ccXv5xm" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">139. </div><div class="csl-right-inline"><strong>Targeted Data Extraction of the MS/MS Spectra Generated by Data-independent Acquisition: A New Concept for Consistent and Accurate Proteome Analysis</strong> <div class="csl-block">Ludovic C Gillet, Pedro Navarro, Stephen Tate, Hannes Röst, Nathalie Selevsek, Lukas Reiter, Ron Bonner, Ruedi Aebersold</div> <em>Molecular &amp;amp; Cellular Proteomics</em> (2012-06) <a href="https://doi.org/fzqdc5">https://doi.org/fzqdc5</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1074/mcp.o111.016717">10.1074/mcp.o111.016717</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/22261725">22261725</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3433915">PMC3433915</a></div></div>
</div>
<div id="ref-1GCjdtGKY" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">140. </div><div class="csl-right-inline"><strong>Multiplexed MS/MS for improved data-independent acquisition</strong> <div class="csl-block">Jarrett D Egertson, Andreas Kuehn, Gennifer E Merrihew, Nicholas W Bateman, Brendan X MacLean, Ying S Ting, Jesse D Canterbury, Donald M Marsh, Markus Kellmann, Vlad Zabrouskov, … Michael J MacCoss</div> <em>Nature Methods</em> (2013-06-23) <a href="https://doi.org/gjqdct">https://doi.org/gjqdct</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1038/nmeth.2528">10.1038/nmeth.2528</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/23793237">23793237</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3881977">PMC3881977</a></div></div>
</div>
<div id="ref-1GvzNjeV3" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">141. </div><div class="csl-right-inline"><strong>Optimization of Experimental Parameters in Data-Independent Mass Spectrometry Significantly Increases Depth and Reproducibility of Results</strong> <div class="csl-block">Roland Bruderer, Oliver M Bernhardt, Tejas Gandhi, Yue Xuan, Julia Sondermann, Manuela Schmidt, David Gomez-Varela, Lukas Reiter</div> <em>Molecular &amp;amp; Cellular Proteomics</em> (2017-12) <a href="https://doi.org/gcqvm4">https://doi.org/gcqvm4</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1074/mcp.ra117.000314">10.1074/mcp.ra117.000314</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/29070702">29070702</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5724188">PMC5724188</a></div></div>
</div>
<div id="ref-YKRUqViz" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">142. </div><div class="csl-right-inline"><strong>diaPASEF: parallel accumulation–serial fragmentation combined with data-independent acquisition</strong> <div class="csl-block">Florian Meier, Andreas-David Brunner, Max Frank, Annie Ha, Isabell Bludau, Eugenia Voytik, Stephanie Kaspar-Schoenefeld, Markus Lubeck, Oliver Raether, Nicolai Bache, … Matthias Mann</div> <em>Nature Methods</em> (2020-11-30) <a href="https://doi.org/gmhqdm">https://doi.org/gmhqdm</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1038/s41592-020-00998-0">10.1038/s41592-020-00998-0</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/33257825">33257825</a></div></div>
</div>
<div id="ref-44oI1g68" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">143. </div><div class="csl-right-inline"><strong>Single-Shot 10K Proteome Approach: Over 10,000 Protein Identifications by Data-Independent Acquisition-Based Single-Shot Proteomics with Ion Mobility Spectrometry</strong> <div class="csl-block">Yusuke Kawashima, Hirotaka Nagai, Ryo Konno, Masaki Ishikawa, Daisuke Nakajima, Hironori Sato, Ren Nakamura, Tomoyuki Furuyashiki, Osamu Ohara</div> <em>Journal of Proteome Research</em> (2022-05-06) <a href="https://doi.org/gqgg5x">https://doi.org/gqgg5x</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1021/acs.jproteome.2c00023">10.1021/acs.jproteome.2c00023</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/35522919">35522919</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC9171847">PMC9171847</a></div></div>
</div>
<div id="ref-imR1dFei" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">144. </div><div class="csl-right-inline"><strong>Interpretation of shotgun proteomic data: the protein inference problem.</strong> <div class="csl-block">Alexey I Nesvizhskii, Ruedi Aebersold</div> <em>Molecular &amp; cellular proteomics : MCP</em> (2005-07-11) <a href="https://www.ncbi.nlm.nih.gov/pubmed/16009968">https://www.ncbi.nlm.nih.gov/pubmed/16009968</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1074/mcp.r500012-mcp200">10.1074/mcp.r500012-mcp200</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/16009968">16009968</a></div></div>
</div>
<div id="ref-wtH5a4Ax" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">145. </div><div class="csl-right-inline"><strong>In-depth analysis of protein inference algorithms using multiple search engines and well-defined metrics</strong> <div class="csl-block">Enrique Audain, Julian Uszkoreit, Timo Sachsenberg, Julianus Pfeuffer, Xiao Liang, Henning Hermjakob, Aniel Sanchez, Martin Eisenacher, Knut Reinert, David L Tabb, … Yasset Perez-Riverol</div> <em>Journal of Proteomics</em> (2017-01) <a href="https://doi.org/f9r8r6">https://doi.org/f9r8r6</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1016/j.jprot.2016.08.002">10.1016/j.jprot.2016.08.002</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/27498275">27498275</a></div></div>
</div>
<div id="ref-18PIzg2Co" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">146. </div><div class="csl-right-inline"><strong>MaxQuant enables high peptide identification rates, individualized p.p.b.-range mass accuracies and proteome-wide protein quantification</strong> <div class="csl-block">Jürgen Cox, Matthias Mann</div> <em>Nature Biotechnology</em> (2008-11-30) <a href="https://doi.org/crn24x">https://doi.org/crn24x</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1038/nbt.1511">10.1038/nbt.1511</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/19029910">19029910</a></div></div>
</div>
<div id="ref-bKpYR6PB" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">147. </div><div class="csl-right-inline"><strong>MSFragger: ultrafast and comprehensive peptide identification in mass spectrometry–based proteomics</strong> <div class="csl-block">Andy T Kong, Felipe V Leprevost, Dmitry M Avtonomov, Dattatreya Mellacheruvu, Alexey I Nesvizhskii</div> <em>Nature Methods</em> (2017-04-10) <a href="https://doi.org/f9z6p7">https://doi.org/f9z6p7</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1038/nmeth.4256">10.1038/nmeth.4256</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/28394336">28394336</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5409104">PMC5409104</a></div></div>
</div>
<div id="ref-rKtsv6G" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">148. </div><div class="csl-right-inline"><strong>Probability-based protein identification by searching sequence databases using mass spectrometry data.</strong> <div class="csl-block">DN Perkins, DJ Pappin, DM Creasy, JS Cottrell</div> <em>Electrophoresis</em> (1999-12) <a href="https://www.ncbi.nlm.nih.gov/pubmed/10612281">https://www.ncbi.nlm.nih.gov/pubmed/10612281</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1002/(sici)1522-2683(19991201)20:18&lt;3551::aid-elps3551&gt;3.0.co;2-2">10.1002/(sici)1522-2683(19991201)20:18&lt;3551::aid-elps3551&gt;3.0.co;2-2</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/10612281">10612281</a></div></div>
</div>
<div id="ref-1B2RVFOKK" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">149. </div><div class="csl-right-inline"><strong>MS-GF+ makes progress towards a universal database search tool for proteomics</strong> <div class="csl-block">Sangtae Kim, Pavel A Pevzner</div> <em>Nature Communications</em> (2014-10-31) <a href="https://doi.org/ggkdq8">https://doi.org/ggkdq8</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1038/ncomms6277">10.1038/ncomms6277</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/25358478">25358478</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5036525">PMC5036525</a></div></div>
</div>
<div id="ref-IcvYCrHi" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">150. </div><div class="csl-right-inline"><strong>A method for reducing the time required to match protein sequences with tandem mass spectra</strong> <div class="csl-block">Robertson Craig, Ronald C Beavis</div> <em>Rapid Communications in Mass Spectrometry</em> (2003) <a href="https://doi.org/b7bgb9">https://doi.org/b7bgb9</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1002/rcm.1198">10.1002/rcm.1198</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/14558131">14558131</a></div></div>
</div>
<div id="ref-iK4IJ8ED" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">151. </div><div class="csl-right-inline"><strong>TANDEM: matching proteins with tandem mass spectra</strong> <div class="csl-block">R Craig, RC Beavis</div> <em>Bioinformatics</em> (2004-02-19) <a href="https://doi.org/cthw6n">https://doi.org/cthw6n</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1093/bioinformatics/bth092">10.1093/bioinformatics/bth092</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/14976030">14976030</a></div></div>
</div>
<div id="ref-CObAggjA" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">152. </div><div class="csl-right-inline"><strong>Skyline: an open source document editor for creating and analyzing targeted proteomics experiments</strong> <div class="csl-block">Brendan MacLean, Daniela M Tomazela, Nicholas Shulman, Matthew Chambers, Gregory L Finney, Barbara Frewen, Randall Kern, David L Tabb, Daniel C Liebler, Michael J MacCoss</div> <em>Bioinformatics</em> (2010-02-09) <a href="https://doi.org/bqx9rq">https://doi.org/bqx9rq</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1093/bioinformatics/btq054">10.1093/bioinformatics/btq054</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/20147306">20147306</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2844992">PMC2844992</a></div></div>
</div>
<div id="ref-1HLH2DWkK" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">153. </div><div class="csl-right-inline"><strong>DIA-NN: neural networks and interference correction enable deep proteome coverage in high throughput</strong> <div class="csl-block">Vadim Demichev, Christoph B Messner, Spyros I Vernardis, Kathryn S Lilley, Markus Ralser</div> <em>Nature Methods</em> (2019-11-25) <a href="https://doi.org/gj9xgj">https://doi.org/gj9xgj</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1038/s41592-019-0638-x">10.1038/s41592-019-0638-x</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/31768060">31768060</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6949130">PMC6949130</a></div></div>
</div>
<div id="ref-IUqTUA43" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">154. </div><div class="csl-right-inline"><strong>Deep learning enables de novo peptide sequencing from data-independent-acquisition mass spectrometry</strong> <div class="csl-block">Ngoc Hieu Tran, Rui Qiao, Lei Xin, Xin Chen, Chuyi Liu, Xianglilan Zhang, Baozhen Shan, Ali Ghodsi, Ming Li</div> <em>Nature Methods</em> (2018-12-20) <a href="https://doi.org/gftvmn">https://doi.org/gftvmn</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1038/s41592-018-0260-3">10.1038/s41592-018-0260-3</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/30573815">30573815</a></div></div>
</div>
<div id="ref-UKcaDUJ7" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">155. </div><div class="csl-right-inline"><strong>Extending the Limits of Quantitative Proteome Profiling with Data-Independent Acquisition and Application to Acetaminophen-Treated Three-Dimensional Liver Microtissues</strong> <div class="csl-block">Roland Bruderer, Oliver M Bernhardt, Tejas Gandhi, Saša M Miladinović, Lin-Yang Cheng, Simon Messner, Tobias Ehrenberger, Vito Zanotelli, Yulia Butscheid, Claudia Escher, … Lukas Reiter</div> <em>Molecular &amp;amp; Cellular Proteomics</em> (2015-05) <a href="https://doi.org/f7b76h">https://doi.org/f7b76h</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1074/mcp.m114.044305">10.1074/mcp.m114.044305</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/25724911">25724911</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4424408">PMC4424408</a></div></div>
</div>
<div id="ref-B10jxZjh" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">156. </div><div class="csl-right-inline"><strong>PeptideShaker enables reanalysis of MS-derived proteomics data sets</strong> <div class="csl-block">Marc Vaudel, Julia M Burkhart, René P Zahedi, Eystein Oveland, Frode S Berven, Albert Sickmann, Lennart Martens, Harald Barsnes</div> <em>Nature Biotechnology</em> (2015-01) <a href="https://doi.org/ggkds8">https://doi.org/ggkds8</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1038/nbt.3109">10.1038/nbt.3109</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/25574629">25574629</a></div></div>
</div>
<div id="ref-tZ6JV4UF" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">157. </div><div class="csl-right-inline"><strong>PeptideShaker Online: A User-Friendly Web-Based Framework for the Identification of Mass Spectrometry-Based Proteomics Data</strong> <div class="csl-block">Yehia Mokhtar Farag, Carlos Horro, Marc Vaudel, Harald Barsnes</div> <em>Journal of Proteome Research</em> (2021-10-28) <a href="https://doi.org/gpdd85">https://doi.org/gpdd85</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1021/acs.jproteome.1c00678">10.1021/acs.jproteome.1c00678</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/34709836">34709836</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8650087">PMC8650087</a></div></div>
</div>
<div id="ref-zsmAmqZJ" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">158. </div><div class="csl-right-inline"><strong>mzML—a Community Standard for Mass Spectrometry Data</strong> <div class="csl-block">Lennart Martens, Matthew Chambers, Marc Sturm, Darren Kessner, Fredrik Levander, Jim Shofstahl, Wilfred H Tang, Andreas Römpp, Steffen Neumann, Angel D Pizarro, … Eric W Deutsch</div> <em>Molecular &amp;amp; Cellular Proteomics</em> (2011-01) <a href="https://doi.org/dxkg99">https://doi.org/dxkg99</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1074/mcp.r110.000133">10.1074/mcp.r110.000133</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/20716697">20716697</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3013463">PMC3013463</a></div></div>
</div>
<div id="ref-fQHXeOG1" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">159. </div><div class="csl-right-inline"><strong>Mass spectrometer output file format mzML.</strong> <div class="csl-block">Eric W Deutsch</div> <em>Methods in molecular biology (Clifton, N.J.)</em> (2010) <a href="https://www.ncbi.nlm.nih.gov/pubmed/20013381">https://www.ncbi.nlm.nih.gov/pubmed/20013381</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1007/978-1-60761-444-9_22">10.1007/978-1-60761-444-9_22</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/20013381">20013381</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3073315">PMC3073315</a></div></div>
</div>
<div id="ref-r2hPariY" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">160. </div><div class="csl-right-inline"><strong>File Formats Commonly Used in Mass Spectrometry Proteomics</strong> <div class="csl-block">Eric W Deutsch</div> <em>Molecular &amp;amp; Cellular Proteomics</em> (2012-12) <a href="https://doi.org/ggkdvv">https://doi.org/ggkdvv</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1074/mcp.r112.019695">10.1074/mcp.r112.019695</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/22956731">22956731</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3518119">PMC3518119</a></div></div>
</div>
<div id="ref-1GxDrwPIW" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">161. </div><div class="csl-right-inline"><strong>Anatomy and evolution of database search engines—a central component of mass spectrometry based proteomic workflows</strong> <div class="csl-block">Kenneth Verheggen, Helge Ræder, Frode S Berven, Lennart Martens, Harald Barsnes, Marc Vaudel</div> <em>Mass Spectrometry Reviews</em> (2020-05) <a href="https://doi.org/gbwkmf">https://doi.org/gbwkmf</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1002/mas.21543">10.1002/mas.21543</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/28902424">28902424</a></div></div>
</div>
<div id="ref-bjANWlBN" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">162. </div><div class="csl-right-inline"><strong>Assigning Significance to Peptides Identified by Tandem Mass Spectrometry Using Decoy Databases</strong> <div class="csl-block">Lukas Käll, John D Storey, Michael J MacCoss, William Stafford Noble</div> <em>Journal of Proteome Research</em> (2008-01) <a href="https://doi.org/fbxhxp">https://doi.org/fbxhxp</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1021/pr700600n">10.1021/pr700600n</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/18067246">18067246</a></div></div>
</div>
<div id="ref-BrkRiRe8" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">163. </div><div class="csl-right-inline"><strong>Target-decoy search strategy for increased confidence in large-scale protein identifications by mass spectrometry.</strong> <div class="csl-block">Joshua E Elias, Steven P Gygi</div> <em>Nature methods</em> (2007-03) <a href="https://www.ncbi.nlm.nih.gov/pubmed/17327847">https://www.ncbi.nlm.nih.gov/pubmed/17327847</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1038/nmeth1019">10.1038/nmeth1019</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/17327847">17327847</a></div></div>
</div>
<div id="ref-5mQtZh5o" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">164. </div><div class="csl-right-inline"><strong>False Discovery Rate Estimation in Proteomics</strong> <div class="csl-block">Suruchi Aggarwal, Amit Kumar Yadav</div> <em>Methods in Molecular Biology</em> (2016) <a href="https://doi.org/f79mzp">https://doi.org/f79mzp</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1007/978-1-4939-3106-4_7">10.1007/978-1-4939-3106-4_7</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/26519173">26519173</a></div></div>
</div>
<div id="ref-KDnDuk25" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">165. </div><div class="csl-right-inline"><strong>MAKER2: an annotation pipeline and genome-database management tool for second-generation genome projects.</strong> <div class="csl-block">Carson Holt, Mark Yandell</div> <em>BMC bioinformatics</em> (2011-12-22) <a href="https://www.ncbi.nlm.nih.gov/pubmed/22192575">https://www.ncbi.nlm.nih.gov/pubmed/22192575</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1186/1471-2105-12-491">10.1186/1471-2105-12-491</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/22192575">22192575</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3280279">PMC3280279</a></div></div>
</div>
<div id="ref-12mAW7BXf" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">166. </div><div class="csl-right-inline"><strong>The NCBI Eukaryotic Genome Annotation Pipeline</strong> <a href="https://www.ncbi.nlm.nih.gov/genome/annotation_euk/process/">https://www.ncbi.nlm.nih.gov/genome/annotation_euk/process/</a></div>
</div>
<div id="ref-BRlMNUOW" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">167. </div><div class="csl-right-inline"><strong>UniProt</strong> <a href="https://www.uniprot.org/help/biocuration">https://www.uniprot.org/help/biocuration</a></div>
</div>
<div id="ref-1EbCD9r1v" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">168. </div><div class="csl-right-inline"><a href="https://www.psidev.info/sites/default/files/2018-03/MIAPE_MSI_1.1.pdf">https://www.psidev.info/sites/default/files/2018-03/MIAPE_MSI_1.1.pdf</a></div>
</div>
<div id="ref-paTtvUb8" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">169. </div><div class="csl-right-inline"><strong>Guidelines for reporting quantitative mass spectrometry based experiments in proteomics.</strong> <div class="csl-block">Salvador Martínez-Bartolomé, Eric W Deutsch, Pierre-Alain Binz, Andrew R Jones, Martin Eisenacher, Gerhard Mayer, Alex Campos, Francesc Canals, Joan-Josep Bech-Serra, Montserrat Carrascal, … Juan P Albar</div> <em>Journal of proteomics</em> (2013-03-14) <a href="https://www.ncbi.nlm.nih.gov/pubmed/23500130">https://www.ncbi.nlm.nih.gov/pubmed/23500130</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1016/j.jprot.2013.02.026">10.1016/j.jprot.2013.02.026</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/23500130">23500130</a></div></div>
</div>
<div id="ref-16uefmiYk" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">170. </div><div class="csl-right-inline"><strong>UniProt: the Universal Protein knowledgebase.</strong> <div class="csl-block">Rolf Apweiler, Amos Bairoch, Cathy H Wu, Winona C Barker, Brigitte Boeckmann, Serenella Ferro, Elisabeth Gasteiger, Hongzhan Huang, Rodrigo Lopez, Michele Magrane, … Lai-Su L Yeh</div> <em>Nucleic acids research</em> (2004-01-01) <a href="https://www.ncbi.nlm.nih.gov/pubmed/14681372">https://www.ncbi.nlm.nih.gov/pubmed/14681372</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1093/nar/gkh131">10.1093/nar/gkh131</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/14681372">14681372</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC308865">PMC308865</a></div></div>
</div>
<div id="ref-1BIF8v4YE" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">171. </div><div class="csl-right-inline"><strong>UniProt</strong> <a href="https://www.uniprot.org/help/sequence_origin">https://www.uniprot.org/help/sequence_origin</a></div>
</div>
<div id="ref-Q2scu90J" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">172. </div><div class="csl-right-inline"><strong>UniProt</strong> <a href="https://www.uniprot.org/help/manual_curation">https://www.uniprot.org/help/manual_curation</a></div>
</div>
<div id="ref-16BHXvDGS" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">173. </div><div class="csl-right-inline"><strong>GitHub - pwilmart/fasta_utilities: Utilities for downloading and managing protein FASTA files.</strong> <div class="csl-block">GitHub</div> <a href="https://github.com/pwilmart/fasta_utilities">https://github.com/pwilmart/fasta_utilities</a></div>
</div>
<div id="ref-1GfGl6YGi" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">174. </div><div class="csl-right-inline"><strong>Global Identification of Protein Post-translational Modifications in a Single-Pass Database Search.</strong> <div class="csl-block">Michael R Shortreed, Craig D Wenger, Brian L Frey, Gloria M Sheynkman, Mark Scalf, Mark P Keller, Alan D Attie, Lloyd M Smith</div> <em>Journal of proteome research</em> (2015-09-29) <a href="https://www.ncbi.nlm.nih.gov/pubmed/26418581">https://www.ncbi.nlm.nih.gov/pubmed/26418581</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1021/acs.jproteome.5b00599">10.1021/acs.jproteome.5b00599</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/26418581">26418581</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4642219">PMC4642219</a></div></div>
</div>
<div id="ref-uPHDl2Vm" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">175. </div><div class="csl-right-inline"><strong>NCBI Datasets</strong> <div class="csl-block">NCBI</div> <a href="https://www.ncbi.nlm.nih.gov/datasets/">https://www.ncbi.nlm.nih.gov/datasets/</a></div>
</div>
<div id="ref-tg8zlLqI" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">176. </div><div class="csl-right-inline"><strong>Eukaryotic genomes annotated at NCBI</strong> <a href="https://www.ncbi.nlm.nih.gov/genome/annotation_euk/all/">https://www.ncbi.nlm.nih.gov/genome/annotation_euk/all/</a></div>
</div>
<div id="ref-xpZWsuGI" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">177. </div><div class="csl-right-inline"><strong>Prokaryotic RefSeq Genomes</strong> <a href="https://www.ncbi.nlm.nih.gov/refseq/about/prokaryotes/">https://www.ncbi.nlm.nih.gov/refseq/about/prokaryotes/</a></div>
</div>
<div id="ref-BfmLo0uT" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">178. </div><div class="csl-right-inline"><strong>Eukaryotic RefSeq Genome Annotation Status</strong> <a href="https://www.ncbi.nlm.nih.gov/genome/annotation_euk/status/">https://www.ncbi.nlm.nih.gov/genome/annotation_euk/status/</a></div>
</div>
<div id="ref-fWjGnKWy" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">179. </div><div class="csl-right-inline"><strong>Training &amp; Tutorials - Site Guide - NCBI</strong> <a href="https://www.ncbi.nlm.nih.gov/guide/training-tutorials/">https://www.ncbi.nlm.nih.gov/guide/training-tutorials/</a></div>
</div>
<div id="ref-17gB3bs6x" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">180. </div><div class="csl-right-inline"><strong>Proteomics in Non-model Organisms: A New Analytical Frontier.</strong> <div class="csl-block">Michelle Heck, Benjamin A Neely</div> <em>Journal of proteome research</em> (2020-08-20) <a href="https://www.ncbi.nlm.nih.gov/pubmed/32786681">https://www.ncbi.nlm.nih.gov/pubmed/32786681</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1021/acs.jproteome.0c00448">10.1021/acs.jproteome.0c00448</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32786681">32786681</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7874939">PMC7874939</a></div></div>
</div>
<div id="ref-69obv795" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">181. </div><div class="csl-right-inline"><strong>Ensembl genome browser 107</strong> <a href="https://uswest.ensembl.org/index.html">https://uswest.ensembl.org/index.html</a></div>
</div>
<div id="ref-1EGyqGdLr" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">182. </div><div class="csl-right-inline"><strong>Ensembl Genomes</strong> <a href="http://ensemblgenomes.org/">http://ensemblgenomes.org/</a></div>
</div>
<div id="ref-OeXy2hrh" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">183. </div><div class="csl-right-inline"><strong>Interferences and contaminants encountered in modern mass spectrometry.</strong> <div class="csl-block">Bernd O Keller, Jie Sui, Alex B Young, Randy M Whittal</div> <em>Analytica chimica acta</em> (2008-04-25) <a href="https://www.ncbi.nlm.nih.gov/pubmed/18790129">https://www.ncbi.nlm.nih.gov/pubmed/18790129</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1016/j.aca.2008.04.043">10.1016/j.aca.2008.04.043</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/18790129">18790129</a></div></div>
</div>
<div id="ref-jwgBjKOg" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">184. </div><div class="csl-right-inline"><strong>cRAP protein sequences</strong> <a href="https://www.thegpm.org/crap/">https://www.thegpm.org/crap/</a></div>
</div>
<div id="ref-WdwXXrRx" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">185. </div><div class="csl-right-inline"><strong>Philosopher: a versatile toolkit for shotgun proteomics data analysis.</strong> <div class="csl-block">Felipe da Veiga Leprevost, Sarah E Haynes, Dmitry M Avtonomov, Hui-Yin Chang, Avinash K Shanmugam, Dattatreya Mellacheruvu, Andy T Kong, Alexey I Nesvizhskii</div> <em>Nature methods</em> (2020-09) <a href="https://www.ncbi.nlm.nih.gov/pubmed/32669682">https://www.ncbi.nlm.nih.gov/pubmed/32669682</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1038/s41592-020-0912-y">10.1038/s41592-020-0912-y</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32669682">32669682</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7509848">PMC7509848</a></div></div>
</div>
<div id="ref-m8bjbzJ5" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">186. </div><div class="csl-right-inline"><strong>Protein Contaminants Matter: Building Universal Protein Contaminant Libraries for DDA and DIA Proteomics</strong> <div class="csl-block">Ashley M Frankenfield, Jiawei Ni, Mustafa Ahmed, Ling Hao</div> <em>Cold Spring Harbor Laboratory</em> (2022-04-28) <a href="https://doi.org/gp4xcp">https://doi.org/gp4xcp</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1101/2022.04.27.489766">10.1101/2022.04.27.489766</a></div></div>
</div>
<div id="ref-CnwgtVyY" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">187. </div><div class="csl-right-inline"><strong>The minotaur proteome: avoiding cross-species identifications deriving from bovine serum in cell culture models.</strong> <div class="csl-block">Jakob Bunkenborg, Guadalupe Espadas García, Marcia Ivonne Peña Paz, Jens S Andersen, Henrik Molina</div> <em>Proteomics</em> (2010-08) <a href="https://www.ncbi.nlm.nih.gov/pubmed/20641139">https://www.ncbi.nlm.nih.gov/pubmed/20641139</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1002/pmic.201000103">10.1002/pmic.201000103</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/20641139">20641139</a></div></div>
</div>
<div id="ref-1CIFGLtYQ" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">188. </div><div class="csl-right-inline"><strong>Foetal bovine serum influence on in vitro extracellular vesicle analyses.</strong> <div class="csl-block">Brandon M Lehrich, Yaxuan Liang, Massimo S Fiandaca</div> <em>Journal of extracellular vesicles</em> (2021-01-25) <a href="https://www.ncbi.nlm.nih.gov/pubmed/33532042">https://www.ncbi.nlm.nih.gov/pubmed/33532042</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1002/jev2.12061">10.1002/jev2.12061</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/33532042">33532042</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7830136">PMC7830136</a></div></div>
</div>
<div id="ref-wUllSafI" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">189. </div><div class="csl-right-inline"><strong>The CRAPome: a contaminant repository for affinity purification-mass spectrometry data.</strong> <div class="csl-block">Dattatreya Mellacheruvu, Zachary Wright, Amber L Couzens, Jean-Philippe Lambert, Nicole A St-Denis, Tuo Li, Yana V Miteva, Simon Hauri, Mihaela E Sardiu, Teck Yew Low, … Alexey I Nesvizhskii</div> <em>Nature methods</em> (2013-07-07) <a href="https://www.ncbi.nlm.nih.gov/pubmed/23921808">https://www.ncbi.nlm.nih.gov/pubmed/23921808</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1038/nmeth.2557">10.1038/nmeth.2557</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/23921808">23921808</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3773500">PMC3773500</a></div></div>
</div>
<div id="ref-EyM6Y0Xm" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">190. </div><div class="csl-right-inline"><strong>Common Repository of FBS Proteins (cRFP) To Be Added to a Search Database for Mass Spectrometric Analysis of Cell Secretome.</strong> <div class="csl-block">Jihye Shin, Yumi Kwon, Seonjeong Lee, Seungjin Na, Eun Young Hong, Shinyeong Ju, Hyun-Gyo Jung, Prashant Kaushal, Sungho Shin, Ji Hyun Back, … Cheolju Lee</div> <em>Journal of proteome research</em> (2019-09-10) <a href="https://www.ncbi.nlm.nih.gov/pubmed/31475827">https://www.ncbi.nlm.nih.gov/pubmed/31475827</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1021/acs.jproteome.9b00475">10.1021/acs.jproteome.9b00475</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/31475827">31475827</a></div></div>
</div>
<div id="ref-qgZxt4hu" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">191. </div><div class="csl-right-inline"><strong>Accurately Assigning Peptides to Spectra When Only a Subset of Peptides Are Relevant.</strong> <div class="csl-block">Andy Lin, Deanna L Plubell, Uri Keich, William S Noble</div> <em>Journal of proteome research</em> (2021-07-08) <a href="https://www.ncbi.nlm.nih.gov/pubmed/34236864">https://www.ncbi.nlm.nih.gov/pubmed/34236864</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1021/acs.jproteome.1c00483">10.1021/acs.jproteome.1c00483</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/34236864">34236864</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8489664">PMC8489664</a></div></div>
</div>
<div id="ref-12o52Fhcj" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">192. </div><div class="csl-right-inline"><strong>Averaging Strategy To Reduce Variability in Target-Decoy Estimates of False Discovery Rate.</strong> <div class="csl-block">Uri Keich, Kaipo Tamura, William Stafford Noble</div> <em>Journal of proteome research</em> (2019-01-03) <a href="https://www.ncbi.nlm.nih.gov/pubmed/30560673">https://www.ncbi.nlm.nih.gov/pubmed/30560673</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1021/acs.jproteome.8b00802">10.1021/acs.jproteome.8b00802</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/30560673">30560673</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6919216">PMC6919216</a></div></div>
</div>
<div id="ref-Lovg2liS" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">193. </div><div class="csl-right-inline"><strong>Mass spectrometrists should search only for peptides they care about.</strong> <div class="csl-block">William Stafford Noble</div> <em>Nature methods</em> (2015-07) <a href="https://www.ncbi.nlm.nih.gov/pubmed/26125591">https://www.ncbi.nlm.nih.gov/pubmed/26125591</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1038/nmeth.3450">10.1038/nmeth.3450</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/26125591">26125591</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4711994">PMC4711994</a></div></div>
</div>
<div id="ref-nMWQwf9T" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">194. </div><div class="csl-right-inline"><strong>Choosing an Optimal Database for Protein Identification from Tandem Mass Spectrometry Data.</strong> <div class="csl-block">Dhirendra Kumar, Amit Kumar Yadav, Debasis Dash</div> <em>Methods in molecular biology (Clifton, N.J.)</em> (2017) <a href="https://www.ncbi.nlm.nih.gov/pubmed/27975281">https://www.ncbi.nlm.nih.gov/pubmed/27975281</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1007/978-1-4939-6740-7_3">10.1007/978-1-4939-6740-7_3</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/27975281">27975281</a></div></div>
</div>
<div id="ref-JmTKP2r8" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">195. </div><div class="csl-right-inline"><strong>Proteomic analyses using an accurate mass and time tag strategy</strong> <div class="csl-block">Ljiljana Paša-Tolić, Christophe Masselon, Richard C Barry, Yufeng Shen, Richard D Smith</div> <em>BioTechniques</em> (2004-10) <a href="https://doi.org/gqcns4">https://doi.org/gqcns4</a> <div class="csl-block">DOI: <a href="https://doi.org/10.2144/04374rv01">10.2144/04374rv01</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/15517975">15517975</a></div></div>
</div>
</div>
<!-- default theme -->

<style>
  /* import google fonts */
  @import url("https://fonts.googleapis.com/css?family=Open+Sans:400,600,700");
  @import url("https://fonts.googleapis.com/css?family=Source+Code+Pro");

  /* -------------------------------------------------- */
  /* global */
  /* -------------------------------------------------- */

  /* all elements */
  * {
    /* force sans-serif font unless specified otherwise */
    font-family: "Open Sans", "Helvetica", sans-serif;

    /* prevent text inflation on some mobile browsers */
    -webkit-text-size-adjust: none !important;
    -moz-text-size-adjust: none !important;
    -o-text-size-adjust: none !important;
    text-size-adjust: none !important;
  }

  @media only screen {
    /* "page" element */
    body {
      position: relative;
      box-sizing: border-box;
      font-size: 12pt;
      line-height: 1.5;
      max-width: 8.5in;
      margin: 20px auto;
      padding: 40px;
      border-radius: 5px;
      border: solid 1px #bdbdbd;
      box-shadow: 0 0 20px rgba(0, 0, 0, 0.05);
      background: #ffffff;
    }
  }

  /* when on screen < 8.5in wide */
  @media only screen and (max-width: 8.5in) {
    /* "page" element */
    body {
      padding: 20px;
      margin: 0;
      border-radius: 0;
      border: none;
      box-shadow: 0 0 20px rgba(0, 0, 0, 0.05) inset;
      background: none;
    }
  }

  /* -------------------------------------------------- */
  /* headings */
  /* -------------------------------------------------- */

  /* all headings */
  h1,
  h2,
  h3,
  h4,
  h5,
  h6 {
    margin: 20px 0;
    padding: 0;
    font-weight: bold;
  }

  /* biggest heading */
  h1 {
    margin: 40px 0;
    text-align: center;
  }

  /* second biggest heading */
  h2 {
    margin-top: 30px;
    padding-bottom: 5px;
    border-bottom: solid 1px #bdbdbd;
  }

  /* heading font sizes */
  h1 {
    font-size: 2em;
  }
  h2 {
    font-size: 1.5em;
  }
  h3 {
    font-size: 1.35em;
  }
  h4 {
    font-size: 1.25em;
  }
  h5 {
    font-size: 1.15em;
  }
  h6 {
    font-size: 1em;
  }

  /* -------------------------------------------------- */
  /* manuscript header */
  /* -------------------------------------------------- */

  /* manuscript title */
  header > h1 {
    margin: 0;
  }

  /* manuscript title caption text (ie "automatically generated on") */
  header + p {
    text-align: center;
    margin-top: 10px;
  }

  /* -------------------------------------------------- */
  /* text elements */
  /* -------------------------------------------------- */

  /* links */
  a {
    color: #2196f3;
    overflow-wrap: break-word;
  }

  /* superscripts and subscripts */
  sub,
  sup {
    /* prevent from affecting line height */
    line-height: 0;
  }

  /* unordered and ordered lists*/
  ul,
  ol {
    padding-left: 20px;
  }

  /* class for styling text semibold */
  .semibold {
    font-weight: 600;
  }

  /* class for styling elements horizontally left aligned */
  .left {
    display: block;
    text-align: left;
    margin-left: auto;
    margin-right: 0;
    justify-content: left;
  }

  /* class for styling elements horizontally centered */
  .center {
    display: block;
    text-align: center;
    margin-left: auto;
    margin-right: auto;
    justify-content: center;
  }

  /* class for styling elements horizontally right aligned */
  .right {
    display: block;
    text-align: right;
    margin-left: 0;
    margin-right: auto;
    justify-content: right;
  }

  /* -------------------------------------------------- */
  /* section elements */
  /* -------------------------------------------------- */

  /* horizontal divider line */
  hr {
    border: none;
    height: 1px;
    background: #bdbdbd;
  }

  /* paragraphs, horizontal dividers, figures, tables, code */
  p,
  hr,
  figure,
  table,
  pre {
    /* treat all as "paragraphs", with consistent vertical margins */
    margin-top: 20px;
    margin-bottom: 20px;
  }

  /* -------------------------------------------------- */
  /* figures */
  /* -------------------------------------------------- */

  /* figure */
  figure {
    max-width: 100%;
    margin-left: auto;
    margin-right: auto;
  }

  /* figure caption */
  figcaption {
    padding: 0;
    padding-top: 10px;
  }

  /* figure image element */
  figure > img,
  figure > svg {
    max-width: 100%;
    display: block;
    margin-left: auto;
    margin-right: auto;
  }

  /* figure auto-number */
  img + figcaption > span:first-of-type,
  svg + figcaption > span:first-of-type {
    font-weight: bold;
    margin-right: 5px;
  }

  /* -------------------------------------------------- */
  /* tables */
  /* -------------------------------------------------- */

  /* table */
  table {
    border-collapse: collapse;
    border-spacing: 0;
    width: 100%;
    margin-left: auto;
    margin-right: auto;
  }

  /* table cells */
  th,
  td {
    border: solid 1px #bdbdbd;
    padding: 10px;
    /* squash table if too wide for page by forcing line breaks */
    overflow-wrap: break-word;
    word-break: break-word;
  }

  /* header row and even rows */
  th,
  tr:nth-child(2n) {
    background-color: #fafafa;
  }

  /* odd rows */
  tr:nth-child(2n + 1) {
    background-color: #ffffff;
  }

  /* table caption */
  caption {
    text-align: left;
    padding: 0;
    padding-bottom: 10px;
  }

  /* table auto-number */
  table > caption > span:first-of-type {
    font-weight: bold;
    margin-right: 5px;
  }

  /* -------------------------------------------------- */
  /* code */
  /* -------------------------------------------------- */

  /* multi-line code block */
  pre {
    padding: 10px;
    background-color: #eeeeee;
    color: #000000;
    border-radius: 5px;
    break-inside: avoid;
    text-align: left;
  }

  /* inline code, ie code within normal text */
  :not(pre) > code {
    padding: 0 4px;
    background-color: #eeeeee;
    color: #000000;
    border-radius: 5px;
  }

  /* code text */
  /* apply all children, to reach syntax highlighting sub-elements */
  code,
  code * {
    /* force monospace font */
    font-family: "Source Code Pro", "Courier New", monospace;
  }

  /* -------------------------------------------------- */
  /* quotes */
  /* -------------------------------------------------- */

  /* quoted text */
  blockquote {
    margin: 0;
    padding: 0;
    border-left: 4px solid #bdbdbd;
    padding-left: 16px;
    break-inside: avoid;
  }

  /* -------------------------------------------------- */
  /* banners */
  /* -------------------------------------------------- */

  /* info banners */
  .banner {
    box-sizing: border-box;
    display: block;
    position: relative;
    width: 100%;
    margin-top: 20px;
    margin-bottom: 20px;
    padding: 20px;
    text-align: center;
  }

  /* paragraph in banner */
  .banner > p {
    margin: 0;
  }

  /* -------------------------------------------------- */
  /* highlight colors */
  /* -------------------------------------------------- */

  .white {
    background: #ffffff;
  }
  .lightgrey {
    background: #eeeeee;
  }
  .grey {
    background: #757575;
  }
  .darkgrey {
    background: #424242;
  }
  .black {
    background: #000000;
  }
  .lightred {
    background: #ffcdd2;
  }
  .lightyellow {
    background: #ffecb3;
  }
  .lightgreen {
    background: #dcedc8;
  }
  .lightblue {
    background: #e3f2fd;
  }
  .lightpurple {
    background: #f3e5f5;
  }
  .red {
    background: #f44336;
  }
  .orange {
    background: #ff9800;
  }
  .yellow {
    background: #ffeb3b;
  }
  .green {
    background: #4caf50;
  }
  .blue {
    background: #2196f3;
  }
  .purple {
    background: #9c27b0;
  }
  .white,
  .lightgrey,
  .lightred,
  .lightyellow,
  .lightgreen,
  .lightblue,
  .lightpurple,
  .orange,
  .yellow,
  .white a,
  .lightgrey a,
  .lightred a,
  .lightyellow a,
  .lightgreen a,
  .lightblue a,
  .lightpurple a,
  .orange a,
  .yellow a {
    color: #000000;
  }
  .grey,
  .darkgrey,
  .black,
  .red,
  .green,
  .blue,
  .purple,
  .grey a,
  .darkgrey a,
  .black a,
  .red a,
  .green a,
  .blue a,
  .purple a {
    color: #ffffff;
  }

  /* -------------------------------------------------- */
  /* buttons */
  /* -------------------------------------------------- */

  /* class for styling links like buttons */
  .button {
    display: inline-flex;
    justify-content: center;
    align-items: center;
    margin: 5px;
    padding: 10px 20px;
    font-size: 0.75em;
    font-weight: 600;
    text-transform: uppercase;
    text-decoration: none;
    letter-spacing: 1px;
    background: none;
    color: #2196f3;
    border: solid 1px #bdbdbd;
    border-radius: 5px;
  }

  /* buttons when hovered */
  .button:hover:not([disabled]),
  .icon_button:hover:not([disabled]) {
    cursor: pointer;
    background: #f5f5f5;
  }

  /* buttons when disabled */
  .button[disabled],
  .icon_button[disabled] {
    opacity: 0.35;
    pointer-events: none;
  }

  /* class for styling buttons containg only single icon */
  .icon_button {
    display: inline-flex;
    justify-content: center;
    align-items: center;
    text-decoration: none;
    margin: 0;
    padding: 0;
    background: none;
    border-radius: 5px;
    border: none;
    width: 20px;
    height: 20px;
    min-width: 20px;
    min-height: 20px;
  }

  /* icon button inner svg image */
  .icon_button > svg {
    height: 16px;
  }

  /* -------------------------------------------------- */
  /* icons */
  /* -------------------------------------------------- */

  /* class for styling icons inline with text */
  .inline_icon {
    height: 1em;
    position: relative;
    top: 0.125em;
  }

  /* -------------------------------------------------- */
  /* references */
  /* -------------------------------------------------- */

  .csl-entry {
    margin-top: 15px;
    margin-bottom: 15px;
  }

  /* -------------------------------------------------- */
  /* print control */
  /* -------------------------------------------------- */

  @media print {
    @page {
      /* suggested printing margin */
      margin: 0.5in;
    }

    /* document and "page" elements */
    html,
    body {
      margin: 0;
      padding: 0;
      width: 100%;
      height: 100%;
    }

    /* "page" element */
    body {
      font-size: 11pt !important;
      line-height: 1.35;
    }

    /* all headings */
    h1,
    h2,
    h3,
    h4,
    h5,
    h6 {
      margin: 15px 0;
    }

    /* figures and tables */
    figure,
    table {
      font-size: 0.85em;
    }

    /* table cells */
    th,
    td {
      padding: 5px;
    }

    /* shrink font awesome icons */
    i.fas,
    i.fab,
    i.far,
    i.fal {
      transform: scale(0.85);
    }

    /* decrease banner margins */
    .banner {
      margin-top: 15px;
      margin-bottom: 15px;
      padding: 15px;
    }

    /* class for centering an element vertically on its own page */
    .page_center {
      margin: auto;
      width: 100%;
      height: 100%;
      display: flex;
      align-items: center;
      vertical-align: middle;
      break-before: page;
      break-after: page;
    }

    /* always insert a page break before the element */
    .page_break_before {
      break-before: page;
    }

    /* always insert a page break after the element */
    .page_break_after {
      break-after: page;
    }

    /* avoid page break before the element */
    .page_break_before_avoid {
      break-before: avoid;
    }

    /* avoid page break after the element */
    .page_break_after_avoid {
      break-after: avoid;
    }

    /* avoid page break inside the element */
    .page_break_inside_avoid {
      break-inside: avoid;
    }
  }

  /* -------------------------------------------------- */
  /* override pandoc css quirks */
  /* -------------------------------------------------- */

  .sourceCode {
    /* prevent unsightly overflow in wide code blocks */
    overflow: auto !important;
  }

  div.sourceCode {
    /* prevent background fill on top-most code block  container */
    background: none !important;
  }

  .sourceCode * {
    /* force consistent line spacing */
    line-height: 1.5 !important;
  }

  div.sourceCode {
    /* style code block margins same as <pre> element */
    margin-top: 20px;
    margin-bottom: 20px;
  }

  /* -------------------------------------------------- */
  /* tablenos */
  /* -------------------------------------------------- */

  /* tablenos wrapper */
  .tablenos {
    width: 100%;
    margin: 20px 0;
  }

  .tablenos > table {
    /* move margins from table to table_wrapper to allow margin collapsing */
    margin: 0;
  }

  @media only screen {
    /* tablenos wrapper */
    .tablenos {
      /* show scrollbar on tables if necessary to prevent overflow */
      overflow-x: auto !important;
    }

    .tablenos th,
    .tablenos td {
      overflow-wrap: unset !important;
      word-break: unset !important;
    }

    /* table in wrapper */
    .tablenos table,
    .tablenos table * {
      /* don't break table words */
      overflow-wrap: normal !important;
    }
  }
</style>
<!-- 
    Plugin Core

    Functions needed for and shared across all first-party plugins.
-->

<script>
  // get element that is target of hash (from link element or url)
  function getHashTarget(link) {
    const hash = link ? link.hash : window.location.hash;
    const id = hash.slice(1);
    let target = document.querySelector(`[id="${id}"]`);
    if (!target) return;

    // if figure or table, modify target to get expected element
    if (id.indexOf("fig:") === 0) target = target.querySelector("figure");
    if (id.indexOf("tbl:") === 0) target = target.querySelector("table");

    return target;
  }

  // get position/dimensions of element or viewport
  function getRectInView(element) {
    let rect = {};
    rect.left = 0;
    rect.top = 0;
    rect.right = document.documentElement.clientWidth;
    rect.bottom = document.documentElement.clientHeight;
    let style = {};

    if (element instanceof HTMLElement) {
      rect = element.getBoundingClientRect();
      style = window.getComputedStyle(element);
    }

    const margin = {};
    margin.left = parseFloat(style.marginLeftWidth) || 0;
    margin.top = parseFloat(style.marginTopWidth) || 0;
    margin.right = parseFloat(style.marginRightWidth) || 0;
    margin.bottom = parseFloat(style.marginBottomWidth) || 0;

    const border = {};
    border.left = parseFloat(style.borderLeftWidth) || 0;
    border.top = parseFloat(style.borderTopWidth) || 0;
    border.right = parseFloat(style.borderRightWidth) || 0;
    border.bottom = parseFloat(style.borderBottomWidth) || 0;

    const newRect = {};
    newRect.left = rect.left + margin.left + border.left;
    newRect.top = rect.top + margin.top + border.top;
    newRect.right = rect.right + margin.right + border.right;
    newRect.bottom = rect.bottom + margin.bottom + border.bottom;
    newRect.width = newRect.right - newRect.left;
    newRect.height = newRect.bottom - newRect.top;

    return newRect;
  }

  // get position of element relative to page
  function getRectInPage(element) {
    const rect = getRectInView(element);
    const body = getRectInView(document.body);

    const newRect = {};
    newRect.left = rect.left - body.left;
    newRect.top = rect.top - body.top;
    newRect.right = rect.right - body.left;
    newRect.bottom = rect.bottom - body.top;
    newRect.width = rect.width;
    newRect.height = rect.height;

    return newRect;
  }

  // get closest element before specified element that matches query
  function firstBefore(element, query) {
    while (element && element !== document.body && !element.matches(query))
      element = element.previousElementSibling || element.parentNode;

    return element;
  }

  // check if element is part of collapsed heading
  function isCollapsed(element) {
    while (element && element !== document.body) {
      if (element.dataset.collapsed === "true") return true;
      element = element.parentNode;
    }
    return false;
  }

  // expand any collapsed parent containers of element if necessary
  function expandElement(element) {
    if (isCollapsed(element)) {
      // accordion plugin
      const heading = firstBefore(element, "h2");
      if (heading) heading.click();
      // details/summary HTML element
      const summary = firstBefore(element, "summary");
      if (summary) summary.click();
    }
  }

  // scroll to and focus element
  function goToElement(element, offset) {
    // expand accordion section if collapsed
    expandElement(element);
    const y =
      getRectInView(element).top -
      getRectInView(document.documentElement).top -
      (offset || 0);

    // trigger any function listening for "onscroll" event
    window.dispatchEvent(new Event("scroll"));
    window.scrollTo(0, y);
    document.activeElement.blur();
    element.focus();
  }

  // get list of elements after a start element up to element matching query
  function nextUntil(element, query, exclude) {
    const elements = [];
    while (((element = element.nextElementSibling), element)) {
      if (element.matches(query)) break;
      if (!element.matches(exclude)) elements.push(element);
    }
    return elements;
  }
</script>
<!--
  Accordion Plugin

  Allows sections of content under h2 headings to be collapsible.
-->

<script type="module">
  // whether to always start expanded ('false'), always start collapsed
  // ('true'), or start collapsed when screen small ('auto')
  const startCollapsed = "auto";

  // start script
  function start() {
    // run through each <h2> heading
    const headings = document.querySelectorAll("h2");
    for (const heading of headings) {
      addArrow(heading);

      // start expanded/collapsed based on option
      if (
        startCollapsed === "true" ||
        (startCollapsed === "auto" && isSmallScreen()) ||
        heading.dataset.collapsed === "true"
      )
        collapseHeading(heading);
      else expandElement(heading);
    }

    // attach hash change listener to window
    window.addEventListener("hashchange", onHashChange);
  }

  // when hash (eg manuscript.html#introduction) changes
  function onHashChange() {
    const target = getHashTarget();
    if (target) goToElement(target);
  }

  // add arrow to heading
  function addArrow(heading) {
    // add arrow button
    const arrow = document.createElement("button");
    arrow.innerHTML = document.querySelector(".icon_angle_down").innerHTML;
    arrow.classList.add("icon_button", "accordion_arrow");
    heading.insertBefore(arrow, heading.firstChild);

    // attach click listener to heading and button
    heading.addEventListener("click", onHeadingClick);
    arrow.addEventListener("click", onArrowClick);
  }

  // determine if on mobile-like device with small screen
  function isSmallScreen() {
    return Math.min(window.innerWidth, window.innerHeight) < 480;
  }

  // when <h2> heading is clicked
  function onHeadingClick(event) {
    // only collapse if <h2> itself is target of click (eg, user did
    // not click on anchor within <h2>)
    if (event.target === this) toggleCollapse(this);
  }

  // when arrow button is clicked
  function onArrowClick() {
    toggleCollapse(this.parentNode);
  }

  // collapse section if expanded, expand if collapsed
  function toggleCollapse(heading) {
    if (heading.dataset.collapsed === "false") collapseHeading(heading);
    else expandElement(heading);
  }

  // elements to exclude from collapse, such as table of contents panel,
  // hypothesis panel, etc
  const exclude = "#toc_panel, div.annotator-frame, #lightbox_overlay";

  // collapse section
  function collapseHeading(heading) {
    heading.setAttribute("data-collapsed", "true");
    const children = getChildren(heading);
    for (const child of children) child.setAttribute("data-collapsed", "true");
  }

  // expand section
  function expandElement(heading) {
    heading.setAttribute("data-collapsed", "false");
    const children = getChildren(heading);
    for (const child of children) child.setAttribute("data-collapsed", "false");
  }

  // get list of elements between this <h2> and next <h2> or <h1>
  // ("children" of the <h2> section)
  function getChildren(heading) {
    return nextUntil(heading, "h2, h1", exclude);
  }

  // start script when document is finished loading
  window.addEventListener("load", start);
</script>

<!-- angle down icon -->

<template class="icon_angle_down">
  <!-- modified from: https://fontawesome.com/icons/angle-down -->
  <svg width="16" height="16" viewBox="0 0 448 512">
    <path
      fill="currentColor"
      d="M207.029 381.476L12.686 187.132c-9.373-9.373-9.373-24.569 0-33.941l22.667-22.667c9.357-9.357 24.522-9.375 33.901-.04L224 284.505l154.745-154.021c9.379-9.335 24.544-9.317 33.901.04l22.667 22.667c9.373 9.373 9.373 24.569 0 33.941L240.971 381.476c-9.373 9.372-24.569 9.372-33.942 0z"
    ></path>
  </svg>
</template>

<style>
  @media only screen {
    /* accordion arrow button */
    .accordion_arrow {
      margin-right: 10px;
    }

    /* arrow icon when <h2> data-collapsed attribute true */
    h2[data-collapsed="true"] > .accordion_arrow > svg {
      transform: rotate(-90deg);
    }

    /* all elements (except <h2>'s) when data-collapsed attribute true */
    *:not(h2)[data-collapsed="true"] {
      display: none;
    }

    /* accordion arrow button when hovered and <h2>'s when hovered */
    .accordion_arrow:hover,
    h2[data-collapsed="true"]:hover,
    h2[data-collapsed="false"]:hover {
      cursor: pointer;
    }
  }

  /* always hide accordion arrow button on print */
  @media only print {
    .accordion_arrow {
      display: none;
    }
  }
</style>
<!--
  Anchors Plugin

  Adds an anchor next to each of a certain type of element that provides a
  human-readable url to that specific item/position in the document (e.g.
  "manuscript.html#abstract"). It also makes it such that scrolling out of view
  of a target removes its identifier from the url.
-->

<script type="module">
  // which types of elements to add anchors next to, in "document.querySelector"
  // format
  const typesQuery =
    'h1, h2, h3, div[id^="fig:"], div[id^="tbl:"], span[id^="eq:"]';

  // start script
  function start() {
    // add anchor to each element of specified types
    const elements = document.querySelectorAll(typesQuery);
    for (const element of elements) addAnchor(element);

    // attach scroll listener to window
    window.addEventListener("scroll", onScroll);
  }

  // when window is scrolled
  function onScroll() {
    // if url has hash and user has scrolled out of view of hash
    // target, remove hash from url
    const tolerance = 100;
    const target = getHashTarget();
    if (target) {
      if (
        target.getBoundingClientRect().top > window.innerHeight + tolerance ||
        target.getBoundingClientRect().bottom < 0 - tolerance
      )
        history.pushState(null, null, " ");
    }
  }

  // add anchor to element
  function addAnchor(element) {
    let addTo; // element to add anchor button to

    // if figure or table, modify withId and addTo to get expected
    // elements
    if (element.id.indexOf("fig:") === 0) {
      addTo = element.querySelector("figcaption");
    } else if (element.id.indexOf("tbl:") === 0) {
      addTo = element.querySelector("caption");
    } else if (element.id.indexOf("eq:") === 0) {
      addTo = element.querySelector(".eqnos-number");
    }

    addTo = addTo || element;
    const id = element.id || null;

    // do not add anchor if element doesn't have assigned id.
    // id is generated by pandoc and is assumed to be unique and
    // human-readable
    if (!id) return;

    // create anchor button
    const anchor = document.createElement("a");
    anchor.innerHTML = document.querySelector(".icon_link").innerHTML;
    anchor.title = "Link to this part of the document";
    anchor.classList.add("icon_button", "anchor");
    anchor.dataset.ignore = "true";
    anchor.href = "#" + id;
    addTo.appendChild(anchor);
  }

  // start script when document is finished loading
  window.addEventListener("load", start);
</script>

<!-- link icon -->

<template class="icon_link">
  <!-- modified from: https://fontawesome.com/icons/link -->
  <svg width="16" height="16" viewBox="0 0 512 512">
    <path
      fill="currentColor"
      d="M326.612 185.391c59.747 59.809 58.927 155.698.36 214.59-.11.12-.24.25-.36.37l-67.2 67.2c-59.27 59.27-155.699 59.262-214.96 0-59.27-59.26-59.27-155.7 0-214.96l37.106-37.106c9.84-9.84 26.786-3.3 27.294 10.606.648 17.722 3.826 35.527 9.69 52.721 1.986 5.822.567 12.262-3.783 16.612l-13.087 13.087c-28.026 28.026-28.905 73.66-1.155 101.96 28.024 28.579 74.086 28.749 102.325.51l67.2-67.19c28.191-28.191 28.073-73.757 0-101.83-3.701-3.694-7.429-6.564-10.341-8.569a16.037 16.037 0 0 1-6.947-12.606c-.396-10.567 3.348-21.456 11.698-29.806l21.054-21.055c5.521-5.521 14.182-6.199 20.584-1.731a152.482 152.482 0 0 1 20.522 17.197zM467.547 44.449c-59.261-59.262-155.69-59.27-214.96 0l-67.2 67.2c-.12.12-.25.25-.36.37-58.566 58.892-59.387 154.781.36 214.59a152.454 152.454 0 0 0 20.521 17.196c6.402 4.468 15.064 3.789 20.584-1.731l21.054-21.055c8.35-8.35 12.094-19.239 11.698-29.806a16.037 16.037 0 0 0-6.947-12.606c-2.912-2.005-6.64-4.875-10.341-8.569-28.073-28.073-28.191-73.639 0-101.83l67.2-67.19c28.239-28.239 74.3-28.069 102.325.51 27.75 28.3 26.872 73.934-1.155 101.96l-13.087 13.087c-4.35 4.35-5.769 10.79-3.783 16.612 5.864 17.194 9.042 34.999 9.69 52.721.509 13.906 17.454 20.446 27.294 10.606l37.106-37.106c59.271-59.259 59.271-155.699.001-214.959z"
    ></path>
  </svg>
</template>

<style>
  @media only screen {
    /* anchor button */
    .anchor {
      opacity: 0;
      margin-left: 5px;
    }

    /* anchor buttons within <h2>'s */
    h2 .anchor {
      margin-left: 10px;
    }

    /* anchor buttons when hovered/focused and anything containing an anchor button when hovered */
    *:hover > .anchor,
    .anchor:hover,
    .anchor:focus {
      opacity: 1;
    }

    /* anchor button when hovered */
    .anchor:hover {
      cursor: pointer;
    }
  }

  /* always show anchor button on devices with no mouse/hover ability */
  @media (hover: none) {
    .anchor {
      opacity: 1;
    }
  }

  /* always hide anchor button on print */
  @media only print {
    .anchor {
      display: none;
    }
  }
</style>
<!-- 
    Attributes Plugin

    Allows arbitrary HTML attributes to be attached to (almost) any element.
    Place an HTML comment inside or next to the desired element with the content:
    $attribute="value"
-->

<script type="module">
  // start script
  function start() {
    // get list of comments in document
    const comments = findComments();

    for (const comment of comments)
      if (comment.parentElement)
        addAttributes(comment.parentElement, comment.nodeValue.trim());
  }

  // add html attributes to specified element based on string of
  // html attributes and values
  function addAttributes(element, text) {
    // regex's for finding attribute/value pairs in the format of
    // attribute="value" or attribute='value
    const regex2 = /\$([a-zA-Z\-]+)?=\"(.+?)\"/;
    const regex1 = /\$([a-zA-Z\-]+)?=\'(.+?)\'/;

    // loop through attribute/value pairs
    let match;
    while ((match = text.match(regex2) || text.match(regex1))) {
      // get attribute and value from regex capture groups
      let attribute = match[1];
      let value = match[2];

      // remove from string
      text = text.substring(match.index + match[0].length);

      if (!attribute || !value) break;

      // set attribute of parent element
      try {
        element.setAttribute(attribute, value);
      } catch (error) {
        console.log(error);
      }

      // special case for colspan
      if (attribute === "colspan") removeTableCells(element, value);
    }
  }

  // get list of comment elements in document
  function findComments() {
    const comments = [];

    // iterate over comment nodes in document
    function acceptNode(node) {
      return NodeFilter.FILTER_ACCEPT;
    }
    const iterator = document.createNodeIterator(
      document.body,
      NodeFilter.SHOW_COMMENT,
      acceptNode
    );
    let node;
    while ((node = iterator.nextNode())) comments.push(node);

    return comments;
  }

  // remove certain number of cells after specified cell
  function removeTableCells(cell, number) {
    number = parseInt(number);
    if (!number) return;

    // remove elements
    for (; number > 1; number--) {
      if (cell.nextElementSibling) cell.nextElementSibling.remove();
    }
  }

  // start script on DOMContentLoaded instead of load to ensure this plugins
  // runs before other plugins
  window.addEventListener("DOMContentLoaded", start);
</script>
<!--
  Jump to First Plugin

  Adds a button next to each reference entry, figure, and table that jumps the
  page to the first occurrence of a link to that item in the manuscript.
-->

<script type="module">
  // whether to add buttons next to reference entries
  const references = "true";
  // whether to add buttons next to figures
  const figures = "true";
  // whether to add buttons next to tables
  const tables = "true";

  // start script
  function start() {
    if (references !== "false")
      makeButtons(`div[id^="ref-"]`, ".csl-left-margin", "reference");
    if (figures !== "false")
      makeButtons(`div[id^="fig:"]`, "figcaption", "figure");
    if (tables !== "false") makeButtons(`div[id^="tbl:"]`, "caption", "table");
  }

  // when jump button clicked
  function onButtonClick() {
    const first = getFirstOccurrence(this.dataset.id);
    if (!first) return;

    // update url hash so navigating "back" in history will return user to button
    window.location.hash = this.dataset.id;
    // scroll to link
    const timeout = function () {
      goToElement(first, window.innerHeight * 0.5);
    };
    window.setTimeout(timeout, 0);
  }

  // get first occurrence of link to item in document
  function getFirstOccurrence(id) {
    let query = "a";
    query += '[href="#' + id + '"]';
    // exclude buttons, anchor links, toc links, etc
    query += ":not(.button):not(.icon_button):not(.anchor):not(.toc_link)";
    return document.querySelector(query);
  }

  // add button next to each reference entry, figure, or table
  function makeButtons(query, containerQuery, subject) {
    const elements = document.querySelectorAll(query);
    for (const element of elements) {
      const id = element.id;
      const buttonContainer = element.querySelector(containerQuery);
      const first = getFirstOccurrence(id);

      // if can't find link to reference or place to put button, ignore
      if (!first || !buttonContainer) continue;

      // make jump button
      let button = document.createElement("button");
      button.classList.add("icon_button", "jump_arrow");
      button.title = `Jump to the first occurrence of this ${subject} in the document`;
      const icon = document.querySelector(".icon_angle_double_up");
      button.innerHTML = icon.innerHTML;
      button.dataset.id = id;
      button.dataset.ignore = "true";
      button.addEventListener("click", onButtonClick);
      buttonContainer.prepend(button);
    }
  }

  // start script when document is finished loading
  window.addEventListener("load", start);
</script>

<!-- angle double up icon -->

<template class="icon_angle_double_up">
  <!-- modified from: https://fontawesome.com/icons/angle-double-up -->
  <svg width="16" height="16" viewBox="0 0 320 512">
    <path
      fill="currentColor"
      d="M177 255.7l136 136c9.4 9.4 9.4 24.6 0 33.9l-22.6 22.6c-9.4 9.4-24.6 9.4-33.9 0L160 351.9l-96.4 96.4c-9.4 9.4-24.6 9.4-33.9 0L7 425.7c-9.4-9.4-9.4-24.6 0-33.9l136-136c9.4-9.5 24.6-9.5 34-.1zm-34-192L7 199.7c-9.4 9.4-9.4 24.6 0 33.9l22.6 22.6c9.4 9.4 24.6 9.4 33.9 0l96.4-96.4 96.4 96.4c9.4 9.4 24.6 9.4 33.9 0l22.6-22.6c9.4-9.4 9.4-24.6 0-33.9l-136-136c-9.2-9.4-24.4-9.4-33.8 0z"
    ></path>
  </svg>
</template>

<style>
  @media only screen {
    /* jump button */
    .jump_arrow {
      position: relative;
      top: 0.125em;
      margin-right: 5px;
    }
  }

  /* always hide jump button on print */
  @media only print {
    .jump_arrow {
      display: none;
    }
  }
</style>
<!-- 
    Lightbox Plugin

    Makes it such that when a user clicks on an image, the image fills the
    screen and the user can pan/drag/zoom the image and navigate between other
    images in the document.
-->

<script type="module">
  // list of possible zoom/scale factors
  const zooms =
    "0.1, 0.25, 0.333333, 0.5, 0.666666, 0.75, 1, 1.25, 1.5, 1.75, 2, 2.5, 3, 3.5, 4, 5, 6, 7, 8";
  // whether to fit image to view ('fit'), display at 100% and shrink if
  // necessary ('shrink'), or always display at 100% ('100')
  const defaultZoom = "fit";
  // whether to zoom in/out toward center of view ('true') or mouse ('false')
  const centerZoom = "false";

  // start script
  function start() {
    // run through each <img> element
    const imgs = document.querySelectorAll("figure > img");
    let count = 1;
    for (const img of imgs) {
      img.classList.add("lightbox_document_img");
      img.dataset.number = count;
      img.dataset.total = imgs.length;
      img.addEventListener("click", openLightbox);
      count++;
    }

    // attach mouse and key listeners to window
    window.addEventListener("mousemove", onWindowMouseMove);
    window.addEventListener("keyup", onKeyUp);
  }

  // when mouse is moved anywhere in window
  function onWindowMouseMove(event) {
    window.mouseX = event.clientX;
    window.mouseY = event.clientY;
  }

  // when key pressed
  function onKeyUp(event) {
    if (!event || !event.key) return;

    switch (event.key) {
      // trigger click of prev button
      case "ArrowLeft":
        const prevButton = document.getElementById("lightbox_prev_button");
        if (prevButton) prevButton.click();
        break;
      // trigger click of next button
      case "ArrowRight":
        const nextButton = document.getElementById("lightbox_next_button");
        if (nextButton) nextButton.click();
        break;
      // close on esc
      case "Escape":
        closeLightbox();
        break;
    }
  }

  // open lightbox
  function openLightbox() {
    const lightbox = makeLightbox(this);
    if (!lightbox) return;

    blurBody(lightbox);
    document.body.appendChild(lightbox);
  }

  // make lightbox
  function makeLightbox(img) {
    // delete lightbox if it exists, start fresh
    closeLightbox();

    // create screen overlay containing lightbox
    const overlay = document.createElement("div");
    overlay.id = "lightbox_overlay";

    // create image info boxes
    const numberInfo = document.createElement("div");
    const zoomInfo = document.createElement("div");
    numberInfo.id = "lightbox_number_info";
    zoomInfo.id = "lightbox_zoom_info";

    // create container for image
    const imageContainer = document.createElement("div");
    imageContainer.id = "lightbox_image_container";
    const lightboxImg = makeLightboxImg(
      img,
      imageContainer,
      numberInfo,
      zoomInfo
    );
    imageContainer.appendChild(lightboxImg);

    // create bottom container for caption and navigation buttons
    const bottomContainer = document.createElement("div");
    bottomContainer.id = "lightbox_bottom_container";
    const caption = makeCaption(img);
    const prevButton = makePrevButton(img);
    const nextButton = makeNextButton(img);
    bottomContainer.appendChild(prevButton);
    bottomContainer.appendChild(caption);
    bottomContainer.appendChild(nextButton);

    // attach top middle and bottom to overlay
    overlay.appendChild(numberInfo);
    overlay.appendChild(zoomInfo);
    overlay.appendChild(imageContainer);
    overlay.appendChild(bottomContainer);

    return overlay;
  }

  // make <img> object that is intuitively draggable and zoomable
  function makeLightboxImg(sourceImg, container, numberInfoBox, zoomInfoBox) {
    // create copy of source <img>
    const img = sourceImg.cloneNode(true);
    img.classList.remove("lightbox_document_img");
    img.removeAttribute("id");
    img.removeAttribute("width");
    img.removeAttribute("height");
    img.style.position = "unset";
    img.style.margin = "0";
    img.style.padding = "0";
    img.style.width = "";
    img.style.height = "";
    img.style.minWidth = "";
    img.style.minHeight = "";
    img.style.maxWidth = "";
    img.style.maxHeight = "";
    img.id = "lightbox_img";

    // build sorted list of zoomSteps
    const zoomSteps = zooms.split(/[^0-9.]/).map((step) => parseFloat(step));
    zoomSteps.sort((a, b) => a - b);

    // <img> object property variables
    let zoom = 1;
    let translateX = 0;
    let translateY = 0;
    let clickMouseX = undefined;
    let clickMouseY = undefined;
    let clickTranslateX = undefined;
    let clickTranslateY = undefined;

    updateNumberInfo();

    // update image numbers displayed in info box
    function updateNumberInfo() {
      numberInfoBox.innerHTML =
        sourceImg.dataset.number + " of " + sourceImg.dataset.total;
    }

    // update zoom displayed in info box
    function updateZoomInfo() {
      let zoomInfo = zoom * 100;
      if (!Number.isInteger(zoomInfo)) zoomInfo = zoomInfo.toFixed(2);
      zoomInfoBox.innerHTML = zoomInfo + "%";
    }

    // move to closest zoom step above current zoom
    const zoomIn = function () {
      for (const zoomStep of zoomSteps) {
        if (zoomStep > zoom) {
          zoom = zoomStep;
          break;
        }
      }
      updateTransform();
    };

    // move to closest zoom step above current zoom
    const zoomOut = function () {
      zoomSteps.reverse();
      for (const zoomStep of zoomSteps) {
        if (zoomStep < zoom) {
          zoom = zoomStep;
          break;
        }
      }
      zoomSteps.reverse();

      updateTransform();
    };

    // update display of <img> based on scale/translate properties
    const updateTransform = function () {
      // set transform
      img.style.transform =
        "translate(" +
        (translateX || 0) +
        "px," +
        (translateY || 0) +
        "px) scale(" +
        (zoom || 1) +
        ")";

      // get new width/height after scale
      const rect = img.getBoundingClientRect();
      // limit translate
      translateX = Math.max(translateX, -rect.width / 2);
      translateX = Math.min(translateX, rect.width / 2);
      translateY = Math.max(translateY, -rect.height / 2);
      translateY = Math.min(translateY, rect.height / 2);

      // set transform
      img.style.transform =
        "translate(" +
        (translateX || 0) +
        "px," +
        (translateY || 0) +
        "px) scale(" +
        (zoom || 1) +
        ")";

      updateZoomInfo();
    };

    // fit <img> to container
    const fit = function () {
      // no x/y offset, 100% zoom by default
      translateX = 0;
      translateY = 0;
      zoom = 1;

      // widths of <img> and container
      const imgWidth = img.naturalWidth;
      const imgHeight = img.naturalHeight;
      const containerWidth = parseFloat(
        window.getComputedStyle(container).width
      );
      const containerHeight = parseFloat(
        window.getComputedStyle(container).height
      );

      // how much zooming is needed to fit <img> to container
      const xRatio = imgWidth / containerWidth;
      const yRatio = imgHeight / containerHeight;
      const maxRatio = Math.max(xRatio, yRatio);
      const newZoom = 1 / maxRatio;

      // fit <img> to container according to option
      if (defaultZoom === "shrink") {
        if (maxRatio > 1) zoom = newZoom;
      } else if (defaultZoom === "fit") zoom = newZoom;

      updateTransform();
    };

    // when mouse wheel is rolled anywhere in container
    const onContainerWheel = function (event) {
      if (!event) return;

      // let ctrl + mouse wheel to zoom behave as normal
      if (event.ctrlKey) return;

      // prevent normal scroll behavior
      event.preventDefault();
      event.stopPropagation();

      // point around which to scale img
      const viewRect = container.getBoundingClientRect();
      const viewX = (viewRect.left + viewRect.right) / 2;
      const viewY = (viewRect.top + viewRect.bottom) / 2;
      const originX = centerZoom === "true" ? viewX : mouseX;
      const originY = centerZoom === "true" ? viewY : mouseY;

      // get point on image under origin
      const oldRect = img.getBoundingClientRect();
      const oldPercentX = (originX - oldRect.left) / oldRect.width;
      const oldPercentY = (originY - oldRect.top) / oldRect.height;

      // increment/decrement zoom
      if (event.deltaY < 0) zoomIn();
      if (event.deltaY > 0) zoomOut();

      // get offset between previous image point and origin
      const newRect = img.getBoundingClientRect();
      const offsetX = originX - (newRect.left + newRect.width * oldPercentX);
      const offsetY = originY - (newRect.top + newRect.height * oldPercentY);

      // translate image to keep image point under origin
      translateX += offsetX;
      translateY += offsetY;

      // perform translate
      updateTransform();
    };

    // when container is clicked
    function onContainerClick(event) {
      // if container itself is target of click, and not other
      // element above it
      if (event.target === this) closeLightbox();
    }

    // when mouse button is pressed on image
    const onImageMouseDown = function (event) {
      // store original mouse position relative to image
      clickMouseX = window.mouseX;
      clickMouseY = window.mouseY;
      clickTranslateX = translateX;
      clickTranslateY = translateY;
      event.stopPropagation();
      event.preventDefault();
    };

    // when mouse button is released anywhere in window
    const onWindowMouseUp = function (event) {
      // reset original mouse position
      clickMouseX = undefined;
      clickMouseY = undefined;
      clickTranslateX = undefined;
      clickTranslateY = undefined;

      // remove global listener if lightbox removed from document
      if (!document.body.contains(container))
        window.removeEventListener("mouseup", onWindowMouseUp);
    };

    // when mouse is moved anywhere in window
    const onWindowMouseMove = function (event) {
      if (
        clickMouseX === undefined ||
        clickMouseY === undefined ||
        clickTranslateX === undefined ||
        clickTranslateY === undefined
      )
        return;

      // offset image based on original and current mouse position
      translateX = clickTranslateX + window.mouseX - clickMouseX;
      translateY = clickTranslateY + window.mouseY - clickMouseY;
      updateTransform();
      event.preventDefault();

      // remove global listener if lightbox removed from document
      if (!document.body.contains(container))
        window.removeEventListener("mousemove", onWindowMouseMove);
    };

    // when window is resized
    const onWindowResize = function (event) {
      fit();

      // remove global listener if lightbox removed from document
      if (!document.body.contains(container))
        window.removeEventListener("resize", onWindowResize);
    };

    // attach the necessary event listeners
    img.addEventListener("dblclick", fit);
    img.addEventListener("mousedown", onImageMouseDown);
    container.addEventListener("wheel", onContainerWheel);
    container.addEventListener("mousedown", onContainerClick);
    container.addEventListener("touchstart", onContainerClick);
    window.addEventListener("mouseup", onWindowMouseUp);
    window.addEventListener("mousemove", onWindowMouseMove);
    window.addEventListener("resize", onWindowResize);

    // run fit() after lightbox atttached to document and <img> Loaded
    // so needed container and img dimensions available
    img.addEventListener("load", fit);

    return img;
  }

  // make caption
  function makeCaption(img) {
    const caption = document.createElement("div");
    caption.id = "lightbox_caption";
    const captionSource = img.nextElementSibling;
    if (captionSource.tagName.toLowerCase() === "figcaption") {
      const captionCopy = makeCopy(captionSource);
      caption.innerHTML = captionCopy.innerHTML;
    }

    caption.addEventListener("touchstart", function (event) {
      event.stopPropagation();
    });

    return caption;
  }

  // make carbon copy of html dom element
  function makeCopy(source) {
    const sourceCopy = source.cloneNode(true);

    // delete elements marked with ignore (eg anchor and jump buttons)
    const deleteFromCopy = sourceCopy.querySelectorAll('[data-ignore="true"]');
    for (const element of deleteFromCopy) element.remove();

    // delete certain element attributes
    const attributes = [
      "id",
      "data-collapsed",
      "data-selected",
      "data-highlighted",
      "data-glow",
    ];
    for (const attribute of attributes) {
      sourceCopy.removeAttribute(attribute);
      const elements = sourceCopy.querySelectorAll("[" + attribute + "]");
      for (const element of elements) element.removeAttribute(attribute);
    }

    return sourceCopy;
  }

  // make button to jump to previous image in document
  function makePrevButton(img) {
    const prevButton = document.createElement("button");
    prevButton.id = "lightbox_prev_button";
    prevButton.title = "Jump to the previous image in the document [←]";
    prevButton.classList.add("icon_button", "lightbox_button");
    prevButton.innerHTML = document.querySelector(".icon_caret_left").innerHTML;

    // attach click listeners to button
    prevButton.addEventListener("click", function () {
      getPrevImg(img).click();
    });

    return prevButton;
  }

  // make button to jump to next image in document
  function makeNextButton(img) {
    const nextButton = document.createElement("button");
    nextButton.id = "lightbox_next_button";
    nextButton.title = "Jump to the next image in the document [→]";
    nextButton.classList.add("icon_button", "lightbox_button");
    nextButton.innerHTML = document.querySelector(
      ".icon_caret_right"
    ).innerHTML;

    // attach click listeners to button
    nextButton.addEventListener("click", function () {
      getNextImg(img).click();
    });

    return nextButton;
  }

  // get previous image in document
  function getPrevImg(img) {
    const imgs = document.querySelectorAll(".lightbox_document_img");

    // find index of provided img
    let index;
    for (index = 0; index < imgs.length; index++) {
      if (imgs[index] === img) break;
    }

    // wrap index to other side if < 1
    if (index - 1 >= 0) index--;
    else index = imgs.length - 1;
    return imgs[index];
  }

  // get next image in document
  function getNextImg(img) {
    const imgs = document.querySelectorAll(".lightbox_document_img");

    // find index of provided img
    let index;
    for (index = 0; index < imgs.length; index++) {
      if (imgs[index] === img) break;
    }

    // wrap index to other side if > total
    if (index + 1 <= imgs.length - 1) index++;
    else index = 0;
    return imgs[index];
  }

  // close lightbox
  function closeLightbox() {
    focusBody();

    const lightbox = document.getElementById("lightbox_overlay");
    if (lightbox) lightbox.remove();
  }

  // make all elements behind lightbox non-focusable
  function blurBody(overlay) {
    const all = document.querySelectorAll("*");
    for (const element of all) element.tabIndex = -1;
    document.body.classList.add("body_no_scroll");
  }

  // make all elements focusable again
  function focusBody() {
    const all = document.querySelectorAll("*");
    for (const element of all) element.removeAttribute("tabIndex");
    document.body.classList.remove("body_no_scroll");
  }

  // start script when document is finished loading
  window.addEventListener("load", start);
</script>

<!-- caret left icon -->

<template class="icon_caret_left">
  <!-- modified from: https://fontawesome.com/icons/caret-left -->
  <svg width="16" height="16" viewBox="0 0 192 512">
    <path
      fill="currentColor"
      d="M192 127.338v257.324c0 17.818-21.543 26.741-34.142 14.142L29.196 270.142c-7.81-7.81-7.81-20.474 0-28.284l128.662-128.662c12.599-12.6 34.142-3.676 34.142 14.142z"
    ></path>
  </svg>
</template>

<!-- caret right icon -->

<template class="icon_caret_right">
  <!-- modified from: https://fontawesome.com/icons/caret-right -->
  <svg width="16" height="16" viewBox="0 0 192 512">
    <path
      fill="currentColor"
      d="M0 384.662V127.338c0-17.818 21.543-26.741 34.142-14.142l128.662 128.662c7.81 7.81 7.81 20.474 0 28.284L34.142 398.804C21.543 411.404 0 402.48 0 384.662z"
    ></path>
  </svg>
</template>

<style>
  @media only screen {
    /* regular <img> in document when hovered */
    img.lightbox_document_img:hover {
      cursor: pointer;
    }

    .body_no_scroll {
      overflow: hidden !important;
    }

    /* screen overlay */
    #lightbox_overlay {
      display: flex;
      flex-direction: column;
      position: fixed;
      left: 0;
      top: 0;
      right: 0;
      bottom: 0;
      background: rgba(0, 0, 0, 0.75);
      z-index: 3;
    }

    /* middle area containing lightbox image */
    #lightbox_image_container {
      flex-grow: 1;
      display: flex;
      justify-content: center;
      align-items: center;
      overflow: hidden;
      position: relative;
      padding: 20px;
    }

    /* bottom area containing caption */
    #lightbox_bottom_container {
      display: flex;
      justify-content: center;
      align-items: center;
      height: 100px;
      min-height: 100px;
      max-height: 100px;
      background: rgba(0, 0, 0, 0.5);
    }

    /* image number info text box */
    #lightbox_number_info {
      position: absolute;
      color: #ffffff;
      font-weight: 600;
      left: 2px;
      top: 0;
      z-index: 4;
    }

    /* zoom info text box */
    #lightbox_zoom_info {
      position: absolute;
      color: #ffffff;
      font-weight: 600;
      right: 2px;
      top: 0;
      z-index: 4;
    }

    /* copy of image caption */
    #lightbox_caption {
      box-sizing: border-box;
      display: inline-block;
      width: 100%;
      max-height: 100%;
      padding: 10px 0;
      text-align: center;
      overflow-y: auto;
      color: #ffffff;
    }

    /* navigation previous/next button */
    .lightbox_button {
      width: 100px;
      height: 100%;
      min-width: 100px;
      min-height: 100%;
      color: #ffffff;
    }

    /* navigation previous/next button when hovered */
    .lightbox_button:hover {
      background: none !important;
    }

    /* navigation button icon */
    .lightbox_button > svg {
      height: 25px;
    }

    /* figure auto-number */
    #lightbox_caption > span:first-of-type {
      font-weight: bold;
      margin-right: 5px;
    }

    /* lightbox image when hovered */
    #lightbox_img:hover {
      cursor: grab;
    }

    /* lightbox image when grabbed */
    #lightbox_img:active {
      cursor: grabbing;
    }
  }

  /* when on screen < 480px wide */
  @media only screen and (max-width: 480px) {
    /* make navigation buttons skinnier on small screens to make more room for caption text */
    .lightbox_button {
      width: 50px;
      min-width: 50px;
    }
  }

  /* always hide lightbox on print */
  @media only print {
    #lightbox_overlay {
      display: none;
    }
  }
</style>
<!-- 
  Link Highlight Plugin

  Makes it such that when a user hovers or focuses a link, other links that have
  the same target will be highlighted. It also makes it such that when clicking
  a link, the target of the link (eg reference, figure, table) is briefly
  highlighted.
-->

<script type="module">
  // whether to also highlight links that go to external urls
  const externalLinks = "false";
  // whether user must click off to unhighlight instead of just
  // un-hovering
  const clickUnhighlight = "false";
  // whether to also highlight links that are unique
  const highlightUnique = "true";

  // start script
  function start() {
    const links = getLinks();
    for (const link of links) {
      // attach mouse and focus listeners to link
      link.addEventListener("mouseenter", onLinkFocus);
      link.addEventListener("focus", onLinkFocus);
      link.addEventListener("mouseleave", onLinkUnhover);
    }

    // attach click and hash change listeners to window
    window.addEventListener("click", onClick);
    window.addEventListener("touchstart", onClick);
    window.addEventListener("hashchange", onHashChange);

    // run hash change on window load in case user has navigated
    // directly to hash
    onHashChange();
  }

  // when link is focused (tabbed to) or hovered
  function onLinkFocus() {
    highlight(this);
  }

  // when link is unhovered
  function onLinkUnhover() {
    if (clickUnhighlight !== "true") unhighlightAll();
  }

  // when the mouse is clicked anywhere in window
  function onClick(event) {
    unhighlightAll();
  }

  // when hash (eg manuscript.html#introduction) changes
  function onHashChange() {
    const target = getHashTarget();
    if (target) glowElement(target);
  }

  // start glow sequence on an element
  function glowElement(element) {
    const startGlow = function () {
      onGlowEnd();
      element.dataset.glow = "true";
      element.addEventListener("animationend", onGlowEnd);
    };
    const onGlowEnd = function () {
      element.removeAttribute("data-glow");
      element.removeEventListener("animationend", onGlowEnd);
    };
    startGlow();
  }

  // highlight link and all others with same target
  function highlight(link) {
    // force unhighlight all to start fresh
    unhighlightAll();

    // get links with same target
    if (!link) return;
    const sameLinks = getSameLinks(link);

    // if link unique and option is off, exit and don't highlight
    if (sameLinks.length <= 1 && highlightUnique !== "true") return;

    // highlight all same links, and "select" (special highlight) this
    // one
    for (const sameLink of sameLinks) {
      if (sameLink === link) sameLink.setAttribute("data-selected", "true");
      else sameLink.setAttribute("data-highlighted", "true");
    }
  }

  // unhighlight all links
  function unhighlightAll() {
    const links = getLinks();
    for (const link of links) {
      link.setAttribute("data-selected", "false");
      link.setAttribute("data-highlighted", "false");
    }
  }

  // get links with same target
  function getSameLinks(link) {
    const results = [];
    const links = getLinks();
    for (const otherLink of links) {
      if (otherLink.getAttribute("href") === link.getAttribute("href"))
        results.push(otherLink);
    }
    return results;
  }

  // get all links of types we wish to handle
  function getLinks() {
    let query = "a";
    if (externalLinks !== "true") query += '[href^="#"]';
    // exclude buttons, anchor links, toc links, etc
    query += ":not(.button):not(.icon_button):not(.anchor):not(.toc_link)";
    return document.querySelectorAll(query);
  }

  // start script when document is finished loading
  window.addEventListener("load", start);
</script>

<style>
  @media only screen {
    /* anything with data-highlighted attribute true */
    [data-highlighted="true"] {
      background: #ffeb3b;
    }

    /* anything with data-selected attribute true */
    [data-selected="true"] {
      background: #ff8a65 !important;
    }

    /* animation definition for glow */
    @keyframes highlight_glow {
      0% {
        background: none;
      }
      10% {
        background: #bbdefb;
      }
      100% {
        background: none;
      }
    }

    /* anything with data-glow attribute true */
    [data-glow="true"] {
      animation: highlight_glow 2s;
    }
  }
</style>
<!--
  Table of Contents Plugin

  Provides a "table of contents" (toc) panel on the side of the document that
  allows the user to conveniently navigate between sections of the document.
-->

<script type="module">
  // which types of elements to add links for, in "document.querySelector" format
  const typesQuery = "h1, h2, h3";
  // whether toc starts open. use 'true' or 'false', or 'auto' to
  // use 'true' behavior when screen wide enough and 'false' when not
  const startOpen = "false";
  // whether toc closes when clicking on toc link. use 'true' or
  // 'false', or 'auto' to use 'false' behavior when screen wide
  // enough and 'true' when not
  const clickClose = "auto";
  // if list item is more than this many characters, text will be
  // truncated
  const charLimit = "50";
  // whether or not to show bullets next to each toc item
  const bullets = "false";

  // start script
  function start() {
    // make toc panel and populate with entries (links to document
    // sections)
    const panel = makePanel();
    if (!panel) return;
    makeEntries(panel);
    // attach panel to document after making entries, so 'toc' heading
    // in panel isn't included in toc
    document.body.insertBefore(panel, document.body.firstChild);

    // initial panel state
    if (startOpen === "true" || (startOpen === "auto" && !isSmallScreen()))
      openPanel();
    else closePanel();

    // attach click, scroll, and hash change listeners to window
    window.addEventListener("click", onClick);
    window.addEventListener("scroll", onScroll);
    window.addEventListener("hashchange", onScroll);
    window.addEventListener("keyup", onKeyUp);
    onScroll();

    // add class to push document body down out of way of toc button
    document.body.classList.add("toc_body_nudge");
  }

  // determine if screen wide enough to fit toc panel
  function isSmallScreen() {
    // in default theme:
    // 816px = 8.5in = width of "page" (<body>) element
    // 260px = min width of toc panel (*2 for both sides of <body>)
    return window.innerWidth < 816 + 260 * 2;
  }

  // when mouse is clicked anywhere in window
  function onClick() {
    if (isSmallScreen()) closePanel();
  }

  // when window is scrolled or hash changed
  function onScroll() {
    highlightViewed();
  }

  // when key pressed
  function onKeyUp(event) {
    if (!event || !event.key) return;

    // close on esc
    if (event.key === "Escape") closePanel();
  }

  // find entry of currently viewed document section in toc and highlight
  function highlightViewed() {
    const firstId = getFirstInView(typesQuery);

    // get toc entries (links), unhighlight all, then highlight viewed
    const list = document.getElementById("toc_list");
    if (!firstId || !list) return;
    const links = list.querySelectorAll("a");
    for (const link of links) link.dataset.viewing = "false";
    const link = list.querySelector('a[href="#' + firstId + '"]');
    if (!link) return;
    link.dataset.viewing = "true";
  }

  // get first or previous toc listed element in top half of view
  function getFirstInView(query) {
    // get all elements matching query and with id
    const elements = document.querySelectorAll(query);
    const elementsWithIds = [];
    for (const element of elements) {
      if (element.id) elementsWithIds.push(element);
    }

    // get first or previous element in top half of view
    for (let i = 0; i < elementsWithIds.length; i++) {
      const element = elementsWithIds[i];
      const prevElement = elementsWithIds[Math.max(0, i - 1)];
      if (element.getBoundingClientRect().top >= 0) {
        if (element.getBoundingClientRect().top < window.innerHeight / 2)
          return element.id;
        else return prevElement.id;
      }
    }
  }

  // make panel
  function makePanel() {
    // create panel
    const panel = document.createElement("div");
    panel.id = "toc_panel";
    if (bullets === "true") panel.dataset.bullets = "true";

    // create header
    const header = document.createElement("div");
    header.id = "toc_header";

    // create toc button
    const button = document.createElement("button");
    button.id = "toc_button";
    button.innerHTML = document.querySelector(".icon_th_list").innerHTML;
    button.title = "Table of Contents";
    button.classList.add("icon_button");

    // create header text
    const text = document.createElement("h4");
    text.innerHTML = "Table of Contents";

    // create container for toc list
    const list = document.createElement("div");
    list.id = "toc_list";

    // attach click listeners
    panel.addEventListener("click", onPanelClick);
    header.addEventListener("click", onHeaderClick);
    button.addEventListener("click", onButtonClick);

    // attach elements
    header.appendChild(button);
    header.appendChild(text);
    panel.appendChild(header);
    panel.appendChild(list);

    return panel;
  }

  // create toc entries (links) to each element of the specified types
  function makeEntries(panel) {
    const elements = document.querySelectorAll(typesQuery);
    for (const element of elements) {
      // do not add link if element doesn't have assigned id
      if (!element.id) continue;

      // create link/list item
      const link = document.createElement("a");
      link.classList.add("toc_link");
      switch (element.tagName.toLowerCase()) {
        case "h1":
          link.dataset.level = "1";
          break;
        case "h2":
          link.dataset.level = "2";
          break;
        case "h3":
          link.dataset.level = "3";
          break;
        case "h4":
          link.dataset.level = "4";
          break;
      }
      link.title = element.innerText;
      let text = element.innerText;
      if (text.length > charLimit) text = text.slice(0, charLimit) + "...";
      link.innerHTML = text;
      link.href = "#" + element.id;
      link.addEventListener("click", onLinkClick);

      // attach link
      panel.querySelector("#toc_list").appendChild(link);
    }
  }

  // when panel is clicked
  function onPanelClick(event) {
    // stop click from propagating to window/document and closing panel
    event.stopPropagation();
  }

  // when header itself is clicked
  function onHeaderClick(event) {
    togglePanel();
  }

  // when button is clicked
  function onButtonClick(event) {
    togglePanel();
    // stop header underneath button from also being clicked
    event.stopPropagation();
  }

  // when link is clicked
  function onLinkClick(event) {
    if (clickClose === "true" || (clickClose === "auto" && isSmallScreen()))
      closePanel();
    else openPanel();
  }

  // open panel if closed, close if opened
  function togglePanel() {
    const panel = document.getElementById("toc_panel");
    if (!panel) return;

    if (panel.dataset.open === "true") closePanel();
    else openPanel();
  }

  // open panel
  function openPanel() {
    const panel = document.getElementById("toc_panel");
    if (panel) panel.dataset.open = "true";
  }

  // close panel
  function closePanel() {
    const panel = document.getElementById("toc_panel");
    if (panel) panel.dataset.open = "false";
  }

  // start script when document is finished loading
  window.addEventListener("load", start);
</script>

<!-- th list icon -->

<template class="icon_th_list">
  <!-- modified from: https://fontawesome.com/icons/th-list -->
  <svg width="16" height="16" viewBox="0 0 512 512" tabindex="-1">
    <path
      fill="currentColor"
      d="M96 96c0 26.51-21.49 48-48 48S0 122.51 0 96s21.49-48 48-48 48 21.49 48 48zM48 208c-26.51 0-48 21.49-48 48s21.49 48 48 48 48-21.49 48-48-21.49-48-48-48zm0 160c-26.51 0-48 21.49-48 48s21.49 48 48 48 48-21.49 48-48-21.49-48-48-48zm96-236h352c8.837 0 16-7.163 16-16V76c0-8.837-7.163-16-16-16H144c-8.837 0-16 7.163-16 16v40c0 8.837 7.163 16 16 16zm0 160h352c8.837 0 16-7.163 16-16v-40c0-8.837-7.163-16-16-16H144c-8.837 0-16 7.163-16 16v40c0 8.837 7.163 16 16 16zm0 160h352c8.837 0 16-7.163 16-16v-40c0-8.837-7.163-16-16-16H144c-8.837 0-16 7.163-16 16v40c0 8.837 7.163 16 16 16z"
      tabindex="-1"
    ></path>
  </svg>
</template>

<style>
  @media only screen {
    /* toc panel */
    #toc_panel {
      box-sizing: border-box;
      position: fixed;
      top: 0;
      left: 0;
      background: #ffffff;
      box-shadow: 0 0 20px rgba(0, 0, 0, 0.05);
      z-index: 2;
    }

    /* toc panel when closed */
    #toc_panel[data-open="false"] {
      min-width: 60px;
      width: 60px;
      height: 60px;
      border-right: solid 1px #bdbdbd;
      border-bottom: solid 1px #bdbdbd;
    }

    /* toc panel when open */
    #toc_panel[data-open="true"] {
      min-width: 260px;
      max-width: 480px;
      /* keep panel edge consistent distance away from "page" edge */
      width: calc(((100vw - 8.5in) / 2) - 30px - 40px);
      bottom: 0;
      border-right: solid 1px #bdbdbd;
    }

    /* toc panel header */
    #toc_header {
      box-sizing: border-box;
      display: flex;
      flex-direction: row;
      align-items: center;
      height: 60px;
      margin: 0;
      padding: 20px;
    }

    /* toc panel header when hovered */
    #toc_header:hover {
      cursor: pointer;
    }

    /* toc panel header when panel open */
    #toc_panel[data-open="true"] > #toc_header {
      border-bottom: solid 1px #bdbdbd;
    }

    /* toc open/close header button */
    #toc_button {
      margin-right: 20px;
    }

    /* hide toc list and header text when closed */
    #toc_panel[data-open="false"] > #toc_header > *:not(#toc_button),
    #toc_panel[data-open="false"] > #toc_list {
      display: none;
    }

    /* toc list of entries */
    #toc_list {
      box-sizing: border-box;
      width: 100%;
      padding: 20px;
      position: absolute;
      top: calc(60px + 1px);
      bottom: 0;
      overflow: auto;
    }

    /* toc entry, link to section in document */
    .toc_link {
      display: block;
      padding: 5px;
      position: relative;
      font-weight: 600;
      text-decoration: none;
    }

    /* toc entry when hovered or when "viewed" */
    .toc_link:hover,
    .toc_link[data-viewing="true"] {
      background: #f5f5f5;
    }

    /* toc entry, level 1 indentation */
    .toc_link[data-level="1"] {
      margin-left: 0;
    }

    /* toc entry, level 2 indentation */
    .toc_link[data-level="2"] {
      margin-left: 20px;
    }

    /* toc entry, level 3 indentation */
    .toc_link[data-level="3"] {
      margin-left: 40px;
    }

    /* toc entry, level 4 indentation */
    .toc_link[data-level="4"] {
      margin-left: 60px;
    }

    /* toc entry bullets */
    #toc_panel[data-bullets="true"] .toc_link[data-level]:before {
      position: absolute;
      left: -15px;
      top: -1px;
      font-size: 1.5em;
    }

    /* toc entry, level 2 bullet */
    #toc_panel[data-bullets="true"] .toc_link[data-level="2"]:before {
      content: "\2022";
    }

    /* toc entry, level 3 bullet */
    #toc_panel[data-bullets="true"] .toc_link[data-level="3"]:before {
      content: "\25AB";
    }

    /* toc entry, level 4 bullet */
    #toc_panel[data-bullets="true"] .toc_link[data-level="4"]:before {
      content: "-";
    }
  }

  /* when on screen < 8.5in wide */
  @media only screen and (max-width: 8.5in) {
    /* push <body> ("page") element down to make room for toc icon */
    .toc_body_nudge {
      padding-top: 60px;
    }

    /* toc icon when panel closed and not hovered */
    #toc_panel[data-open="false"]:not(:hover) {
      background: rgba(255, 255, 255, 0.75);
    }
  }

  /* always hide toc panel on print */
  @media only print {
    #toc_panel {
      display: none;
    }
  }
</style>
<!-- 
  Tooltips Plugin

  Makes it such that when the user hovers or focuses a link to a citation or
  figure, a tooltip appears with a preview of the reference content, along with
  arrows to navigate between instances of the same reference in the document.
-->

<script type="module">
  // whether user must click off to close tooltip instead of just un-hovering
  const clickClose = "false";
  // delay (in ms) between opening and closing tooltip
  const delay = "100";

  // start script
  function start() {
    const links = getLinks();
    for (const link of links) {
      // attach hover and focus listeners to link
      link.addEventListener("mouseover", onLinkHover);
      link.addEventListener("mouseleave", onLinkUnhover);
      link.addEventListener("focus", onLinkFocus);
      link.addEventListener("touchend", onLinkTouch);
    }

    // attach mouse, key, and resize listeners to window
    window.addEventListener("mousedown", onClick);
    window.addEventListener("touchstart", onClick);
    window.addEventListener("keyup", onKeyUp);
    window.addEventListener("resize", onResize);
  }

  // when link is hovered
  function onLinkHover() {
    // function to open tooltip
    const delayOpenTooltip = function () {
      openTooltip(this);
    }.bind(this);

    // run open function after delay
    this.openTooltipTimer = window.setTimeout(delayOpenTooltip, delay);
  }

  // when mouse leaves link
  function onLinkUnhover() {
    // cancel opening tooltip
    window.clearTimeout(this.openTooltipTimer);

    // don't close on unhover if option specifies
    if (clickClose === "true") return;

    // function to close tooltip
    const delayCloseTooltip = function () {
      // if tooltip open and if mouse isn't over tooltip, close
      const tooltip = document.getElementById("tooltip");
      if (tooltip && !tooltip.matches(":hover")) closeTooltip();
    };

    // run close function after delay
    this.closeTooltipTimer = window.setTimeout(delayCloseTooltip, delay);
  }

  // when link is focused (tabbed to)
  function onLinkFocus(event) {
    openTooltip(this);
  }

  // when link is touched on touch screen
  function onLinkTouch(event) {
    // attempt to force hover state on first tap always, and trigger
    // regular link click (and navigation) on second tap
    if (event.target === document.activeElement) event.target.click();
    else {
      document.activeElement.blur();
      event.target.focus();
    }
    if (event.cancelable) event.preventDefault();
    event.stopPropagation();
    return false;
  }

  // when mouse is clicked anywhere in window
  function onClick(event) {
    closeTooltip();
  }

  // when key pressed
  function onKeyUp(event) {
    if (!event || !event.key) return;

    switch (event.key) {
      // trigger click of prev button
      case "ArrowLeft":
        const prevButton = document.getElementById("tooltip_prev_button");
        if (prevButton) prevButton.click();
        break;
      // trigger click of next button
      case "ArrowRight":
        const nextButton = document.getElementById("tooltip_next_button");
        if (nextButton) nextButton.click();
        break;
      // close on esc
      case "Escape":
        closeTooltip();
        break;
    }
  }

  // when window is resized or zoomed
  function onResize() {
    closeTooltip();
  }

  // get all links of types we wish to handle
  function getLinks() {
    const queries = [];
    // exclude buttons, anchor links, toc links, etc
    const exclude =
      ":not(.button):not(.icon_button):not(.anchor):not(.toc_link)";
    queries.push('a[href^="#ref-"]' + exclude); // citation links
    queries.push('a[href^="#fig:"]' + exclude); // figure links
    const query = queries.join(", ");
    return document.querySelectorAll(query);
  }

  // get links with same target, get index of link in set, get total
  // same links
  function getSameLinks(link) {
    const sameLinks = [];
    const links = getLinks();
    for (const otherLink of links) {
      if (otherLink.getAttribute("href") === link.getAttribute("href"))
        sameLinks.push(otherLink);
    }

    return {
      elements: sameLinks,
      index: sameLinks.indexOf(link),
      total: sameLinks.length,
    };
  }

  // open tooltip
  function openTooltip(link) {
    // delete tooltip if it exists, start fresh
    closeTooltip();

    // make tooltip element
    const tooltip = makeTooltip(link);

    // if source couldn't be found and tooltip not made, exit
    if (!tooltip) return;

    // make navbar elements
    const navBar = makeNavBar(link);
    if (navBar) tooltip.firstElementChild.appendChild(navBar);

    // attach tooltip to page
    document.body.appendChild(tooltip);

    // position tooltip
    const position = function () {
      positionTooltip(link);
    };
    position();

    // if tooltip contains images, position again after they've loaded
    const imgs = tooltip.querySelectorAll("img");
    for (const img of imgs) img.addEventListener("load", position);
  }

  // close (delete) tooltip
  function closeTooltip() {
    const tooltip = document.getElementById("tooltip");
    if (tooltip) tooltip.remove();
  }

  // make tooltip
  function makeTooltip(link) {
    // get target element that link points to
    const source = getSource(link);

    // if source can't be found, exit
    if (!source) return;

    // create new tooltip
    const tooltip = document.createElement("div");
    tooltip.id = "tooltip";
    const tooltipContent = document.createElement("div");
    tooltipContent.id = "tooltip_content";
    tooltip.appendChild(tooltipContent);

    // make copy of source node and put in tooltip
    const sourceCopy = makeCopy(source);
    tooltipContent.appendChild(sourceCopy);

    // attach mouse event listeners
    tooltip.addEventListener("click", onTooltipClick);
    tooltip.addEventListener("mousedown", onTooltipClick);
    tooltip.addEventListener("touchstart", onTooltipClick);
    tooltip.addEventListener("mouseleave", onTooltipUnhover);

    // (for interaction with lightbox plugin)
    // transfer click on tooltip copied img to original img
    const sourceImg = source.querySelector("img");
    const sourceCopyImg = sourceCopy.querySelector("img");
    if (sourceImg && sourceCopyImg) {
      const clickImg = function () {
        sourceImg.click();
        closeTooltip();
      };
      sourceCopyImg.addEventListener("click", clickImg);
    }

    return tooltip;
  }

  // make carbon copy of html dom element
  function makeCopy(source) {
    const sourceCopy = source.cloneNode(true);

    // delete elements marked with ignore (eg anchor and jump buttons)
    const deleteFromCopy = sourceCopy.querySelectorAll('[data-ignore="true"]');
    for (const element of deleteFromCopy) element.remove();

    // delete certain element attributes
    const attributes = [
      "id",
      "data-collapsed",
      "data-selected",
      "data-highlighted",
      "data-glow",
      "class",
    ];
    for (const attribute of attributes) {
      sourceCopy.removeAttribute(attribute);
      const elements = sourceCopy.querySelectorAll("[" + attribute + "]");
      for (const element of elements) element.removeAttribute(attribute);
    }

    return sourceCopy;
  }

  // when tooltip is clicked
  function onTooltipClick(event) {
    // when user clicks on tooltip, stop click from transferring
    // outside of tooltip (eg, click off to close tooltip, or eg click
    // off to unhighlight same refs)
    event.stopPropagation();
  }

  // when tooltip is unhovered
  function onTooltipUnhover(event) {
    if (clickClose === "true") return;

    // make sure new mouse/touch/focus no longer over tooltip or any
    // element within it
    const tooltip = document.getElementById("tooltip");
    if (!tooltip) return;
    if (this.contains(event.relatedTarget)) return;

    closeTooltip();
  }

  // make nav bar to go betwen prev/next instances of same reference
  function makeNavBar(link) {
    // find other links to the same source
    const sameLinks = getSameLinks(link);

    // don't show nav bar when singular reference
    if (sameLinks.total <= 1) return;

    // find prev/next links with same target
    const prevLink = getPrevLink(link, sameLinks);
    const nextLink = getNextLink(link, sameLinks);

    // create nav bar
    const navBar = document.createElement("div");
    navBar.id = "tooltip_nav_bar";
    const text = sameLinks.index + 1 + " of " + sameLinks.total;

    // create nav bar prev/next buttons
    const prevButton = document.createElement("button");
    const nextButton = document.createElement("button");
    prevButton.id = "tooltip_prev_button";
    nextButton.id = "tooltip_next_button";
    prevButton.title =
      "Jump to the previous occurence of this item in the document [←]";
    nextButton.title =
      "Jump to the next occurence of this item in the document [→]";
    prevButton.classList.add("icon_button");
    nextButton.classList.add("icon_button");
    prevButton.innerHTML = document.querySelector(".icon_caret_left").innerHTML;
    nextButton.innerHTML =
      document.querySelector(".icon_caret_right").innerHTML;
    navBar.appendChild(prevButton);
    navBar.appendChild(document.createTextNode(text));
    navBar.appendChild(nextButton);

    // attach click listeners to buttons
    prevButton.addEventListener("click", function () {
      onPrevNextClick(link, prevLink);
    });
    nextButton.addEventListener("click", function () {
      onPrevNextClick(link, nextLink);
    });

    return navBar;
  }

  // get previous link with same target
  function getPrevLink(link, sameLinks) {
    if (!sameLinks) sameLinks = getSameLinks(link);
    // wrap index to other side if < 1
    let index;
    if (sameLinks.index - 1 >= 0) index = sameLinks.index - 1;
    else index = sameLinks.total - 1;
    return sameLinks.elements[index];
  }

  // get next link with same target
  function getNextLink(link, sameLinks) {
    if (!sameLinks) sameLinks = getSameLinks(link);
    // wrap index to other side if > total
    let index;
    if (sameLinks.index + 1 <= sameLinks.total - 1) index = sameLinks.index + 1;
    else index = 0;
    return sameLinks.elements[index];
  }

  // get element that is target of link or url hash
  function getSource(link) {
    const hash = link ? link.hash : window.location.hash;
    const id = hash.slice(1);
    let target = document.querySelector('[id="' + id + '"]');
    if (!target) return;

    // if ref or figure, modify target to get expected element
    if (id.indexOf("ref-") === 0) target = target.querySelector(":nth-child(2)");
    else if (id.indexOf("fig:") === 0) target = target.querySelector("figure");

    return target;
  }

  // when prev/next arrow button is clicked
  function onPrevNextClick(link, prevNextLink) {
    if (link && prevNextLink)
      goToElement(prevNextLink, window.innerHeight * 0.5);
  }

  // scroll to and focus element
  function goToElement(element, offset) {
    // expand accordion section if collapsed
    expandElement(element);
    const y =
      getRectInView(element).top -
      getRectInView(document.documentElement).top -
      (offset || 0);
    // trigger any function listening for "onscroll" event
    window.dispatchEvent(new Event("scroll"));
    window.scrollTo(0, y);
    document.activeElement.blur();
    element.focus();
  }

  // determine position to place tooltip based on link position in
  // viewport and tooltip size
  function positionTooltip(link, left, top) {
    const tooltipElement = document.getElementById("tooltip");
    if (!tooltipElement) return;

    // get convenient vars for position/dimensions of
    // link/tooltip/page/view
    link = getRectInPage(link);
    const tooltip = getRectInPage(tooltipElement);
    const view = getRectInPage();

    // horizontal positioning
    if (left)
      // use explicit value
      left = left;
    else if (link.left + tooltip.width < view.right)
      // fit tooltip to right of link
      left = link.left;
    else if (link.right - tooltip.width > view.left)
      // fit tooltip to left of link
      left = link.right - tooltip.width;
    // center tooltip in view
    else left = (view.right - view.left) / 2 - tooltip.width / 2;

    // vertical positioning
    if (top)
      // use explicit value
      top = top;
    else if (link.top - tooltip.height > view.top)
      // fit tooltip above link
      top = link.top - tooltip.height;
    else if (link.bottom + tooltip.height < view.bottom)
      // fit tooltip below link
      top = link.bottom;
    else {
      // center tooltip in view
      top = view.top + view.height / 2 - tooltip.height / 2;
      // nudge off of link to left/right if possible
      if (link.right + tooltip.width < view.right) left = link.right;
      else if (link.left - tooltip.width > view.left)
        left = link.left - tooltip.width;
    }

    tooltipElement.style.left = left + "px";
    tooltipElement.style.top = top + "px";
  }

  // start script when document is finished loading
  window.addEventListener("load", start);
</script>

<!-- caret left icon -->

<template class="icon_caret_left">
  <!-- modified from: https://fontawesome.com/icons/caret-left -->
  <svg width="16" height="16" viewBox="0 0 192 512">
    <path
      fill="currentColor"
      d="M192 127.338v257.324c0 17.818-21.543 26.741-34.142 14.142L29.196 270.142c-7.81-7.81-7.81-20.474 0-28.284l128.662-128.662c12.599-12.6 34.142-3.676 34.142 14.142z"
    ></path>
  </svg>
</template>

<!-- caret right icon -->

<template class="icon_caret_right">
  <!-- modified from: https://fontawesome.com/icons/caret-right -->
  <svg width="16" height="16" viewBox="0 0 192 512">
    <path
      fill="currentColor"
      d="M0 384.662V127.338c0-17.818 21.543-26.741 34.142-14.142l128.662 128.662c7.81 7.81 7.81 20.474 0 28.284L34.142 398.804C21.543 411.404 0 402.48 0 384.662z"
    ></path>
  </svg>
</template>

<style>
  @media only screen {
    /* tooltip container */
    #tooltip {
      position: absolute;
      width: 50%;
      min-width: 240px;
      max-width: 75%;
      z-index: 1;
    }

    /* tooltip content */
    #tooltip_content {
      margin-bottom: 5px;
      padding: 20px;
      border-radius: 5px;
      border: solid 1px #bdbdbd;
      box-shadow: 0 0 20px rgba(0, 0, 0, 0.05);
      background: #ffffff;
      overflow-wrap: break-word;
    }

    /* tooltip copy of paragraphs and figures */
    #tooltip_content > p,
    #tooltip_content > figure {
      margin: 0;
      max-height: 320px;
      overflow-y: auto;
    }

    /* tooltip copy of <img> */
    #tooltip_content > figure > img,
    #tooltip_content > figure > svg {
      max-height: 260px;
    }

    /* navigation bar */
    #tooltip_nav_bar {
      margin-top: 10px;
      text-align: center;
    }

    /* navigation bar previous/next buton */
    #tooltip_nav_bar > .icon_button {
      position: relative;
      top: 3px;
    }

    /* navigation bar previous button */
    #tooltip_nav_bar > .icon_button:first-of-type {
      margin-right: 5px;
    }

    /* navigation bar next button */
    #tooltip_nav_bar > .icon_button:last-of-type {
      margin-left: 5px;
    }
  }

  /* always hide tooltip on print */
  @media only print {
    #tooltip {
      display: none;
    }
  }
</style>
<!--
  Analytics Plugin (third-party) 
  
  Copy and paste code from Google Analytics or similar service here.
-->
<!-- 
  Annotations Plugin

  Allows public annotation of the  manuscript. See https://web.hypothes.is/.
-->

<script type="module">
  // configuration
  window.hypothesisConfig = function () {
    return {
      branding: {
        accentColor: "#2196f3",
        appBackgroundColor: "#f8f8f8",
        ctaBackgroundColor: "#f8f8f8",
        ctaTextColor: "#000000",
        selectionFontFamily: "Open Sans, Helvetica, sans serif",
        annotationFontFamily: "Open Sans, Helvetica, sans serif",
      },
    };
  };

  // hypothesis client script
  const embed = "https://hypothes.is/embed.js";
  // hypothesis annotation count query url
  const query = "https://api.hypothes.is/api/search?limit=0&url=";

  // start script
  function start() {
    const button = makeButton();
    document.body.insertBefore(button, document.body.firstChild);
    insertCount(button);
  }

  // make button
  function makeButton() {
    // create button
    const button = document.createElement("button");
    button.id = "hypothesis_button";
    button.innerHTML = document.querySelector(".icon_hypothesis").innerHTML;
    button.title = "Hypothesis annotations";
    button.classList.add("icon_button");

    function onClick(event) {
      onButtonClick(event, button);
    }

    // attach click listeners
    button.addEventListener("click", onClick);

    return button;
  }

  // insert annotations count
  async function insertCount(button) {
    // get annotation count from Hypothesis based on url
    let count = "-";
    try {
      const canonical = document.querySelector('link[rel="canonical"]');
      const location = window.location;
      const url = encodeURIComponent((canonical || location).href);
      const response = await fetch(query + url);
      const json = await response.json();
      count = json.total || "-";
    } catch (error) {
      console.log(error);
    }

    // put count into button
    const counter = document.createElement("span");
    counter.id = "hypothesis_count";
    counter.innerHTML = count;
    button.title = "View " + count + " Hypothesis annotations";
    button.append(counter);
  }

  // when button is clicked
  function onButtonClick(event, button) {
    const script = document.createElement("script");
    script.src = embed;
    document.body.append(script);
    button.remove();
  }

  // start script when document is finished loading
  window.addEventListener("load", start);
</script>

<!-- hypothesis icon -->

<template class="icon_hypothesis">
  <!-- modified from: https://simpleicons.org/icons/hypothesis.svg / https://git.io/Jf1VB -->
  <svg width="16" height="16" viewBox="0 0 24 24" tabindex="-1">
    <path
      fill="currentColor"
      d="M3.43 0C2.5 0 1.72 .768 1.72 1.72V18.86C1.72 19.8 2.5 20.57 3.43 20.57H9.38L12 24L14.62 20.57H20.57C21.5 20.57 22.29 19.8 22.29 18.86V1.72C22.29 .77 21.5 0 20.57 0H3.43M5.14 3.43H7.72V9.43S8.58 7.72 10.28 7.72C12 7.72 13.74 8.57 13.74 11.24V17.14H11.16V12C11.16 10.61 10.28 10.07 9.43 10.29C8.57 10.5 7.72 11.41 7.72 13.29V17.14H5.14V3.43M18 13.72C18.95 13.72 19.72 14.5 19.72 15.42A1.71 1.71 0 0 1 18 17.13A1.71 1.71 0 0 1 16.29 15.42C16.29 14.5 17.05 13.71 18 13.71Z"
      tabindex="-1"
    ></path>
  </svg>
</template>

<style>
  /* hypothesis activation button */
  #hypothesis_button {
    box-sizing: border-box;
    position: fixed;
    top: 0;
    right: 0;
    width: 60px;
    height: 60px;
    background: #ffffff;
    border-radius: 0;
    border-left: solid 1px #bdbdbd;
    border-bottom: solid 1px #bdbdbd;
    box-shadow: 0 0 20px rgba(0, 0, 0, 0.05);
    z-index: 2;
  }

  /* hypothesis button svg */
  #hypothesis_button > svg {
    position: relative;
    top: -4px;
  }

  /* hypothesis annotation count */
  #hypothesis_count {
    position: absolute;
    left: 0;
    right: 0;
    bottom: 5px;
  }

  /* side panel */
  .annotator-frame {
    width: 280px !important;
  }

  /* match highlight color to rest of theme */
  .annotator-highlights-always-on .annotator-hl {
    background-color: #ffeb3b !important;
  }

  /* match focused color to rest of theme */
  .annotator-hl.annotator-hl-focused {
    background-color: #ff8a65 !important;
  }

  /* match bucket bar color to rest of theme */
  .annotator-bucket-bar {
    background: #f5f5f5 !important;
  }

  /* always hide button, toolbar, and tooltip on print */
  @media only print {
    #hypothesis_button {
      display: none;
    }

    .annotator-frame {
      display: none !important;
    }

    hypothesis-adder {
      display: none !important;
    }
  }
</style>
<!-- 
  Mathjax Plugin (third-party) 

  Allows the proper rendering of math/equations written in LaTeX.
  See https://www.mathjax.org/.
-->

<script type="text/x-mathjax-config">
  // configuration
  MathJax.Hub.Config({
    "CommonHTML": { linebreaks: { automatic: true } },
    "HTML-CSS": { linebreaks: { automatic: true } },
    "SVG": { linebreaks: { automatic: true } },
    "fast-preview": { disabled: true }
  });
</script>

<script
  src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js?config=TeX-MML-AM_CHTML"
  integrity="sha512-0doc9hKxR3PYwso42RD1p5ySZpzzuDiOwMrdCEh2WdJZCjcmFKc/wEnL+z8fBQrnHoiNWbo+3fiGkOYXBdQp4A=="
  crossorigin="anonymous"
></script>

<style>
  /* mathjax containers */
  .math.display > span:not(.MathJax_Preview) {
    /* turn inline element (no dimensions) into block (allows fixed width and thus scrolling) */
    display: flex !important;
    overflow-x: auto !important;
    overflow-y: hidden !important;
    justify-content: center;
    align-items: center;
    margin: 0 !important;
  }

  /* right click menu */
  .MathJax_Menu {
    border-radius: 5px !important;
    border: solid 1px #bdbdbd !important;
    box-shadow: none !important;
  }

  /* equation auto-number */
  span[id^="eq:"] > span.math.display + span {
    font-weight: 600;
  }

  /* equation */
  span[id^="eq:"] > span.math.display > span {
    /* nudge to make room for equation auto-number and anchor */
    margin-right: 60px !important;
  }
</style>
</body>
</html>
